<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.353">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Meenu Patil">
<meta name="author" content="Abhishek Kumar">
<meta name="author" content="Pardeep Kumar">
<meta name="author" content="Anand Narain Singh">
<meta name="dcterms.date" content="2023-07-17">

<title>Abhishek Kumar - Mycorrhizal fungi accelerates litter decomposition rates in forest ecosystems</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>

  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<link rel="stylesheet" href="../../styles.css">
</head>

<body class="floating nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="../../index.html">
    <span class="navbar-title">Abhishek Kumar</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../../about.html" rel="" target="">
 <span class="menu-text">About</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../index.html" rel="" target="">
 <span class="menu-text">Blog</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../publications.html" rel="" target="">
 <span class="menu-text">Publications</span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/kumar-a" rel="" target=""><i class="bi bi-github" role="img">
</i> 
 <span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://twitter.com/abkumar_" rel="" target=""><i class="bi bi-twitter" role="img">
</i> 
 <span class="menu-text"></span></a>
  </li>  
</ul>
            <div class="quarto-navbar-tools">
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation floating overflow-auto">
    <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Contents</h2>
   
  <ul>
  <li><a href="#abstract" id="toc-abstract" class="nav-link active" data-scroll-target="#abstract">Abstract</a></li>
  <li><a href="#graphical-abstract" id="toc-graphical-abstract" class="nav-link" data-scroll-target="#graphical-abstract">Graphical abstract</a></li>
  <li><a href="#introduction" id="toc-introduction" class="nav-link" data-scroll-target="#introduction">Introduction</a></li>
  <li><a href="#methodology" id="toc-methodology" class="nav-link" data-scroll-target="#methodology">Methodology</a>
  <ul class="collapse">
  <li><a href="#literature-search" id="toc-literature-search" class="nav-link" data-scroll-target="#literature-search">Literature search</a></li>
  <li><a href="#article-screening" id="toc-article-screening" class="nav-link" data-scroll-target="#article-screening">Article screening</a></li>
  <li><a href="#data-extraction" id="toc-data-extraction" class="nav-link" data-scroll-target="#data-extraction">Data extraction</a></li>
  <li><a href="#phylogenetic-data" id="toc-phylogenetic-data" class="nav-link" data-scroll-target="#phylogenetic-data">Phylogenetic data</a></li>
  <li><a href="#effect-size-and-variance" id="toc-effect-size-and-variance" class="nav-link" data-scroll-target="#effect-size-and-variance">Effect size and variance</a></li>
  <li><a href="#meta-analytic-model" id="toc-meta-analytic-model" class="nav-link" data-scroll-target="#meta-analytic-model">Meta-analytic model</a></li>
  <li><a href="#heterogeneity" id="toc-heterogeneity" class="nav-link" data-scroll-target="#heterogeneity">Heterogeneity</a></li>
  <li><a href="#publication-bias" id="toc-publication-bias" class="nav-link" data-scroll-target="#publication-bias">Publication bias</a></li>
  </ul></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results">Results</a>
  <ul class="collapse">
  <li><a href="#overview-of-dataset" id="toc-overview-of-dataset" class="nav-link" data-scroll-target="#overview-of-dataset">Overview of dataset</a></li>
  <li><a href="#overall-effect-estimate" id="toc-overall-effect-estimate" class="nav-link" data-scroll-target="#overall-effect-estimate">Overall effect estimate</a></li>
  <li><a href="#publication-bias-1" id="toc-publication-bias-1" class="nav-link" data-scroll-target="#publication-bias-1">Publication bias</a></li>
  </ul></li>
  <li><a href="#discussion" id="toc-discussion" class="nav-link" data-scroll-target="#discussion">Discussion</a></li>
  <li><a href="#conclusion" id="toc-conclusion" class="nav-link" data-scroll-target="#conclusion">Conclusion</a></li>
  
  
  
  
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references">References</a></li>
  </ul>
</nav>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Mycorrhizal fungi accelerates litter decomposition rates in forest ecosystems</h1>
</div>


<div class="quarto-title-meta-author">
  <div class="quarto-title-meta-heading">Authors</div>
  <div class="quarto-title-meta-heading">Affiliation</div>
  
    <div class="quarto-title-meta-contents">
    <p class="author"><a href="https://www.researchgate.net/profile/Meenu-Patil">Meenu Patil</a> <a href="https://orcid.org/0000-0002-7664-7877" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            <a href="https://puchd.ac.in">
            Panjab University, Chandigarh
            </a>
          </p>
      </div>
      <div class="quarto-title-meta-contents">
    <p class="author"><a href="https://akumar.netlify.app">Abhishek Kumar</a> <a href="https://orcid.org/0000-0003-2252-7623" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            <a href="https://puchd.ac.in">
            Panjab University, Chandigarh
            </a>
          </p>
      </div>
      <div class="quarto-title-meta-contents">
    <p class="author"><a href="https://www.researchgate.net/profile/Pardeep-Kumar-22">Pardeep Kumar</a> <a href="https://orcid.org/0000-0001-6707-1485" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            <a href="https://puchd.ac.in">
            Panjab University, Chandigarh
            </a>
          </p>
      </div>
      <div class="quarto-title-meta-contents">
    <p class="author"><a href="https://www.researchgate.net/profile/Anand-Singh-15">Anand Narain Singh</a> <a href="https://orcid.org/0000-0002-0148-8680" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            <a href="https://puchd.ac.in">
            Panjab University, Chandigarh
            </a>
          </p>
      </div>
    </div>

<div class="quarto-title-meta">

      
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">July 17, 2023</p>
    </div>
  </div>
  
    
  </div>
  

</header>

<section id="abstract" class="level2">
<h2 class="anchored" data-anchor-id="abstract">Abstract</h2>
<p>The role of mycorrhizal fungi in modulating litter decomposition rates is complex, with potential outcomes of both facilitation and competition. However, the magnitude and direction of these effects have remained uncertain due to limited empirical evidence. We conducted a meticulous meta-analysis of published peer-reviewed literature to address this knowledge gap. Surprisingly, our systematic literature search, guided by pre-defined inclusion criteria, yielded only five comparable studies. Despite this constraint, our dataset comprised 14 estimates across nine species to evaluate the effects of mycorrhizal fungi on litter decomposition rates. Applying a random-effect meta-analytic model, we discovered compelling evidence (<em>RR</em> = 1.07, <em>Z</em> = 3.58, <em>p</em> &lt; 0.001) supporting accelerated litter decomposition rates in the presence of mycorrhizal fungi. However, the influence of mycorrhizal fungi on decomposition varied significantly with species identity, while no discernible effects were observed based on phylogeny. These findings challenge the generality of the widely recognised “Gadgil effect” and underscore the importance of species-specific considerations. Our results highlight the potential of mycorrhizal-mediated mechanisms to enhance litter decomposition, prompting a re-evaluation of nutrient cycling dynamics. Incorporating mycorrhizal effects into ecological models and management strategies could significantly advance our understanding of ecosystem responses to global environmental change. Overall, this work contributes to the broader understanding of the ecological implications of mycorrhizal fungi and calls for additional research efforts to elucidate the mechanisms and generalizability of these effects comprehensively.</p>
<p><strong>Keywords:</strong> gadgil effect; ectomycorrhizal fungi; arbuscular mycorrhizal fungi; forest ecosystem; litter decomposition; meta-analysis</p>
</section>
<section id="graphical-abstract" class="level2">
<h2 class="anchored" data-anchor-id="graphical-abstract">Graphical abstract</h2>
<p><img src="data/graphical_abstract.svg" class="img-fluid"></p>
</section>
<section id="introduction" class="level2">
<h2 class="anchored" data-anchor-id="introduction">Introduction</h2>
<p>Global environmental change is expected to substantially alter the biogeochemistry and nutrient cycling in terrestrial ecosystems <span class="citation" data-cites="Shu2019">(<a href="#ref-Shu2019" role="doc-biblioref">Shu et al. 2019</a>)</span>. Among terrestrial ecosystems, forest ecosystems acts as significant sink for soil organic matter and therefore, substantially influence the global nutrient cycling <span class="citation" data-cites="Pan2011">(<a href="#ref-Pan2011" role="doc-biblioref">Pan et al. 2011</a>)</span>. The cycling of nutrients is primarily regulated by the process of litter decomposition, which releases nutrients from plant biomass to the soils <span class="citation" data-cites="Shu2019 Patil2020azb">(<a href="#ref-Shu2019" role="doc-biblioref">Shu et al. 2019</a>; <a href="#ref-Patil2020azb" role="doc-biblioref">Patil, Kumar, Kumar, and Singh 2020</a>)</span>. Much of the variations in the litter decomposition rates are explained by climate and litter quality <span class="citation" data-cites="Zhang2008 Cornwell2008 Patil2020srep">(<a href="#ref-Zhang2008" role="doc-biblioref">Zhang et al. 2008</a>; <a href="#ref-Cornwell2008" role="doc-biblioref">Cornwell et al. 2008</a>; <a href="#ref-Patil2020srep" role="doc-biblioref">Patil, Kumar, Kumar, Cheema, et al. 2020</a>)</span>. However, the remaining proportion of variation is often attributed to the composition and activity of decomposer organisms <span class="citation" data-cites="McGuire2010 Sulman2017">(<a href="#ref-McGuire2010" role="doc-biblioref">McGuire and Treseder 2010</a>; <a href="#ref-Sulman2017" role="doc-biblioref">Sulman et al. 2017</a>)</span>. Among decomposer organisms, bacteria and fungi are two major group organisms involved in litter decomposition <span class="citation" data-cites="Schneider2012 Heijden2015">(<a href="#ref-Schneider2012" role="doc-biblioref">Schneider et al. 2012</a>; <a href="#ref-Heijden2015" role="doc-biblioref">Heijden et al. 2015</a>)</span>. Although the role of saprotrophic fungi is appreciated in litter decomposition, mycorrhizal fungi can also substantially influence litter decomposition because they often act as mediators of nutrients between plants and soil <span class="citation" data-cites="Finlay2005 Frey2019 Kumar2021">(<a href="#ref-Finlay2005" role="doc-biblioref">Finlay 2005</a>; <a href="#ref-Frey2019" role="doc-biblioref">Frey 2019</a>; <a href="#ref-Kumar2021" role="doc-biblioref">Kumar et al. 2021</a>)</span>. Among the mycorrhizal associations, arbuscular mycorrhizal fungi (AMF) and ectomycorrhizal fungi (EMF) are commonly associated with vascular plants <span class="citation" data-cites="Brundrett2018">(<a href="#ref-Brundrett2018" role="doc-biblioref">Brundrett and Tedersoo 2018</a>)</span>. While the AMF dominates the tropical forests, EMF associations are frequently observed in extra-tropical forests <span class="citation" data-cites="Steidinger2019">(<a href="#ref-Steidinger2019" role="doc-biblioref">Steidinger et al. 2019</a>)</span>. Thus, the effects of mycorrhizal fungi are expected to vary not only among mycorrhizal groups, but also across the plant species, ecosystems and environmental conditions.</p>
<p>Mycorrhizal fungi have been observed to decrease the litter decomposition rates <span class="citation" data-cites="Gadgil1971 Gadgil1975">(<a href="#ref-Gadgil1971" role="doc-biblioref">Ruth L. Gadgil and Gadgil 1971</a>; <a href="#ref-Gadgil1975" role="doc-biblioref">Ruth L. Gadgil and Gadgil 1975</a>)</span>, which is frequently referred as ‘Gadgil effect’ in the literature <span class="citation" data-cites="Gadgil1971 Fernandez2016">(<a href="#ref-Gadgil1971" role="doc-biblioref">Ruth L. Gadgil and Gadgil 1971</a>; <a href="#ref-Fernandez2016" role="doc-biblioref">Fernandez and Kennedy 2016</a>)</span>. The ‘Gadgil effect’ has been observed for both ectomycorrhizal <span class="citation" data-cites="Gadgil1971">(<a href="#ref-Gadgil1971" role="doc-biblioref">Ruth L. Gadgil and Gadgil 1971</a>)</span> and arbuscular mycorrhizal fungi <span class="citation" data-cites="Leifheit2015">(<a href="#ref-Leifheit2015" role="doc-biblioref">Leifheit, Verbruggen, and Rillig 2015</a>)</span>. The resource competition between saprotrophs (free living microbes) and symbionts (mycorrhizal fungi) have been suggested to retard the decomposition rates in presence of mycorrhizal fungi <span class="citation" data-cites="Gadgil1971 Brzostek2015">(<a href="#ref-Gadgil1971" role="doc-biblioref">Ruth L. Gadgil and Gadgil 1971</a>; <a href="#ref-Brzostek2015" role="doc-biblioref">Brzostek et al. 2015</a>)</span>. The specialized enzymatic suite and foraging behaviour of these mycorrhizal fungi have been attributed to their ability to outcompete the saprotrophic fungi or other detritivore community <span class="citation" data-cites="Averill2014 Talbot2015">(<a href="#ref-Averill2014" role="doc-biblioref">Averill, Turner, and Finzi 2014</a>; <a href="#ref-Talbot2015" role="doc-biblioref">Talbot et al. 2015</a>)</span>. Further, a literature review <span class="citation" data-cites="Fernandez2016">(<a href="#ref-Fernandez2016" role="doc-biblioref">Fernandez and Kennedy 2016</a>)</span> has also ascribed Gadgil effect to nitrogen competition <span class="citation" data-cites="Orwin2011 Averill2014">(<a href="#ref-Orwin2011" role="doc-biblioref">Orwin et al. 2011</a>; <a href="#ref-Averill2014" role="doc-biblioref">Averill, Turner, and Finzi 2014</a>)</span>, chemical inhibition or allelopathy <span class="citation" data-cites="Krywolap1964 Rasanayagam1992">(<a href="#ref-Krywolap1964" role="doc-biblioref">Krywolap, Grand, and Casida Jr. 1964</a>; <a href="#ref-Rasanayagam1992" role="doc-biblioref">Rasanayagam and Jeffries 1992</a>)</span>, myco-parasitism <span class="citation" data-cites="Kubicek2011">(<a href="#ref-Kubicek2011" role="doc-biblioref">Kubicek et al. 2011</a>)</span>, and altered moisture-regimes <span class="citation" data-cites="Koide2003">(<a href="#ref-Koide2003" role="doc-biblioref">Koide and Wu 2003</a>)</span>.</p>
<p>In contrast to Gadgil effect, mycorrhizal fungi also reported to enhance the litter decomposition rates <span class="citation" data-cites="Frey2019">(<a href="#ref-Frey2019" role="doc-biblioref">Frey 2019</a>)</span> through extracellular enzymatic degradation <span class="citation" data-cites="Talbot2015">(<a href="#ref-Talbot2015" role="doc-biblioref">Talbot et al. 2015</a>)</span>, oxidation via Fenton chemistry <span class="citation" data-cites="Beeck2018">(<a href="#ref-Beeck2018" role="doc-biblioref">Beeck et al. 2018</a>)</span>, and priming effects, i.e., stimulation of growth and activities of saprotrophic decomposers by providing plant-derived carbon sources <span class="citation" data-cites="Herman2012 Gorka2019">(<a href="#ref-Herman2012" role="doc-biblioref">Herman et al. 2012</a>; <a href="#ref-Gorka2019" role="doc-biblioref">Gorka et al. 2019</a>)</span>. The priming effects appear to be more widespread as mycorrhizal fungi have limited genetic capabilities to produce organic matter degrading enzymes <span class="citation" data-cites="Frey2019">(<a href="#ref-Frey2019" role="doc-biblioref">Frey 2019</a>)</span>. The availability of labile exudates together with mycorrhizal necromass stimulates the activity of free-living bacteria and saprotrophic fungi, which leads to higher decomposition rates <span class="citation" data-cites="Fernandez2016">(<a href="#ref-Fernandez2016" role="doc-biblioref">Fernandez and Kennedy 2016</a>)</span>. Further, the the differences in ecophysiology of mycorrhizal groups are expected to modulate their effects on litter decomposition rates <span class="citation" data-cites="Tedersoo2019">(<a href="#ref-Tedersoo2019" role="doc-biblioref">Tedersoo and Bahram 2019</a>)</span>. For instance, the capacities for enzymatic degradation of organic matter widely vary among mycorrhizal taxa and it appears that EMF taxa have more degrading capacities than the AMF taxa <span class="citation" data-cites="Frey2019">(<a href="#ref-Frey2019" role="doc-biblioref">Frey 2019</a>)</span>. Therefore, considering the type of mycorrhizal association can be important in understanding the role of mycorrhizal fungi in nutrient cycling.</p>
<p>Thus, available literature suggest that mycorrhizal fungi can either enhance the litter decomposition by providing plant-derived carbon-source <span class="citation" data-cites="Zhu1996 Shah2016 Gui2017 Sterkenburg2018">(<a href="#ref-Zhu1996" role="doc-biblioref">Zhu and Ehrenfeld 1996</a>; <a href="#ref-Shah2016" role="doc-biblioref">Shah et al. 2016</a>; <a href="#ref-Gui2017" role="doc-biblioref">Gui et al. 2017</a>; <a href="#ref-Sterkenburg2018" role="doc-biblioref">Sterkenburg et al. 2018</a>)</span> or suppress it by competing for the same resources with saprotrophic fungi, also known as ‘Gadgil effect’ <span class="citation" data-cites="Gadgil1971 Gadgil1975 Brzostek2015">(<a href="#ref-Gadgil1971" role="doc-biblioref">Ruth L. Gadgil and Gadgil 1971</a>; <a href="#ref-Gadgil1975" role="doc-biblioref">Ruth L. Gadgil and Gadgil 1975</a>; <a href="#ref-Brzostek2015" role="doc-biblioref">Brzostek et al. 2015</a>)</span>. Nevertheless, some studies found little or no effects of mycorrhizal association on litter decomposition <span class="citation" data-cites="Staaf1988">(<a href="#ref-Staaf1988" role="doc-biblioref">Staaf 1988</a>)</span>. Therefore, the effects of mycorrhizal association on litter decomposition remained equivocal and highly variable in magnitude and direction, suggesting more complex mechanisms and contexts than previously thought <span class="citation" data-cites="Fernandez2016">(<a href="#ref-Fernandez2016" role="doc-biblioref">Fernandez and Kennedy 2016</a>)</span>. Although few exemplary syntheses about the effects of mycorrhizal association on litter decomposition exist <span class="citation" data-cites="Averill2014 Fernandez2016">(<a href="#ref-Averill2014" role="doc-biblioref">Averill, Turner, and Finzi 2014</a>; <a href="#ref-Fernandez2016" role="doc-biblioref">Fernandez and Kennedy 2016</a>)</span>, quantitative estimates and direction of such effects remained ambiguous. Further, none of the earlier syntheses conducted a formal meta-analysis and their general conclusions might be confounded by sampling error or other sources of variations. Therefore, our primary objective is to quantitatively synthesise the magnitude and direction of mycorrhizal effects on litter decomposition rates in forest ecosystems. Further, we can expect substantial differences in effect due to species identity and phylogenetic relatedness among them because each species may have different quality of litter which can influence the decomposition rates. In addition to this, we aim to identify the knowledge gaps and suggestions for future investigations. Specifically, we aim to address the following questions:</p>
<ol type="1">
<li>Do mycorrhizal presence affect rates of litter decomposition in forest ecosystems?</li>
<li>What is the magnitude and direction of mycorrhizal effects on litter decomposition rates?</li>
<li>Are these effects influenced by species or their phylogenetic relatedness?</li>
<li>Do these effects differ between angiosperms and gymnosperms?</li>
</ol>
<p>To accomplish this, we carried out a formal meta-analysis of available data from systematically identified comparable studies. In this meta-analysis, we included a forest plot showing the overall effect mycorrhizal fungi on litter decomposition, between-study heterogeneity, and a funnel plot for showing publication bias of studies. We explored different meta-analytic models and assessed the robustness of our estimates. Then, we discussed the results of all studies briefly. In the end, we draw conclusions of the mycorrhizal effect on litter decomposition based on the results of our meta-analysis.</p>
</section>
<section id="methodology" class="level2">
<h2 class="anchored" data-anchor-id="methodology">Methodology</h2>
<p>Our methodology broadly consisted of five phases, i.e., searching, screening, data extraction, statistical meta-analysis and sensitivity analysis. We followed the guidelines developed by Collaboration on Environmental Evidence <span class="citation" data-cites="CEE2018">(<a href="#ref-CEE2018" role="doc-biblioref">CEE 2018</a>)</span> to identify eligible studies using a systematic literature search and screening protocol. The protocol was developed by organising meetings and implemented in a similar manner as described earlier <span class="citation" data-cites="Kumar2022">(<a href="#ref-Kumar2022" role="doc-biblioref">Kumar et al. 2022</a>)</span>. Further, the quality and reproducibility of meta-analysis was ensured by adhering to previously recommended guidelines <span class="citation" data-cites="Koricheva2014 Nakagawa2017">(<a href="#ref-Koricheva2014" role="doc-biblioref">Koricheva and Gurevitch 2014</a>; <a href="#ref-Nakagawa2017" role="doc-biblioref">Nakagawa et al. 2017</a>)</span>.</p>
<section id="literature-search" class="level3">
<h3 class="anchored" data-anchor-id="literature-search">Literature search</h3>
<p>We systematically searched Scopus (<a href="https://scopus.com/" class="uri">https://scopus.com/</a>) and Web of Science Core Collection (<a href="https://webofknowledge.com/" class="uri">https://webofknowledge.com/</a>) on 17-18 February 2021 with the following search string:</p>
<blockquote class="blockquote">
<p>(mycorrhiza* OR ectomycorrhiza* OR “saprotrophic fungi” OR plant-fung*) AND (litter OR “litter decay” OR (litter AND decompos*) OR decomposition OR “nutrient acquisition”) AND (forest).</p>
</blockquote>
<p>This search string was developed by combining previously identified groups of keywords (see Supplementary Information). Additionally, Google Scholar (<a href="https://scholar.google.com/" class="uri">https://scholar.google.com/</a>) was searched on 18-Feb-2021 using “Mycorrhiza decomposition” and “Mycorrhiza nutrient acquisition”. Thus, we obtained 789 records from Scopus, 1,293 records from Web of Science Core Collection, and 14 records from Google Scholar. All these 2,096 records were exported as BibTex files for further processing (see details in Supplementary Information).</p>
</section>
<section id="article-screening" class="level3">
<h3 class="anchored" data-anchor-id="article-screening">Article screening</h3>
<p>Our article screening involved de-duplication, title and abstract screening, and full-text screening. We left 1,455 records after de-duplication, and 641 duplicates were removed (<a href="#fig-fig1">Figure&nbsp;1</a>). All these records were imported to freely available Mendeley reference management software for further screening. We manually screened titles and abstracts of each document following a pre-defined criterion (see Supplementary information). Based on the title and abstract screening, we excluded 1,265 articles and left with 190 articles (<a href="#fig-fig1">Figure&nbsp;1</a>). We could not retrieve the full-text of six articles (see Supplementary information Table S2) and, therefore, excluded these six articles. Next, the full text of each article was screened based on PICO inclusion and exclusion criteria (Supplementary information Table S3). Full-text screening resulted in the exclusion of 181 articles based on PICO criteria (<a href="#fig-fig1">Figure&nbsp;1</a>). Thus, we were finally left with only three eligible studies <span class="citation" data-cites="Entry1991 Zhu1996 Lang2021">(<a href="#ref-Entry1991" role="doc-biblioref">Entry, Rose, and Cromack 1991</a>; <a href="#ref-Zhu1996" role="doc-biblioref">Zhu and Ehrenfeld 1996</a>; <a href="#ref-Lang2021" role="doc-biblioref">Lang et al. 2021</a>)</span>. We further employed snowballing method to identify related eligible studies. We could not identify any suitable related study based on references of selected studies (backward snowballing). However, two more eligible articles <span class="citation" data-cites="Mayor2006 Fernandez2020">(<a href="#ref-Mayor2006" role="doc-biblioref">Mayor and Henkel 2006</a>; <a href="#ref-Fernandez2020" role="doc-biblioref">Fernandez, See, and Kennedy 2020</a>)</span> were identified through forward snowballing (citation tracking). Thus, five studies were included after article screening and snowballing.</p>
<div id="fig-fig1" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="data/fig1.svg" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;1: Schematic flow diagram for literature search, screening and inclusion process <span class="citation" data-cites="Haddaway2018">(modified from <a href="#ref-Haddaway2018" role="doc-biblioref">Haddaway et al. 2018</a>)</span></figcaption>
</figure>
</div>
</section>
<section id="data-extraction" class="level3">
<h3 class="anchored" data-anchor-id="data-extraction">Data extraction</h3>
<p>We needed data on litter decomposition rates in the presence and absence of mycorrhizal fungi to test our hypothesis. The litter decomposition rate is often expressed as annual decay constant usually estimated from Olson’s single exponential model <span class="citation" data-cites="Olson1963">(<a href="#ref-Olson1963" role="doc-biblioref">Olson 1963</a>)</span>. Therefore, we aimed to extract the decay constant (<span class="math inline">\(\kappa\)</span>) and associated standard error (SE) or standard deviation (SD) for each study to establish consistency in our analysis. Unfortunately, only a single study <span class="citation" data-cites="Entry1991">(<a href="#ref-Entry1991" role="doc-biblioref">Entry, Rose, and Cromack 1991</a>)</span> reported the data in required form. All other studies have represented litter decomposition rates as mass remaining at different time intervals <span class="citation" data-cites="Zhu1996 Mayor2006 Fernandez2020 Lang2021">(<a href="#ref-Zhu1996" role="doc-biblioref">Zhu and Ehrenfeld 1996</a>; <a href="#ref-Mayor2006" role="doc-biblioref">Mayor and Henkel 2006</a>; <a href="#ref-Fernandez2020" role="doc-biblioref">Fernandez, See, and Kennedy 2020</a>; <a href="#ref-Lang2021" role="doc-biblioref">Lang et al. 2021</a>)</span>. In such cases, we extracted each remaining mass value in triplicate (one mean value and two standard error of mean values) at each given time interval using the WebPlotDigitizer (<a href="https://automeris.io/WebPlotDigitizer/" class="uri">https://automeris.io/WebPlotDigitizer/</a>). Thus, the actual number of replicates of litterbags at each sampling time was not taken into account; instead, we considered each data point as a sample to obtain a more uniform and comparable measure of litter mass remaining. In case of any ambiguity in timings, we considered the time stated in the methodology section of the published article <span class="citation" data-cites="Lang2021">(<a href="#ref-Lang2021" role="doc-biblioref">Lang et al. 2021</a>)</span>.</p>
<p>When the mass remaining data was available, the decay constant (<span class="math inline">\(\kappa\)</span>) and its associated standard error (SE) was estimated by fitting a linearised form of Olson’s single exponential decay model <span class="citation" data-cites="Olson1963">(<a href="#ref-Olson1963" role="doc-biblioref">Olson 1963</a>)</span> as represented in (<a href="#eq-olson">Equation&nbsp;1</a>).</p>
<p><span id="eq-olson"><span class="math display">\[
\log(M_t) = -\kappa~t + \log(M_0)
\tag{1}\]</span></span></p>
<p>where <span class="math inline">\(M_0\)</span> is the initial litter mass, <span class="math inline">\(M_t\)</span> is the mass remaining at time <span class="math inline">\(t\)</span>, and <span class="math inline">\(\kappa\)</span> is the decay constant. When a study reported data for the same species at different sub-sites <span class="citation" data-cites="Fernandez2020">(<a href="#ref-Fernandez2020" role="doc-biblioref">Fernandez, See, and Kennedy 2020</a>)</span> or different seasons <span class="citation" data-cites="Entry1991">(<a href="#ref-Entry1991" role="doc-biblioref">Entry, Rose, and Cromack 1991</a>)</span>, we estimated or recorded the decay constant (<span class="math inline">\(\kappa\)</span>) and its associated standard error (SE) or standard deviation (SD) separately. In addition to this, we recorded the plant species, mycorrhizal types, forest ecosystem types, litter types, and geographic coordinates of study sites. Further, we also noted technical information about methodologies, including the number of study experiments, duration of the investigation, exclusion method for mycorrhizal fungi, and the characteristics of litter and litter bags.</p>
</section>
<section id="phylogenetic-data" class="level3">
<h3 class="anchored" data-anchor-id="phylogenetic-data">Phylogenetic data</h3>
<p>To test the influence of phylogenetic relatedness, we used the recently published GBOTB backbone of phylogeny for seed plants, which is based on data from <a href="https://www.ncbi.nlm.nih.gov/genbank/">GenBank</a> and <a href="https://tree.opentreeoflife.org/opentree/argus/opentree13.4@ott93302">Open Tree of Life</a> <span class="citation" data-cites="Smith2018">(<a href="#ref-Smith2018" role="doc-biblioref">Smith and Brown 2018</a>)</span>. It is considered as the largest dated phylogeny for 79,881 taxa of seed plants resolved at species level. We used the package <code>V.PhyloMaker2</code> (version 0.1.0) to prepare a phylogenetic tree for the selected plant species <span class="citation" data-cites="R-VPhyloMaker2">(<a href="#ref-R-VPhyloMaker2" role="doc-biblioref">Jin and Qian 2022</a>)</span>. The phylogeny was generated from the <code>GBOTB.extended.LCVP</code> mega-tree, which is a combination of two published phylogenies for plants <span class="citation" data-cites="Smith2018 Zanne2014">(<a href="#ref-Smith2018" role="doc-biblioref">Smith and Brown 2018</a>; <a href="#ref-Zanne2014" role="doc-biblioref">Zanne et al. 2014</a>)</span>. Plant names in this phylogenetic tree are standardised according to the Leipzig Catalogue of Vascular Plants (LCVP) <span class="citation" data-cites="R-lcvplants">(<a href="#ref-R-lcvplants" role="doc-biblioref">Freiberg et al. 2020</a>)</span>. The nodes were built using the <code>build.nodes.1.LCVP</code> function, which returns the most recent common ancestor of all the tips in the largest cluster of the genus, and define it as the basal node of the genus. Further, the phylogeny was generated under the <em>Scenario 3</em> for binding of new genus tip to the family branch as defined in earlier study <span class="citation" data-cites="Qian2016">(<a href="#ref-Qian2016" role="doc-biblioref">Qian and Jin 2016</a>)</span>.</p>
</section>
<section id="effect-size-and-variance" class="level3">
<h3 class="anchored" data-anchor-id="effect-size-and-variance">Effect size and variance</h3>
<p>Various effect sizes are used to summarize the magnitude and direction of the relationship between two variables <span class="citation" data-cites="Cooper2019 Harrer2021">(<a href="#ref-Cooper2019" role="doc-biblioref">Cooper, Hedges, and Valentine 2019</a>; <a href="#ref-Harrer2021" role="doc-biblioref">Harrer et al. 2021</a>)</span>. Among available effect sizes, response ratios (also called as Ratio of Means) and standardised mean differences have been frequently used to compare the means between two groups (i.e., experimental and control) in ecology and evolution <span class="citation" data-cites="Nakagawa2012 Koricheva2014">(<a href="#ref-Nakagawa2012" role="doc-biblioref">Nakagawa and Santos 2012</a>; <a href="#ref-Koricheva2014" role="doc-biblioref">Koricheva and Gurevitch 2014</a>)</span>. The response ratio or ratio of means is recommended when the mean of control group is not very small as compared to experimental group and both groups have estimates of same sign either positive or negative <span class="citation" data-cites="Hedges1999 Lajeunesse2015">(<a href="#ref-Hedges1999" role="doc-biblioref">Hedges, Gurevitch, and Curtis 1999</a>; <a href="#ref-Lajeunesse2015" role="doc-biblioref">Lajeunesse 2015</a>)</span>. The log transformation of response ratio (<span class="math inline">\(\text{lnRR}\)</span>) makes the metric equally sensitive to changes in both numerator and denominator, apart from making the metric nearly normal. Thus, it measures the actual difference in responses scaled to mean in control groups. However, <span class="math inline">\(\text{lnRR}\)</span> is considered to be slightly biased towards null effects and usually estimate high heterogeneity than other effect size measures <span class="citation" data-cites="Lajeunesse2015">(<a href="#ref-Lajeunesse2015" role="doc-biblioref">Lajeunesse 2015</a>)</span>.</p>
<p>Since the rate of litter decomposition is influenced by time (i.e., greater weight loss during initial phase than the later phases), we decided to compare decay constant rather than the weight loss. The log transformed response ratio (<span class="math inline">\(\text{lnRR}\)</span>) and associated variance was calculated using the (<a href="#eq-lnrr">Equation&nbsp;2</a>).</p>
<p><span id="eq-lnrr"><span class="math display">\[
\text{lnRR} = \ln \left( \frac{\bar X_E}{\bar X_C} \right); \quad
\text{Var(lnRR)} = \frac {SD_E^2}{n_E \bar X_E^2} + \frac {SD_C^2}{n_C \bar X_C^2}
\tag{2}\]</span></span></p>
<p>where, <span class="math inline">\(\bar X_E\)</span> and <span class="math inline">\(\bar X_C\)</span> represents the absolute litter decay constant (<span class="math inline">\(\kappa\)</span>) for experimental (presence of mycorrhizal fungi) and control (absence of mycorrhizal fungi) groups, respectively. The large-sample approximation was used to compute the sampling variances (<a href="#eq-lnrr">Equation&nbsp;2</a>) associated with the effect size <span class="math inline">\(\text{lnRR}\)</span> <span class="citation" data-cites="Hedges1999">(<a href="#ref-Hedges1999" role="doc-biblioref">Hedges, Gurevitch, and Curtis 1999</a>)</span>. The accuracy of <span class="math inline">\(\text{lnRR}\)</span> was ensured with with Geary’s test (<a href="#eq-geary">Equation&nbsp;3</a>) for both control and experimental group <span class="citation" data-cites="Lajeunesse2015">(<a href="#ref-Lajeunesse2015" role="doc-biblioref">Lajeunesse 2015</a>)</span>.</p>
<p><span id="eq-geary"><span class="math display">\[
\frac{\bar X}{SD} \sqrt{N} \geq 3
\tag{3}\]</span></span></p>
<p>A positive value of response ratio (<span class="math inline">\(\text{lnRR} &gt; 0\)</span>) indicates faster whereas a negative value (<span class="math inline">\(\text{lnRR} &lt; 0\)</span>) means slower decomposition rates in presence of mycorrhizal fungi, and a value close to zero (<span class="math inline">\(\text{lnRR} \approx 0\)</span>) indicates little or no effect. Both effect size and associated sampling variances were calculated using the function <code>escalc()</code> from the <code>metafor</code> package <span class="citation" data-cites="R-metafor">(<a href="#ref-R-metafor" role="doc-biblioref">Viechtbauer 2010</a>)</span>.</p>
</section>
<section id="meta-analytic-model" class="level3">
<h3 class="anchored" data-anchor-id="meta-analytic-model">Meta-analytic model</h3>
<p>The overall aim of meta-analyses is to combine the effects or observations from different studies. There are three main types of meta-analytic models i.e., fixed-effect model (FEM), random-effects model (REM), and multilevel model (MLM), which are commonly used to compute the overall effect estimate in meta-analysis. The most basic fixed-effect or common-effect model assumes a common overall mean for all studies. Since it assumes homogeneity between studies, it is far from the reality, at least for ecological studies. In contrary to the fixed effect model, the assumptions of random-effects model are more realistic as it considers between-study heterogeneity. This model assumes that there is not only one actual effect size but a distribution of true effect sizes <span class="citation" data-cites="Borenstein2010 Harrer2021">(<a href="#ref-Borenstein2010" role="doc-biblioref">Borenstein et al. 2010</a>; <a href="#ref-Harrer2021" role="doc-biblioref">Harrer et al. 2021</a>)</span>. According to this model, the observed effect size <span class="math inline">\(y_i\)</span> with variance <span class="math inline">\(v_i\)</span> deviates from the mean true effect size <span class="math inline">\(\mu\)</span> for a single study <span class="math inline">\(i\)</span> (with <span class="math inline">\(i = 1, \ldots , N_{studies}\)</span>) by two error terms <span class="math inline">\(u_i\)</span> (between-study heterogeneity) and <span class="math inline">\(e_i\)</span> (sampling error). Thus, the random effects model can be expressed as (<a href="#eq-rem">Equation&nbsp;4</a>)</p>
<p><span id="eq-rem"><span class="math display">\[
\begin{align}
y_i &amp;= \mu + u_i + e_i\\
\textbf{u} &amp;\sim \mathcal{N}(0, \sigma_u^2 \textbf{I}_u) \\
\textbf{e} &amp;\sim \mathcal{N}(0, \textbf{V})
\end{align}
\tag{4}\]</span></span></p>
<p>where, <span class="math inline">\(u_i\)</span> is the random effect corresponding to the <span class="math inline">\(i^{th}\)</span> study, <span class="math inline">\(\textbf{u}\)</span> is a <span class="math inline">\(1 \times N_{studies}\)</span> column vector with the <span class="math inline">\(u_i\)</span> values (which are assumed to be normally distributed with mean <span class="math inline">\(0\)</span> and variance <span class="math inline">\(\sigma_u^2\)</span>) and <span class="math inline">\(\textbf{I}_u\)</span> is an <span class="math inline">\(N_{studies} \times N_{studies}\)</span> identity matrix, <span class="math inline">\(\textbf{e}\)</span> is a <span class="math inline">\(1 \times N_{studies}\)</span> column vector with the <span class="math inline">\(e_{i}\)</span> values (which are assumed to be normally distributed with mean <span class="math inline">\(0\)</span> and variance <span class="math inline">\(v_i\)</span>), <span class="math inline">\(0\)</span> is a column vector of zeros and <span class="math inline">\(\textbf{V}\)</span> is an <span class="math inline">\(N_{studies} \times N_{studies}\)</span> matrix with the <span class="math inline">\(v_i\)</span> values along the diagonal.</p>
<p>Although the random-effects model is commonly used in ecology, it did not take into consideration the non-independence among effect sizes <span class="citation" data-cites="Nakagawa2012">(<a href="#ref-Nakagawa2012" role="doc-biblioref">Nakagawa and Santos 2012</a>)</span>. The non-independence can arise when the effect sizes are shared by studies (i.e., multiple effect sizes from the same study). Further, phylogenetic relatedness among species can also introduce non-independence among the effect size estimates <span class="citation" data-cites="Cinar2021">(<a href="#ref-Cinar2021" role="doc-biblioref">Cinar, Nakagawa, and Viechtbauer 2021</a>)</span>. This issue of non-independence can be addressed by using a multilevel (hierarchical) meta-analytic model, which considers a random effect at each level of variability in effect sizes. Among the several factors for non-independence of effect sizes, we specifically assessed the dependence due to studies, species and phylogenetic relatedness among species as described recently <span class="citation" data-cites="Cinar2021">(<a href="#ref-Cinar2021" role="doc-biblioref">Cinar, Nakagawa, and Viechtbauer 2021</a>)</span>. We used a complex multilevel meta-analytic model mathematically expressed in (<a href="#eq-mlm">Equation&nbsp;5</a>)</p>
<p><span id="eq-mlm"><span class="math display">\[
\begin{align}
y_{ijk} &amp;= \mu + u_{ij} + s_i + n_k + p_k + e_{ij} \\
\textbf{u} &amp;\sim \mathcal{N}(0, \sigma_u^2 \textbf{I}_u) \\
\textbf{s} &amp;\sim \mathcal{N}(0, \sigma_s^2 \textbf{I}_s) \\
\textbf{n} &amp;\sim \mathcal{N}(0, \sigma_n^2 \textbf{I}_n) \\
\textbf{p} &amp;\sim \mathcal{N}(0, \sigma_p^2 \textbf{A}) \\
\textbf{e} &amp;\sim \mathcal{N}(0, \textbf{V})
\end{align}
\tag{5}\]</span></span></p>
<p>where</p>
<ul>
<li><span class="math inline">\(y_{ijk}\)</span> is the <span class="math inline">\(j^{th}\)</span> effect (with <span class="math inline">\(j = 1, \ldots, N_i\)</span>, where <span class="math inline">\(N_i\)</span> is the number of effect sizes reported in the <span class="math inline">\(i^{th}\)</span> study) in the <span class="math inline">\(i^{th}\)</span> study (with <span class="math inline">\(i = 1, \ldots, N_{studies}\)</span>) for the <span class="math inline">\(k^{th}\)</span> species (with <span class="math inline">\(k = 1, \ldots, N_{species}\)</span>), and <span class="math inline">\(v_{ijk}\)</span> are corresponding sampling variances,</li>
<li><span class="math inline">\(\mu\)</span> is the overall meta-analytic mean,</li>
<li><span class="math inline">\(u_{ij}\)</span> is a random effect corresponding to the <span class="math inline">\(j^{th}\)</span> effect size in the <span class="math inline">\(i^{th}\)</span> study,</li>
<li><span class="math inline">\(s_i\)</span> is a study-specific random effect for <span class="math inline">\(i^{th}\)</span> study,</li>
<li><span class="math inline">\(n_k\)</span> is a species-specific random effect for <span class="math inline">\(k^{th}\)</span> species,</li>
<li><span class="math inline">\(p_k\)</span> denotes the phylogenetic random effect for the <span class="math inline">\(k^{th}\)</span> species,</li>
<li><span class="math inline">\(e_{ij}\)</span> is the sampling error or residual corresponding to the <span class="math inline">\(j^{th}\)</span> effect size in the <span class="math inline">\(i^{th}\)</span> study,</li>
<li><span class="math inline">\(\textbf{u}\)</span> is a <span class="math inline">\(1 \times N_{total}\)</span> column vector with the <span class="math inline">\(u_{ij}\)</span> values (which are assumed to be normally distributed with mean <span class="math inline">\(0\)</span> and within-study variance <span class="math inline">\(\sigma_u^2\)</span>), <span class="math inline">\(N_{total} = \sum_{i = 1}^{N_{studies}} N_i\)</span> represents the total number of the effect sizes, <span class="math inline">\(\textbf{I}_u\)</span> is an <span class="math inline">\(N_{total} \times N_{total}\)</span> identity matrix,</li>
<li><span class="math inline">\(\textbf{s}\)</span> is a <span class="math inline">\(1 \times N_{studies}\)</span> column vector with the <span class="math inline">\(s_i\)</span> values (which are assumed to be normally distributed with mean <span class="math inline">\(0\)</span> and between-study variance <span class="math inline">\(\sigma_s^2\)</span>), <span class="math inline">\(\textbf{I}_s\)</span> is an <span class="math inline">\(N_{studies} \times N_{studies}\)</span> identity matrix,</li>
<li><span class="math inline">\(\textbf{n}\)</span> is a <span class="math inline">\(1 \times N_{species}\)</span> column vector with the <span class="math inline">\(n_k\)</span> values (which are assumed to be normally distributed with mean <span class="math inline">\(0\)</span> and between-species variance <span class="math inline">\(\sigma_n^2\)</span>) and <span class="math inline">\(\textbf{I}_n\)</span> has dimensions <span class="math inline">\(N_{species} × N_{species}\)</span>,</li>
<li><span class="math inline">\(\textbf{p}\)</span> is a <span class="math inline">\(1 \times N_{species}\)</span> column vector with the <span class="math inline">\(p_k\)</span> values (which are assumed to follow a multivariate normal distribution with mean <span class="math inline">\(0\)</span> and variance–covariance matrix <span class="math inline">\(\sigma_p^2 \textbf{A}\)</span>, where <span class="math inline">\(\sigma_p^2\)</span> denotes between-species variance due to the phylogeny, and <span class="math inline">\(\textbf{A}\)</span> is the <span class="math inline">\(N_{species} \times N_{species}\)</span> phylogenetic correlation matrix), and</li>
<li><span class="math inline">\(\textbf{e}\)</span> is a <span class="math inline">\(1 \times N_{total}\)</span> column vector with the <span class="math inline">\(e_{ij}\)</span> values and <span class="math inline">\(\textbf{V}\)</span> is the corresponding (diagonal) variance–covariance matrix with dimensions <span class="math inline">\(N_{total} \times N_{total}\)</span>.</li>
</ul>
<p>The random-effects model was fitted using the <code>rma()</code> function whereas the complex multilevel meta-analytic model was fitted using the <code>rma.mv()</code> function from the <code>metafor</code> package <span class="citation" data-cites="R-metafor">(<a href="#ref-R-metafor" role="doc-biblioref">Viechtbauer 2010</a>)</span>. The amount of heterogeneity under the both models were estimated using the less biased restricted maximum likelihood (REML) estimator <span class="citation" data-cites="Viechtbauer2005">(<a href="#ref-Viechtbauer2005" role="doc-biblioref">Viechtbauer 2005</a>)</span>. The phylogenetic correlation matrix, denoted as <span class="math inline">\(\textbf{A}\)</span> in (<a href="#eq-mlm">Equation&nbsp;5</a>), was computed under the Brownian model using the function <code>vcv()</code> from the <code>ape</code> package version 5.7.1 <span class="citation" data-cites="R-ape">(<a href="#ref-R-ape" role="doc-biblioref">Paradis and Schliep 2019</a>)</span>. The branch lengths for each species was computed with the Grafen’s method <span class="citation" data-cites="Grafen1989">(<a href="#ref-Grafen1989" role="doc-biblioref">Grafen 1989</a>)</span> using the function <code>compute.brlen()</code> from the same package <span class="citation" data-cites="R-ape">(<a href="#ref-R-ape" role="doc-biblioref">Paradis and Schliep 2019</a>)</span>.</p>
</section>
<section id="heterogeneity" class="level3">
<h3 class="anchored" data-anchor-id="heterogeneity">Heterogeneity</h3>
<p>The heterogeneity reflects the variations in true effect sizes, which is not accounted by the sampling error variance within the meta-analysis. Cochran’s <span class="math inline">\(Q\)</span> is traditionally used in meta-analysis to assess the heterogeneity in meta-analysis <span class="citation" data-cites="Cochran1954">(<a href="#ref-Cochran1954" role="doc-biblioref">Cochran 1954</a>)</span>. It is the inverse variance weighted sum of squares, which can be mathematically represented by (<a href="#eq-cochran">Equation&nbsp;6</a>)</p>
<p><span id="eq-cochran"><span class="math display">\[
Q = \sum_{i=1}^{N_{studies}} w_i (y_i - \mu)^2;
\quad w_i = \frac{1}{v_i}
\tag{6}\]</span></span></p>
<p>where <span class="math inline">\(y_i\)</span> is the observed effect for each study <span class="math inline">\(i^{th}\)</span>; <span class="math inline">\(\mu\)</span> is the overall summary effect, <span class="math inline">\(w_i\)</span> is the weight defined as inverse of variance (<span class="math inline">\(1/v_i\)</span>) for <span class="math inline">\(i^{th}\)</span> study.</p>
<p>Although Cochran’s <span class="math inline">\(Q\)</span> has been widely used to test heterogeneity in meta-analyses <span class="citation" data-cites="Nakagawa2012 Koricheva2014">(<a href="#ref-Nakagawa2012" role="doc-biblioref">Nakagawa and Santos 2012</a>; <a href="#ref-Koricheva2014" role="doc-biblioref">Koricheva and Gurevitch 2014</a>)</span>, it is sensitive to size and precision of studies included meta-analysis <span class="citation" data-cites="Borenstein2009 Harrer2021">(<a href="#ref-Borenstein2009" role="doc-biblioref">Borenstein et al. 2009</a>; <a href="#ref-Harrer2021" role="doc-biblioref">Harrer et al. 2021</a>)</span>. Higgins and Thompson’s <span class="math inline">\(I^2\)</span> statistic presents the more standardized form to estimate the between-study heterogeneity. It is defined as the percentage of variability in the effect size that is not caused by sampling error <span class="citation" data-cites="Higgins2002">(<a href="#ref-Higgins2002" role="doc-biblioref">Higgins and Thompson 2002</a>)</span>. Similarly, the <span class="math inline">\(H^2\)</span> statistic is used to describe the variance in observed effect sizes due to sampling error <span class="citation" data-cites="Higgins2002">(<a href="#ref-Higgins2002" role="doc-biblioref">Higgins and Thompson 2002</a>)</span>. It is estimated as the ratio of observed variation to the expected variance due to sampling error. Although both <span class="math inline">\(I^2\)</span> and <span class="math inline">\(H^2\)</span> are measured based on the Cochran’s <span class="math inline">\(Q\)</span>, the Higgins and Thompson’s <span class="math inline">\(I^2\)</span> and <span class="math inline">\(H^2\)</span> were calculated based on more general definitions using the (<a href="#eq-i2rem">Equation&nbsp;7</a>).</p>
<p><span id="eq-i2rem"><span class="math display">\[
I^2 = \frac{\sigma_u^2}{\sigma_u^2 + \sigma_e^2} \times 100; \quad
H^2 = \frac{\sigma_u^2 + \sigma_e^2}{\sigma_e^2}; \quad
\sigma_e^2 =\frac{(N_{studies}-1)\sum w_i}{(\sum w_i)^2 - \sum w_i^2}
\tag{7}\]</span></span></p>
<p>where <span class="math inline">\(\sigma_u^2\)</span> is the between-study variance (also referred as <span class="math inline">\(\tau^2\)</span>), <span class="math inline">\(\sigma_e^2\)</span> is the typical within-study variance, <span class="math inline">\(N_{studies}\)</span> is the total number of studies, <span class="math inline">\(w_i\)</span> is the weight defined as inverse of variance (<span class="math inline">\(1/v_i\)</span>) for <span class="math inline">\(i^{th}\)</span> study. We also included a prediction interval for the true overall summary effect (<span class="math inline">\(\mu\)</span>) because high heterogeneity is expected for ecological studies <span class="citation" data-cites="Senior2016">(<a href="#ref-Senior2016" role="doc-biblioref">Senior et al. 2016</a>)</span>.</p>
<p>Since <span class="math inline">\(Q\)</span> and <span class="math inline">\(I^2\)</span> are not designed to assess between-study heterogeneity for multilevel models <span class="citation" data-cites="Nakagawa2012">(<a href="#ref-Nakagawa2012" role="doc-biblioref">Nakagawa and Santos 2012</a>)</span>, we computed heterogeneity <span class="math inline">\(I^2\)</span> for our multilevel model as suggested earlier <span class="citation" data-cites="Nakagawa2012">(<a href="#ref-Nakagawa2012" role="doc-biblioref">Nakagawa and Santos 2012</a>)</span>. Following the previous mathematical notations, the within-study heterogeneity <span class="math inline">\(I_u^2\)</span> can be represented as (<a href="#eq-i2mlm">Equation&nbsp;8</a>) whereas <span class="math inline">\(H^2\)</span> or phylogenetic heritability can be represented by (<a href="#eq-h2mlm">Equation&nbsp;9</a>). The values of <span class="math inline">\(H^2\)</span> can range from 0 to 1, which corresponds to no phylogenetic relatedness to exact proportional to phylogenetic relatedness among effect sizes.</p>
<p><span id="eq-i2mlm"><span class="math display">\[
I_u^2 = \frac {\sigma_u^2} {\sigma_u^2 + \sigma_s^2 + \sigma_n^2 + \sigma_p^2 + \sigma_e^2}
\tag{8}\]</span></span></p>
<p><span id="eq-h2mlm"><span class="math display">\[
H^2 = \frac{\sigma_p^2}{\sigma_u^2 + \sigma_s^2 + \sigma_n^2 + \sigma_p^2}
\tag{9}\]</span></span></p>
<p><strong>Meta-regression:</strong> The heterogeneity in the observed outcomes can be explained using a set of predictors or moderators in meta-analysis. Such effect size partitioning among various variables is sometimes known as subgroup analysis. In absence of potentially better predictors, we explored species identity and clade (angiosperm or gymnosperm) as predictors since they can act as a surrogate for species level traits including the litter quality. We fitted mixed-effects models to assess the clade-specific or species-specific variations in the effects.</p>
<p><strong>Outlier and influential analysis:</strong> The overall pooled effect size can be heavily influenced by one or more outcomes (effect size), so that our estimated effect size is not robust. We assessed the outliers and influential cases using the studentised residuals and Cook’s distances for our random effects model <span class="citation" data-cites="Viechtbauer2010">(<a href="#ref-Viechtbauer2010" role="doc-biblioref">Viechtbauer and Cheung 2010</a>)</span>. Outcomes with a studentised residual larger than the <span class="math inline">\(100 \times (1 − 0.05 / (2 \times N_{total}))^{th}\)</span> percentile of a standard normal distribution are considered potential outliers (i.e., using a Bonferroni correction with two-sided <span class="math inline">\(\alpha = 0.05\)</span> for <span class="math inline">\(N_{total}\)</span> outcomes included in the meta-analysis). Similarly, outcomes with a Cook’s distance larger than the median plus six times the interquartile range of the Cook’s distances are considered to be influential.</p>
</section>
<section id="publication-bias" class="level3">
<h3 class="anchored" data-anchor-id="publication-bias">Publication bias</h3>
<p>The publication bias in the average effect size was assessed using the classical funnel plot. It is a scatter plot of the observed effect size for each study on the <em>x</em>-axis against a measure of the standard error on the <em>y</em>-axis of each study. In the absence of any publication bias, effect size distribution should roughly follow the funnel shape in the plot. We conducted the rank correlation test <span class="citation" data-cites="Begg1994">(<a href="#ref-Begg1994" role="doc-biblioref">Begg and Mazumdar 1994</a>)</span> and the regression test <span class="citation" data-cites="Sterne2006">(<a href="#ref-Sterne2006" role="doc-biblioref">Sterne and Egger 2006</a>)</span> to assess the asymmetry in the funnel plot. The regression was carried using the standard error of the observed outcomes as predictor.</p>
<p>Further, we computed the fail-safe numbers to assess the robustness of estimated average effect. Fail-safe numbers estimate the number of non-significant and or unpublished studies which would reduce the significance of overall average estimate if included in the meta-analysis. We used the unweighted Rosenthal’s <span class="citation" data-cites="Rosenthal1979">(<a href="#ref-Rosenthal1979" role="doc-biblioref">Rosenthal 1979</a>)</span> and weighted Rosenberg’s <span class="citation" data-cites="Rosenberg2005">(<a href="#ref-Rosenberg2005" role="doc-biblioref">Rosenberg 2005</a>)</span> approaches to estimate the fail-safe numbers.</p>
<p>All analyses were performed in <code>R</code> programming language and statistical environment version 4.3.0 <span class="citation" data-cites="R-base">(<a href="#ref-R-base" role="doc-biblioref">R Core Team 2023</a>)</span>. The <code>R</code> package <code>metafor</code> version 4.2.0 <span class="citation" data-cites="R-metafor">(<a href="#ref-R-metafor" role="doc-biblioref">Viechtbauer 2010</a>)</span> for statistical meta-analysis, package <code>ggtree</code> version 3.7.2 used for visualisation of phylogenetic tree <span class="citation" data-cites="R-ggtree">(<a href="#ref-R-ggtree" role="doc-biblioref">Yu et al. 2016</a>)</span> and package <code>tidyverse</code> version 2.0.0 <span class="citation" data-cites="R-tidyverse">(<a href="#ref-R-tidyverse" role="doc-biblioref">Wickham et al. 2019</a>)</span> for general data manipulation and visualization.</p>
</section>
</section>
<section id="results" class="level2">
<h2 class="anchored" data-anchor-id="results">Results</h2>
<section id="overview-of-dataset" class="level3">
<h3 class="anchored" data-anchor-id="overview-of-dataset">Overview of dataset</h3>
<p>Our systematic literature search and screening found five eligible studies for the present analysis (<a href="#fig-fig1">Figure&nbsp;1</a>). Studies in our dataset differed in numerous ways, including experimental duration, sampling frequency, initial litter mass, litter species, mycorrhizal type, and forest type (<a href="#tbl-tab1">Table&nbsp;1</a>).</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="tbl-tab1" class="anchored">
<table class="table table-sm table-striped small">
<caption>Table&nbsp;1: Overview of selected studies included in the meta-analysis</caption>
<colgroup>
<col style="width: 12%">
<col style="width: 10%">
<col style="width: 6%">
<col style="width: 16%">
<col style="width: 16%">
<col style="width: 12%">
<col style="width: 14%">
<col style="width: 10%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Study</th>
<th style="text-align: right;">Experiments</th>
<th style="text-align: right;">Species</th>
<th style="text-align: right;">Sampling frequency</th>
<th style="text-align: right;">Initial weight (g)</th>
<th style="text-align: right;">Mesh size (mm)</th>
<th style="text-align: left;">Mycorrhizal type</th>
<th style="text-align: left;">Forest type</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;"><span class="citation" data-cites="Entry1991">Entry, Rose, and Cromack (<a href="#ref-Entry1991" role="doc-biblioref">1991</a>)</span></td>
<td style="text-align: right;">4</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">5.00</td>
<td style="text-align: right;">1</td>
<td style="text-align: left;">EMF</td>
<td style="text-align: left;">Temperate</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="citation" data-cites="Zhu1996">Zhu and Ehrenfeld (<a href="#ref-Zhu1996" role="doc-biblioref">1996</a>)</span></td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">4</td>
<td style="text-align: right;">1.63</td>
<td style="text-align: right;">1</td>
<td style="text-align: left;">EMF</td>
<td style="text-align: left;">Temperate</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="citation" data-cites="Mayor2006">Mayor and Henkel (<a href="#ref-Mayor2006" role="doc-biblioref">2006</a>)</span></td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">3</td>
<td style="text-align: right;">17.00</td>
<td style="text-align: right;">1</td>
<td style="text-align: left;">EMF</td>
<td style="text-align: left;">Tropical</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="citation" data-cites="Fernandez2020">Fernandez, See, and Kennedy (<a href="#ref-Fernandez2020" role="doc-biblioref">2020</a>)</span></td>
<td style="text-align: right;">4</td>
<td style="text-align: right;">2</td>
<td style="text-align: right;">3</td>
<td style="text-align: right;">2.00</td>
<td style="text-align: right;">2</td>
<td style="text-align: left;">EMF &amp; AMF</td>
<td style="text-align: left;">Temperate</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="citation" data-cites="Lang2021">Lang et al. (<a href="#ref-Lang2021" role="doc-biblioref">2021</a>)</span></td>
<td style="text-align: right;">4</td>
<td style="text-align: right;">4</td>
<td style="text-align: right;">3</td>
<td style="text-align: right;">1.00</td>
<td style="text-align: right;">2</td>
<td style="text-align: left;">EMF</td>
<td style="text-align: left;">Temperate</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<p>These studies conducted total 14 experiments for nine different species. The leaf litter of nine plant species (<em>Acer saccharum</em> Marshall, <em>Betula alleghaniensis</em> Britton, <em>Dicymbe corymbosa</em> Benth., <em>Fagus grandifolia</em> Ehrh., <em>Fraxinus americana</em> L., <em>Pinus rigida</em> Mill., <em>Pinus strobus</em> L., <em>Pseudotsuga menziesii</em> (Mirbel) Franco <em>Quercus ellipsoidalis</em> E. J. Hill) was used during the experiments (<a href="#fig-fig2">Figure&nbsp;2</a> a). The duration of experiments is also differed among studies and ranged from one year to 2.4 years. The initial leaf litter mass varied from 1.0 g to 17.0 g, and three studies used up to 2.0 g of leaf litter, whereas two studies used more than 2.0 g of leaf litter <span class="citation" data-cites="Entry1991 Mayor2006">(<a href="#ref-Entry1991" role="doc-biblioref">Entry, Rose, and Cromack 1991</a>; <a href="#ref-Mayor2006" role="doc-biblioref">Mayor and Henkel 2006</a>)</span>. Further, most of the studies involved ectomycorrhizal fungi <span class="citation" data-cites="Entry1991 Zhu1996 Mayor2006 Fernandez2020">(<a href="#ref-Entry1991" role="doc-biblioref">Entry, Rose, and Cromack 1991</a>; <a href="#ref-Zhu1996" role="doc-biblioref">Zhu and Ehrenfeld 1996</a>; <a href="#ref-Mayor2006" role="doc-biblioref">Mayor and Henkel 2006</a>; <a href="#ref-Fernandez2020" role="doc-biblioref">Fernandez, See, and Kennedy 2020</a>)</span>, while one study showed the involvement of both ectomycorrhizal and arbuscular mycorrhizal fungi <span class="citation" data-cites="Lang2021">(<a href="#ref-Lang2021" role="doc-biblioref">Lang et al. 2021</a>)</span>. All the studies assured mycorrhizal exclusion by trenching to 30 cm soil depth.</p>
<p>Further, our dataset revealed that most studies were conducted in the temperate deciduous forest of North America (n = 4), and only one study was conducted in the tropical forest of Guyana of South America’s North Atlantic Coast (<a href="#fig-fig2">Figure&nbsp;2</a> b). The earliest study included was from 1991 <span class="citation" data-cites="Entry1991">(<a href="#ref-Entry1991" role="doc-biblioref">Entry, Rose, and Cromack 1991</a>)</span>, whereas the latest study was published in 2021 <span class="citation" data-cites="Lang2021">(<a href="#ref-Lang2021" role="doc-biblioref">Lang et al. 2021</a>)</span>.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-fig2" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="index_files/figure-html/fig-fig2-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption class="figure-caption">Figure&nbsp;2: (a) Phylogenetic relationship among the species used for litter decomposition. The symbols represents the studies to which a species belong. (b) Geographical distribution of studies included in our dataset</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="overall-effect-estimate" class="level3">
<h3 class="anchored" data-anchor-id="overall-effect-estimate">Overall effect estimate</h3>
<p>A total of 14 estimates of litter decomposition (<span class="math inline">\(\kappa\)</span>) were included in the meta-analysis (<a href="#fig-fig3">Figure&nbsp;3</a>). The observed response ratios (<span class="math inline">\(\text{RR}\)</span>) ranged from 0.74 to 1.58, with the majority of estimates being positive (71%). Thus, the decomposition rate is also expected to be higher in the presence of mycorrhizal fungi on average (<a href="#fig-fig3">Figure&nbsp;3</a>). Further, a strong positive relationship was observed between annual decay constants for the presence and absence of mycorrhizal fungi (<em>R<sup>2</sup></em> = 0.94, <em>p</em> &lt; 0.001), indicating that our dataset does not include extreme effect estimates.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-fig3" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="index_files/figure-html/fig-fig3-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;3: Effects of mycorrhizal fungi on annual litter decay constant (<span class="math inline">\(\kappa\)</span>). Each point represents a single comparison of <span class="math inline">\(\kappa\)</span>. The color of each point denotes the change in <span class="math inline">\(\kappa\)</span> due to the presence of mycorrhizal fungi. Values above and below the 1:1 line represents the stimulation and inhibition of litter decomposition rate in presence of mycorrhizal fungi, respectively. The values falling on the 1:1 line represent the neutral or no effect</figcaption>
</figure>
</div>
</div>
</div>
<p>Consistent with our expectations, both meta-analytic models also suggested an overall positive effect of mycorrhizal fungi on litter decomposition rates. The overall average effect estimate (<span class="math inline">\(\hat \mu\)</span>) was 1.07 (<em>95% CI</em>: 1.03, 1.11) for the random effects model whereas the same was 1.08 (<em>95% CI</em>: 1, 1.18) for the multilevel model. The wider confidence intervals are expected in the latter model as it included additional sources of variations (number of levels, i.e., id, study, species). In ecological sense, these models indicated that on average the annual decay constant for leaf litter was 7-8% higher in presence of mycorrhizal fungi (<a href="#fig-fig4">Figure&nbsp;4</a>). The random-effects model found strong statistical evidence (<em>Z</em> = 3.58, <em>p</em> &lt; 0.001) whereas the multilevel model showed moderate statistical evidence (<em>Z</em> = 1.96, <em>p</em> = 0.05) for the accelerated decomposition rates. This positive effect is also evident in the forest plot, where the right-side position of the diamond demonstrates that litter decomposition rate increased in the presence of mycorrhizal fungi (<a href="#fig-fig4">Figure&nbsp;4</a>).</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-fig4" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="index_files/figure-html/fig-fig4-1.png" class="img-fluid figure-img" width="768"></p>
<figcaption class="figure-caption">Figure&nbsp;4: Forest plot showing the observed effects of mycorrhizal fungi on litter decomposition rates for each experiment (n = 14). The size of each square is proportional to the weight of each study, and the associated bars represent the 95% confidence interval associated with individual effect size. The diamonds at the bottom represent the overall average effect estimated from meta-analytic models. The length of diamond denotes the 95% confidence intervals whereas associated dotted lines indicate the 95% prediction intervals</figcaption>
</figure>
</div>
</div>
</div>
<p>The Cochran’s <span class="math inline">\(Q\)</span> test for heterogeneity revealed significant variations (<span class="math inline">\(Q\)</span> = 259.45, <em>df</em> = 13, <em>p</em> &lt; 0.001) in true outcomes. Further, very high value of Higgins &amp; Thompson’s statistic (<span class="math inline">\(I^2\)</span> = 97.85%; <em>95% CI</em>: 97.77, 99.94) suggest that most of the observed variations captured the real differences in true effects (<span class="math inline">\(\tau^2\)</span> = 0.002). Similarly, the within-study variations in observed estimates were larger than the sampling error as indicated by <span class="math inline">\(H^2\)</span> statistic (<span class="math inline">\(H^2\)</span> = 46.43).</p>
<p>Consistent with random-effects model, the multilevel model also indicated very high heterogeneity (<span class="math inline">\(I_t^2\)</span> = 99.5%) in true outcomes. The main advantage of multilevel model is that it allows us to estimate the heterogeneity in true outcomes at different levels of meta-analytic model. This model suggested that the heterogeneity due to species-specific effects was very large (<span class="math inline">\(\sigma_n^2\)</span> = 0.007; <span class="math inline">\(I_n^2\)</span> = 84.32%) followed by the within-study heterogeneity (<span class="math inline">\(\sigma_u^2\)</span> = 0.001; <span class="math inline">\(I_u^2\)</span> = 15.22%). Interestingly, neither between-study heterogeneity (<span class="math inline">\(\sigma_s^2\)</span> = 0; <span class="math inline">\(I_s^2\)</span> = 0) nor phylogeny-specific heterogeneity (<span class="math inline">\(\sigma_p^2\)</span> = 0; <span class="math inline">\(I_p^2\)</span> = 0) was detected by our multilevel model. Since no phylogenetic effects were observed, the phylogenetic heritability (<span class="math inline">\(H^2\)</span> = 0) was also absent.</p>
<p>Given the high heterogeneity in true outcomes, the 95% prediction interval ranged from 0.97 to 1.17 as computed by the random effects model. However, the multilevel model suggested a much wider 95% prediction interval ranging from as low as 0.89 to as high as 1.33. Since the predicted values of average estimate included values both lesser and greater than one, we can expect slower decomposition rates in some experiments, though the presence of mycorrhizal fungi enhanced the litter decomposition on average (<a href="#fig-fig4">Figure&nbsp;4</a>).</p>
<p>The meta-regression model with clade (Angiosperm vs Gymnosperm) as categorical moderator suggested that experiments using litter of gymnosperm species obtained on average smaller effect (<span class="math inline">\(\beta_1\)</span> = -0.03, <em>SE</em> = 0.05) as compared to angiosperms, though this difference was not statistically significant (<span class="math inline">\(Q_M\)</span> = 0.31, <em>df</em> = 1, <em>p</em> = 0.578). Thus, inclusion of clade as moderator variable did not explained any variation in true effects. While the mixed-effects model with species as moderator reflected the variations of effect estimates among species (<a href="#fig-fig5">Figure&nbsp;5</a>). This meta-regression model indicated that at least one species had significantly different (higher or lower) response ratio than the response ratio for the reference level (<em>Acer saccharum</em> in our case). Thus, this model provided moderate statistical evidence that some amount of variations in true effects are indeed due to species (<span class="math inline">\(Q_M\)</span> = 16.96, <em>df</em> = 8, <em>p</em> = 0.031). It appeared that about 29% of the total variations in the true outcomes were due to species. Although species explained some proportion of the total variations, there still exist variations in true effects that are not attributable to species. These unexplained variations are evident from the test for residual heterogeneity (<span class="math inline">\(Q\)</span> = 239.52, <em>df</em> = 5, <em>p</em> &lt; 0.001), which suggested large amount of unexplained variations in true effects. Further, this unexplained heterogeneity has large variance (<span class="math inline">\(\tau^2\)</span> = 0.001) and it reflects the real variations that are not explained by species as suggested by high value of <span class="math inline">\(I^2\)</span> = 98.8%.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-fig5" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="index_files/figure-html/fig-fig5-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;5: The average effect of mycorrhizal association on litter decomposition rate for each species included in the meta-analysis. The values represent the average difference in response ratios compared to the reference species (<em>Acer saccharum</em>) and the associated bars denote the 95% confidence intervals</figcaption>
</figure>
</div>
</div>
</div>
<p>Further, the examination of the studentised residuals revealed that none of the studies had a value larger than ± 2.91 and hence there was no indication of outliers in the context of this model. Similarly, the inspection of Cook’s distances indicated that none of the studies could be considered overly influential.</p>
</section>
<section id="publication-bias-1" class="level3">
<h3 class="anchored" data-anchor-id="publication-bias-1">Publication bias</h3>
<p>The visual inspection of the funnel plot did not indicate the publication bias as the outcomes appeared to be symmetrically distributed around the average effect estimate (<a href="#fig-fig6">Figure&nbsp;6</a>). The outcomes were well-fitted within the funnel as larger outcomes (with higher precision) were closely clustered at top whereas smaller outcomes (with lower precision) were dispersed around the average effect at bottom. Further, the funnel plot included nearly half outcomes in the non-significant region again suggesting absence of publication bias due to null findings of studies.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-fig6" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="index_files/figure-html/fig-fig6-1.png" class="img-fluid figure-img" width="768"></p>
<figcaption class="figure-caption">Figure&nbsp;6: The contour-enhanced funnel plot shows the effect size of each study (expressed as the response ratio) on the x-axis against a measure of their standard error (from large to small) on the y-axis. The vertical lines in the middle of the funnel show the average effect estimate and associated 95% confidence interval. The shaded region indicates the different significance levels for the estimated effect size for each outcome</figcaption>
</figure>
</div>
</div>
</div>
<p>The intercept estimated from the regression test (<span class="math inline">\(\beta_0\)</span> = 0.06; <em>95% CI</em>: 0.016, 0.104) was not significantly higher than zero (<em>Z</em> = 0.51, <em>p</em> = 0.609). Also, there was non-significant low rank correlation (Kendall’s <span class="math inline">\(\tau\)</span> = -0.27, <em>p</em> = 0.193) between standard error and response ratio of outcomes. Thus, both the statistical tests for funnel asymmetry also corroborated our initial visual observations of funnel plot symmetry, which can be considered as indication of negligible or no evidence of publication bias for the present meta-analysis. (<a href="#fig-fig6">Figure&nbsp;6</a>).</p>
<p>The Rosenthal’s <em>fail-safe N</em> of 2107 suggested that over 2,000 outcomes with average response ratio of 1.0 need to be added to make the average effect non-significant (<em>p</em> = 0.05). Similarly, the Rosenberg’s <em>fail-safe N</em> was 4019, indicating that there would need to be over 4,000 outcomes with a mean response ratio of 1.0 added to the analysis, before the cumulative effect would become statistically non-significant (<em>p</em> = 0.05). Considering that we were able to identify only 14 relevant outcomes, it is unlikely that we missed over 4,000 or over 2,000 outcomes. Thus, although it may be possible that we might have estimated biased effect, it is unlikely that the actual overall effect is zero. However, we acknowledge that it is just one form of publication bias among the several other sources of bias in the estimate.</p>
</section>
</section>
<section id="discussion" class="level2">
<h2 class="anchored" data-anchor-id="discussion">Discussion</h2>
<p>Considering the initially posed questions, our analysis found moderate to strong evidence that mycorrhizal presence indeed affects the rate of litter decomposition in forest ecosystems (Question 1). The overall average estimate suggested that mycorrhizal fungi tends to enhance the litter decomposition rate by 7-8% on average (Question 2), though it can vary from a decrease of about 10% to an increase of over 30% in presence of mycorrhizal fungi. As with other ecological meta-analyses, large variations in effects were observed as indicated by potentially large heterogeneity. A small proportion (~30%) of these variations can be attributed to differences in species specific traits, possibly through litter quality (Question 3). However, no statistical evidence was found for the differences between angiosperm and gymnosperm species, though slightly higher enhancement was observed for angiosperms than the gymnosperms (Question 4). Further, these effects did not vary due to phylogenetic relationship among the included species (Question 3).</p>
<p>In contrast to expectations of Gadgil effect, our findings suggested an overall increase in the litter decomposition rates in presence of mycorrhizal fungi, which is consistent with previous findings <span class="citation" data-cites="Shah2016 Gui2017 Frey2019 Lang2021">(<a href="#ref-Shah2016" role="doc-biblioref">Shah et al. 2016</a>; <a href="#ref-Gui2017" role="doc-biblioref">Gui et al. 2017</a>; <a href="#ref-Frey2019" role="doc-biblioref">Frey 2019</a>; <a href="#ref-Lang2021" role="doc-biblioref">Lang et al. 2021</a>)</span>. The enhanced litter decomposition rates have been observed in the presence of both ectomycorrhizal <span class="citation" data-cites="Shah2016">(<a href="#ref-Shah2016" role="doc-biblioref">Shah et al. 2016</a>)</span> and arbuscular mycorrhizal fungi <span class="citation" data-cites="Gui2017">(<a href="#ref-Gui2017" role="doc-biblioref">Gui et al. 2017</a>)</span>. The ectomycorrhizal-mediated enhancement was suggested to be associated with the secretion of enzymes to acquire nutrients from organic matter <span class="citation" data-cites="Jackson2019">(<a href="#ref-Jackson2019" role="doc-biblioref">Jackson et al. 2019</a>)</span>. These enzymes degrade the complex compounds and make them available to other decomposer organisms. However, the increased litter decomposition in the presence of arbuscular mycorrhizal fungi was attributed to their saprotrophic capabilities, which tend to acquire nutrients directly from organic matter <span class="citation" data-cites="Hodge2001 Herman2012">(<a href="#ref-Hodge2001" role="doc-biblioref">Hodge, Campbell, and Fitter 2001</a>; <a href="#ref-Herman2012" role="doc-biblioref">Herman et al. 2012</a>)</span>. Further, both ectomycorrhizal <span class="citation" data-cites="Brzostek2015">(<a href="#ref-Brzostek2015" role="doc-biblioref">Brzostek et al. 2015</a>)</span> and arbuscular mycorrhizal fungi <span class="citation" data-cites="Herman2012">(<a href="#ref-Herman2012" role="doc-biblioref">Herman et al. 2012</a>)</span> are observed to increase the activity of microbial communities by providing plant-based carbon sources, i.e., priming effects. Since microbial communities are associated with the breakdown of complex compounds and the release of nutrients, their increased activity enhances the litter decomposition rate <span class="citation" data-cites="Schneider2012">(<a href="#ref-Schneider2012" role="doc-biblioref">Schneider et al. 2012</a>)</span>. Thus, our findings question the generality of ‘Gadgil effect’ and suggest that the magnitude and direction of ‘Gadgil effect’ can be highly inconsistent and variable <span class="citation" data-cites="Fernandez2016">(<a href="#ref-Fernandez2016" role="doc-biblioref">Fernandez and Kennedy 2016</a>)</span>.</p>
<p>The much wider range of average effect (-10% to 30%) in our analysis suggest that multiple variables are involved in deriving the mycorrhizal effects on litter decomposition. Such a wider range is expected because the rate of litter decomposition is not controlled by mycorrhizal effects alone and several factors including climate, litter quality and soil decomposer organisms are involved in determining the decomposition rates <span class="citation" data-cites="Zhang2008 Cornwell2008 Kumar2021">(<a href="#ref-Zhang2008" role="doc-biblioref">Zhang et al. 2008</a>; <a href="#ref-Cornwell2008" role="doc-biblioref">Cornwell et al. 2008</a>; <a href="#ref-Kumar2021" role="doc-biblioref">Kumar et al. 2021</a>)</span>. Thus, the mycorrhizal-mediated litter decomposition rates may either hamper or stimulate depending upon the stage of decomposition, soil profile, climatic condition, plant species, and fungal species <span class="citation" data-cites="Fernandez2016 Sterkenburg2018">(<a href="#ref-Fernandez2016" role="doc-biblioref">Fernandez and Kennedy 2016</a>; <a href="#ref-Sterkenburg2018" role="doc-biblioref">Sterkenburg et al. 2018</a>)</span>. The species-specific effects are observed in our analysis, as species identity appeared to explain a small part of total variations in the mycorrhizal effects. This species-specific variation in mycorrhizal effects can be attributed to the resource allocation strategy of plant species. When mycorrhizal fungi receive more carbon from the host plant, they can invest more in enzymes and secondary metabolites <span class="citation" data-cites="Rineau2013">(<a href="#ref-Rineau2013" role="doc-biblioref">Rineau et al. 2013</a>)</span>. This higher investment can directly increase the enzymatic degradation capacity of mycorrhizal fungi leading to faster decomposition rates. Further, high resource allocation can result in higher production of labile exudates leading to stimulation of growth and activity of saprotrophic microbes (priming effects).</p>
<p>Further, the species identity can indirectly affect mycorrhizal effects through the quality of litter produced by a species <span class="citation" data-cites="Cornwell2008 Patil2020srep">(<a href="#ref-Cornwell2008" role="doc-biblioref">Cornwell et al. 2008</a>; <a href="#ref-Patil2020srep" role="doc-biblioref">Patil, Kumar, Kumar, Cheema, et al. 2020</a>)</span>. It is observed that ‘high quality’ litter (low lignin and high nitrogen) decomposes more rapidly than the ‘low quality’ litter (high lignin, low nitrogen) <span class="citation" data-cites="Patil2020srep">(<a href="#ref-Patil2020srep" role="doc-biblioref">Patil, Kumar, Kumar, Cheema, et al. 2020</a>)</span>. Suppose a species will produce ‘high quality’ litter (low lignin and high nitrogen). In that case, mycorrhizal effects will be weaker due to more activities of saprotrophic fungi and decomposer micro-organisms whereas ‘low quality’ litter (high lignin, low nitrogen and high carbon content) will favour strong mycorrhizal effects as it will become accessible to the mycorrhizal fungi. Thus, it seems that various factors modulate the effects of mycorrhizal fungi on litter decomposition rate; therefore, these effects can be highly context-dependent <span class="citation" data-cites="Fernandez2016">(<a href="#ref-Fernandez2016" role="doc-biblioref">Fernandez and Kennedy 2016</a>)</span>.</p>
<p>Although our meta-analysis supports the mycorrhizal-mediated faster decomposition rates, it suffers from some limitations and caveats. We expect that these limitations and caveats will inspire future investigations to advance our understanding about the role of mycorrhizal fungi in nutrient cycling. One of the major limitation of the present analysis is that the systematic literature search and screening resulted in a small number of comparable studies. Although the practice of systematic literature search and screening substantially improve the reproducibility of overall analysis, it may hamper us to identify all the evidence from the literature. Further, our strict eligibility criteria to identify comparable studies appeared to be overly narrower. Therefore, we suggest future research synthesis to include all possible studies across wider range of plant species and ecosystem types. Further, litter decomposition in natural ecosystems is regulated by a complex interaction of several factors (e.g., climate, soil, plant, and mycorrhizal fungi). However, our limited database did not allowed us to test the effects of these variables due to insufficient sample size. Furthermore, tropical forests are under-represented in our analysis, it may be possible that observed positive effects of mycorrhizal fungi are localised to temperate forests <span class="citation" data-cites="Keller2019">(<a href="#ref-Keller2019" role="doc-biblioref">Keller and Phillips 2019</a>)</span>. Similarly, all the eligible studies considered only leaf litter for decomposition, the inclusion of litter from other sources might lead to variation in the results. Moreover, we have not considered the actual sample size, mean, and variation of measurements; instead, we considered the number of data points as the sample size to ensure the consistency among the studies. Finally, as with any systematic exercise, deviation from the systematic protocol (searching keywords, eligibility criteria) may lead to different results.</p>
<p>We encourage researchers to consider the known limitations of the present study. We observed that most studies considered leaf litter and litter from other sources (root, shoot, and wood) is lacking. Thus, there is a need to experiment with other litter along with leaf litter. Further, we suggest the future investigations to study the mycorrhizal effects on litter decomposition rates across biomes (grassland, heathland, temperate forest and tropical forest), plant species (angiosperm vs gymnosperm, exotic vs native species, leguminous vs non leguminous species, evergreen vs deciduous), litter type (fine roots vs leaf litter, high quality litter vs low quality litter), and mycorrhizal groups (arbuscular mycorrhizal fungi, ectomycorrhizal fungi, ericoid mycorrhizal fungi, orchid mycorrhizal fungi). Moreover, we need to investigate these effects along the gradients of climate, elevation and abundance of mycorrhizal fungi (i.e., trenching depth).</p>
</section>
<section id="conclusion" class="level2">
<h2 class="anchored" data-anchor-id="conclusion">Conclusion</h2>
<p>Our analysis provides valuable insights into the influence of mycorrhizal fungi on litter decomposition rates in forest ecosystems, challenging the generality of the widely recognised ‘Gadgil effect.’ The overall findings suggest a tendency for increased litter decomposition rates in the presence of mycorrhizal fungi. However, the considerable variability observed across species highlights the involvement of multiple underlying mechanisms. While our results shed light on the potential importance of mycorrhizal fungi in regulating litter decomposition, caution must be exercised regarding the generalisability and interpretability of these effects and processes. The complex nature of mycorrhizal associations and their interactions with other factors in ecosystems necessitates a deeper understanding of the specific contexts in which mycorrhizal fungi regulate litter decomposition and the mechanisms involved. To advance our knowledge and modelling capabilities regarding nutrient cycling responses to changing environments, improving our understanding of the mechanisms by which mycorrhizal fungi exert their influence on litter decomposition is crucial. However, the limited availability of evidence on this topic underscores the need for additional data and further research to comprehensively elucidate the role of mycorrhizal fungi in the mechanisms driving litter decomposition. In conclusion, our meta-analysis serves as a catalyst for future research endeavours, stimulating further investigations that will enhance our understanding of the roles of mycorrhizal fungi in nutrient cycling regulation across various scales. By expanding our knowledge in this area, we can ultimately improve the accuracy and effectiveness of ecosystem models and management strategies.</p>
</section>




<section id="references" class="level2">




</section>

<div id="quarto-appendix" class="default"><section id="acknowledgements" class="level2 appendix"><h2 class="anchored quarto-appendix-heading">Acknowledgements</h2><div class="quarto-appendix-contents">

<p>The authors are grateful to the Chairperson, Department of Botany, Panjab University, Chandigarh, to provide all the necessary facilities for the work. University Grants Commission, Government of India, New Delhi is acknowledged for financial support in the form of Junior Research Fellowships to Meenu Patil [(492/ (CSIR-UGC NET JUNE 2017)], Abhishek Kumar [507/ (OBC) (CSIR-UGC NET DEC. 2016)], and Pardeep Kumar [443/ (CSIR-UGC NET DEC. 2017)].</p>
</div></section><section id="author-contributions" class="level2 appendix"><h2 class="anchored quarto-appendix-heading">Author contributions</h2><div class="quarto-appendix-contents">

<p><strong>Meenu Patil:</strong> Conceptualisation (equal), Methodology (equal), Resources (equal), Investigation (equal), Data curation (equal), Formal analysis (equal), Visualisation (equal), Software (equal), Validation (equal), Writing – original draft (lead), Writing – review &amp; editing (equal), Funding acquisition (equal). <strong>Abhishek Kumar:</strong> Conceptualisation (equal), Methodology (equal), Resources (equal), Investigation (equal), Data curation (equal), Formal analysis (equal), Visualisation (equal), Software (equal), Validation (equal), Writing – original draft (equal), Writing – review &amp; editing (equal), Funding acquisition (equal). <strong>Pardeep Kumar:</strong> Conceptualisation (supporting), Methodology (supporting), Resources (supporting), Investigation (supporting), Validation (supporting), Writing – original draft (equal), Writing – review &amp; editing (equal), Funding acquisition (equal). <strong>Anand Narain Singh:</strong> Conceptualisation (equal), Methodology (supporting), Resources (supporting), Investigation (supporting), Data curation (equal), Formal analysis (supporting), Validation (equal), Writing – original draft (equal), Writing – review &amp; editing (lead), Supervision (lead), Project administration (lead).</p>
</div></section><section id="data-availability" class="level2 appendix"><h2 class="anchored quarto-appendix-heading">Data availability</h2><div class="quarto-appendix-contents">

<p>All data and codes are currently available from <a href="https://github.com/kumar-a/kumar-a.github.io/tree/main/publications/2023-patil" class="uri">https://github.com/kumar-a/kumar-a.github.io/tree/main/publications/2023-patil</a>. After acceptance, we will publish all datasets and <code>R</code> codes with a DOI in Zenodo or FigShare repository.</p>
</div></section><section id="funding" class="level2 appendix"><h2 class="anchored quarto-appendix-heading">Funding</h2><div class="quarto-appendix-contents">

<p>University Grants Commission, Government of India, New Delhi is acknowledged for financial support in the form of Senior Research Fellowships to <em>Meenu Patil</em> [(492/ (CSIR-UGC NET JUNE 2017)], <em>Abhishek Kumar</em> [507/ (OBC) (CSIR-UGC NET DEC. 2016)], and <em>Pardeep Kumar</em> [443/ (CSIR-UGC NET DEC. 2017)].</p>
</div></section><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" role="list">
<div id="ref-Averill2014" class="csl-entry" role="listitem">
Averill, Colin, Benjamin L. Turner, and Adrien C. Finzi. 2014. <span>“Mycorrhiza-Mediated Competition Between Plants and Decomposers Drives Soil Carbon Storage.”</span> <em>Nature</em> 505 (7484): 543–45. <a href="https://doi.org/10.1038/nature12901">https://doi.org/10.1038/nature12901</a>.
</div>
<div id="ref-Beeck2018" class="csl-entry" role="listitem">
Beeck, Michiel Op De, Carl Troein, Carsten Peterson, Per Persson, and Anders Tunlid. 2018. <span>“Fenton Reaction Facilitates Organic Nitrogen Acquisition by an Ectomycorrhizal Fungus.”</span> <em>New Phytologist</em> 218 (1): 335–43. <a href="https://doi.org/10.1111/nph.14971">https://doi.org/10.1111/nph.14971</a>.
</div>
<div id="ref-Begg1994" class="csl-entry" role="listitem">
Begg, Colin B., and Madhuchhanda Mazumdar. 1994. <span>“Operating Characteristics of a Rank Correlation Test for Publication Bias.”</span> <em>Biometrics</em> 50 (4): 1088–1101. <a href="https://doi.org/10.2307/2533446">https://doi.org/10.2307/2533446</a>.
</div>
<div id="ref-Borenstein2009" class="csl-entry" role="listitem">
Borenstein, Michael, Larry V. Hedges, Julian P. T. Higgins, and Hannah R. Rothstein. 2009. <em>Introduction to Meta-Analysis</em>. 1st ed. Chichester: John Wiley <span>&amp;</span> Sons, Ltd. <a href="https://doi.org/10.1002/9780470743386">https://doi.org/10.1002/9780470743386</a>.
</div>
<div id="ref-Borenstein2010" class="csl-entry" role="listitem">
———. 2010. <span>“A Basic Introduction to Fixed-Effect and Random-Effects Models for Meta-Analysis.”</span> <em>Research Synthesis Methods</em> 1 (2): 97–111. <a href="https://doi.org/10.1002/jrsm.12">https://doi.org/10.1002/jrsm.12</a>.
</div>
<div id="ref-Brundrett2018" class="csl-entry" role="listitem">
Brundrett, Mark C., and Leho Tedersoo. 2018. <span>“Evolutionary History of Mycorrhizal Symbioses and Global Host Plant Diversity.”</span> <em>New Phytologist</em> 220 (4): 1108–15. <a href="https://doi.org/10.1111/nph.14976">https://doi.org/10.1111/nph.14976</a>.
</div>
<div id="ref-Brzostek2015" class="csl-entry" role="listitem">
Brzostek, Edward R., Danilo Dragoni, Zachary A. Brown, and Richard P. Phillips. 2015. <span>“Mycorrhizal Type Determines the Magnitude and Direction of Root-Induced Changes in Decomposition in a Temperate Forest.”</span> <em>New Phytologist</em> 206 (4): 1274–82. <a href="https://doi.org/10.1111/nph.13303">https://doi.org/10.1111/nph.13303</a>.
</div>
<div id="ref-CEE2018" class="csl-entry" role="listitem">
CEE. 2018. <em>Guidelines and Standards for Evidence Synthesis in Environmental Management</em>. Edited by Andrew S Pullin, Geoff K Frampton, Barbara Livoreil, and Gillian Petrokofsky. 5th ed. Collaboration for Environmental Evidence. <a href="https://environmentalevidence.org/information-for-authors/">https://environmentalevidence.org/information-for-authors/</a>.
</div>
<div id="ref-Cinar2021" class="csl-entry" role="listitem">
Cinar, Ozan, Shinichi Nakagawa, and Wolfgang Viechtbauer. 2021. <span>“Phylogenetic Multilevel Meta-Analysis: A Simulation Study on the Importance of Modelling the Phylogeny.”</span> <em>Methods in Ecology and Evolution</em> 13 (2): 383–95. <a href="https://doi.org/10.1111/2041-210x.13760">https://doi.org/10.1111/2041-210x.13760</a>.
</div>
<div id="ref-Cochran1954" class="csl-entry" role="listitem">
Cochran, William G. 1954. <span>“The Combination of Estimates from Different Experiments.”</span> <em>Biometrics</em> 10 (1): 101–29. <a href="https://doi.org/10.2307/3001666">https://doi.org/10.2307/3001666</a>.
</div>
<div id="ref-Cooper2019" class="csl-entry" role="listitem">
Cooper, Harris, Larry V. Hedges, and Jeffrey C. Valentine, eds. 2019. <em>The Handbook of Research Synthesis and Meta-Analysis</em>. 3rd ed. New York: Russell Sage Foundation.
</div>
<div id="ref-Cornwell2008" class="csl-entry" role="listitem">
Cornwell, William K., Johannes H. C. Cornelissen, Kathryn Amatangelo, Ellen Dorrepaal, Valerie T. Eviner, Oscar Godoy, Sarah E. Hobbie, et al. 2008. <span>“Plant Species Traits Are the Predominant Control on Litter Decomposition Rates Within Biomes Worldwide.”</span> <em>Ecology Letters</em> 11 (10): 1065–71. <a href="https://doi.org/10.1111/j.1461-0248.2008.01219.x">https://doi.org/10.1111/j.1461-0248.2008.01219.x</a>.
</div>
<div id="ref-Entry1991" class="csl-entry" role="listitem">
Entry, James A, Cathy L Rose, and Kermtt Cromack. 1991. <span>“Litter Decomposition and Nutrient Release in Ectomycorrhizal Mat Soils of a Douglas Fir Ecosystem.”</span> <em>Soil Biology and Biochemistry</em> 23 (3): 285–90. <a href="https://doi.org/10.1016/0038-0717(91)90065-r">https://doi.org/10.1016/0038-0717(91)90065-r</a>.
</div>
<div id="ref-Fernandez2016" class="csl-entry" role="listitem">
Fernandez, Christopher W., and Peter G. Kennedy. 2016. <span>“Revisiting the ’Gadgil Effect’: Do Interguild Fungal Interactions Control Carbon Cycling in Forest Soils?”</span> <em>New Phytologist</em> 209 (4): 1382–94. <a href="https://doi.org/10.1111/nph.13648">https://doi.org/10.1111/nph.13648</a>.
</div>
<div id="ref-Fernandez2020" class="csl-entry" role="listitem">
Fernandez, Christopher W., Craig R. See, and Peter G. Kennedy. 2020. <span>“<span class="nocase">Decelerated carbon cycling by ectomycorrhizal fungi is controlled by substrate quality and community composition</span>.”</span> <em>New Phytologist</em> 226 (2): 569–82. <a href="https://doi.org/10.1111/nph.16269">https://doi.org/10.1111/nph.16269</a>.
</div>
<div id="ref-Finlay2005" class="csl-entry" role="listitem">
Finlay, R. 2005. <span>“Action and Interaction in the Mycorrhizal Hyphosphere — a Re-Evaluation of the Role of Mycorrhizas in Nutrient Acquisition and Plant Ecology.”</span> In <em>Nutrient Acquisition by Plants</em>, edited by Hormoz BassiriRad, 221–76. Heidelberg: Springer-Verlag. <a href="https://doi.org/10.1007/3-540-27675-0_9">https://doi.org/10.1007/3-540-27675-0_9</a>.
</div>
<div id="ref-R-lcvplants" class="csl-entry" role="listitem">
Freiberg, Martin, Marten Winter, Alessandro Gentile, Alexander Zizka, Alexandra Nora Muellner-Riehl, Alexandra Weigelt, and Christian Wirth. 2020. <span>“<span>LCVP</span>, the Leipzig Catalogue of Vascular Plants, a New Taxonomic Reference List for All Known Vascular Plants.”</span> <em>Scientific Data</em> 7: 416. <a href="https://doi.org/10.1038/s41597-020-00702-z">https://doi.org/10.1038/s41597-020-00702-z</a>.
</div>
<div id="ref-Frey2019" class="csl-entry" role="listitem">
Frey, Serita D. 2019. <span>“Mycorrhizal Fungi as Mediators of Soil Organic Matter Dynamics.”</span> <em>Annual Review of Ecology, Evolution, and Systematics</em> 50 (1): 237–59. <a href="https://doi.org/10.1146/annurev-ecolsys-110617-062331">https://doi.org/10.1146/annurev-ecolsys-110617-062331</a>.
</div>
<div id="ref-Gadgil1971" class="csl-entry" role="listitem">
Gadgil, Ruth L., and P. D. Gadgil. 1971. <span>“<span class="nocase">Mycorrhiza and litter decomposition</span>.”</span> <em>Nature</em> 233 (5315): 133. <a href="https://doi.org/10.1038/233133a0">https://doi.org/10.1038/233133a0</a>.
</div>
<div id="ref-Gadgil1975" class="csl-entry" role="listitem">
Gadgil, Ruth L, and P D Gadgil. 1975. <span>“Suppression of Litter Decomposition by Mycorrhizal Roots of <span class="nocase">Pinus radiata</span>.”</span> <em>New Zealand Journal of Forestry Science</em> 5 (1): 33–41.
</div>
<div id="ref-Gorka2019" class="csl-entry" role="listitem">
Gorka, Stefan, Marlies Dietrich, Werner Mayerhofer, Raphael Gabriel, Julia Wiesenbauer, Victoria Martin, Qing Zheng, et al. 2019. <span>“Rapid Transfer of Plant Photosynthates to Soil Bacteria via Ectomycorrhizal Hyphae and Its Interaction with Nitrogen Availability.”</span> <em>Frontiers in Microbiology</em> 10 (February): 168. <a href="https://doi.org/10.3389/fmicb.2019.00168">https://doi.org/10.3389/fmicb.2019.00168</a>.
</div>
<div id="ref-Grafen1989" class="csl-entry" role="listitem">
Grafen, Alan. 1989. <span>“The Phylogenetic Regression.”</span> <em>Philosophical Transactions of the Royal Society B: Biological Sciences</em> 326 (1233): 119–57. <a href="https://doi.org/10.1098/rstb.1989.0106">https://doi.org/10.1098/rstb.1989.0106</a>.
</div>
<div id="ref-Gui2017" class="csl-entry" role="listitem">
Gui, Heng, Kevin Hyde, Jianchu Xu, and Peter Mortimer. 2017. <span>“Arbuscular Mycorrhiza Enhance the Rate of Litter Decomposition While Inhibiting Soil Microbial Community Development.”</span> <em>Scientific Reports</em> 7 (1): 42184. <a href="https://doi.org/10.1038/srep42184">https://doi.org/10.1038/srep42184</a>.
</div>
<div id="ref-Haddaway2018" class="csl-entry" role="listitem">
Haddaway, Neal R., Biljana Macura, Paul Whaley, and Andrew S. Pullin. 2018. <span>“<span>ROSES</span> Reporting Standards for Systematic Evidence Syntheses: Pro Forma, Flow-Diagram and Descriptive Summary of the Plan and Conduct of Environmental Systematic Reviews and Systematic Maps.”</span> <em>Environmental Evidence</em> 7 (1): 1–8. <a href="https://doi.org/10.1186/s13750-018-0121-7">https://doi.org/10.1186/s13750-018-0121-7</a>.
</div>
<div id="ref-Harrer2021" class="csl-entry" role="listitem">
Harrer, Mathias, Pim Cuijpers, Toshi A. Furukawa, and David D. Ebert. 2021. <em>Doing Meta-Analysis with <span>R</span></em>. Boca Raton: Chapman <span>and</span> Hall/CRC. <a href="https://doi.org/10.1201/9781003107347">https://doi.org/10.1201/9781003107347</a>.
</div>
<div id="ref-Hedges1999" class="csl-entry" role="listitem">
Hedges, Larry V., Jessica Gurevitch, and Peter S. Curtis. 1999. <span>“The Meta-Analysis of Response Ratios in Experimental Ecology.”</span> <em>Ecology</em> 80 (4): 1150–56. <a href="https://doi.org/10.1890/0012-9658(1999)080[1150:tmaorr]2.0.co;2">https://doi.org/10.1890/0012-9658(1999)080[1150:tmaorr]2.0.co;2</a>.
</div>
<div id="ref-Heijden2015" class="csl-entry" role="listitem">
Heijden, Marcel G. A., Francis M. Martin, Marc-André Selosse, and Ian R. Sanders. 2015. <span>“Mycorrhizal Ecology and Evolution: The Past, the Present, and the Future.”</span> <em>New Phytologist</em> 205 (4): 1406–23. <a href="https://doi.org/10.1111/nph.13288">https://doi.org/10.1111/nph.13288</a>.
</div>
<div id="ref-Herman2012" class="csl-entry" role="listitem">
Herman, Donald J., Mary K. Firestone, Erin Nuccio, and Angela Hodge. 2012. <span>“Interactions Between an Arbuscular Mycorrhizal Fungus and a Soil Microbial Community Mediating Litter Decomposition.”</span> <em>FEMS Microbiology Ecology</em> 80 (1): 236–47. <a href="https://doi.org/10.1111/j.1574-6941.2011.01292.x">https://doi.org/10.1111/j.1574-6941.2011.01292.x</a>.
</div>
<div id="ref-Higgins2002" class="csl-entry" role="listitem">
Higgins, Julian P. T., and Simon G. Thompson. 2002. <span>“Quantifying Heterogeneity in a Meta-Analysis.”</span> <em>Statistics in Medicine</em> 21 (11): 1539–58. <a href="https://doi.org/10.1002/sim.1186">https://doi.org/10.1002/sim.1186</a>.
</div>
<div id="ref-Hodge2001" class="csl-entry" role="listitem">
Hodge, Angela, Colin D. Campbell, and Alastair H. Fitter. 2001. <span>“An Arbuscular Mycorrhizal Fungus Accelerates Decomposition and Acquires Nitrogen Directly from Organic Material.”</span> <em>Nature</em> 413 (6853): 297–99. <a href="https://doi.org/10.1038/35095041">https://doi.org/10.1038/35095041</a>.
</div>
<div id="ref-Jackson2019" class="csl-entry" role="listitem">
Jackson, Oyindamola, Richard S. Quilliam, Andy Stott, Helen Grant, and Jens-Arne Subke. 2019. <span>“Rhizosphere Carbon Supply Accelerates Soil Organic Matter Decomposition in the Presence of Fresh Organic Substrates.”</span> <em>Plant and Soil</em> 440 (1-2): 473–90. <a href="https://doi.org/10.1007/s11104-019-04072-3">https://doi.org/10.1007/s11104-019-04072-3</a>.
</div>
<div id="ref-R-VPhyloMaker2" class="csl-entry" role="listitem">
Jin, Yi, and Hong Qian. 2022. <span>“<span>V.PhyloMaker2</span>: An Updated and Enlarged <span>R</span> Package That Can Generate Very Large Phylogenies for Vascular Plants.”</span> <em>Plant Diversity</em> 44 (4): 335–39. <a href="https://doi.org/10.1016/j.pld.2022.05.005">https://doi.org/10.1016/j.pld.2022.05.005</a>.
</div>
<div id="ref-Keller2019" class="csl-entry" role="listitem">
Keller, Adrienne B., and Richard P. Phillips. 2019. <span>“Leaf Litter Decay Rates Differ Between Mycorrhizal Groups in Temperate, but Not Tropical, Forests.”</span> <em>New Phytologist</em> 222 (1): 556–64. <a href="https://doi.org/10.1111/nph.15524">https://doi.org/10.1111/nph.15524</a>.
</div>
<div id="ref-Koide2003" class="csl-entry" role="listitem">
Koide, Roger T., and T. Wu. 2003. <span>“Ectomycorrhizas and Retarded Decomposition in a Pinus Resinosa Plantation.”</span> <em>New Phytologist</em> 158 (2): 401–7. <a href="https://doi.org/10.1046/j.1469-8137.2003.00732.x">https://doi.org/10.1046/j.1469-8137.2003.00732.x</a>.
</div>
<div id="ref-Koricheva2014" class="csl-entry" role="listitem">
Koricheva, Julia, and Jessica Gurevitch. 2014. <span>“Uses and Misuses of Meta-Analysis in Plant Ecology.”</span> <em>Journal of Ecology</em> 102 (4): 828–44. <a href="https://doi.org/10.1111/1365-2745.12224">https://doi.org/10.1111/1365-2745.12224</a>.
</div>
<div id="ref-Krywolap1964" class="csl-entry" role="listitem">
Krywolap, George N., Larry F. Grand, and L. E. Casida Jr. 1964. <span>“The Natural Occurrence of an Antibiotic in the Mycorrhizal Fungus.”</span> <em>Canadian Journal of Microbiology</em> 10 (3): 323–28. <a href="https://doi.org/10.1139/m64-044">https://doi.org/10.1139/m64-044</a>.
</div>
<div id="ref-Kubicek2011" class="csl-entry" role="listitem">
Kubicek, Christian P, Alfredo Herrera-Estrella, Verena Seidl-Seiboth, Diego A Martinez, Irina S Druzhinina, Michael Thon, Susanne Zeilinger, et al. 2011. <span>“Comparative Genome Sequence Analysis Underscores Mycoparasitism as the Ancestral Life Style of <span>Trichoderma</span>.”</span> <em>Genome Biology</em> 12 (4): R40. <a href="https://doi.org/10.1186/gb-2011-12-4-r40">https://doi.org/10.1186/gb-2011-12-4-r40</a>.
</div>
<div id="ref-Kumar2022" class="csl-entry" role="listitem">
Kumar, Abhishek, Meenu Patil, Pardeep Kumar, Manoj Kumar, and Anand Narain Singh. 2022. <span>“Plant Ecology in <span>Indian Siwalik</span> Range: A Systematic Map and Its Bibliometric Analysis.”</span> <em>Tropical Ecology</em> 63 (September): 338–50. <a href="https://doi.org/10.1007/s42965-022-00229-x">https://doi.org/10.1007/s42965-022-00229-x</a>.
</div>
<div id="ref-Kumar2021" class="csl-entry" role="listitem">
Kumar, Abhishek, Meenu Patil, Pardeep Kumar, and Anand Narain Singh. 2021. <span>“Phosphorus and Litter Decomposability Traits in Tropical Forest Ecosystems Under Changing Environment: A Synthesis.”</span> In <em>Tropical Dry Forests: Emerging Features and Ecological Perspectives</em>, edited by Ravi Kant Chaturvedi, Rishikesh Singh, and Rahul Bhadouria, 311–36. New York: Nova Science Publishers.
</div>
<div id="ref-Lajeunesse2015" class="csl-entry" role="listitem">
Lajeunesse, Marc J. 2015. <span>“Bias and Correction for the Log Response Ratio in Ecological Meta-Analysis.”</span> <em>Ecology</em> 96 (8): 2056–63. <a href="https://doi.org/10.1890/14-2402.1">https://doi.org/10.1890/14-2402.1</a>.
</div>
<div id="ref-Lang2021" class="csl-entry" role="listitem">
Lang, Ashley K., Fiona V. Jevon, Corinne R. Vietorisz, Matthew P. Ayres, and Jaclyn Hatala Matthes. 2021. <span>“Fine Roots and Mycorrhizal Fungi Accelerate Leaf Litter Decomposition in a Northern Hardwood Forest Regardless of Dominant Tree Mycorrhizal Associations.”</span> <em>New Phytologist</em> 230 (1): 316–26. <a href="https://doi.org/10.1111/nph.17155">https://doi.org/10.1111/nph.17155</a>.
</div>
<div id="ref-Leifheit2015" class="csl-entry" role="listitem">
Leifheit, E. F., E. Verbruggen, and M. C. Rillig. 2015. <span>“Arbuscular Mycorrhizal Fungi Reduce Decomposition of Woody Plant Litter While Increasing Soil Aggregation.”</span> <em>Soil Biology and Biochemistry</em> 81 (February): 323–28. <a href="https://doi.org/10.1016/j.soilbio.2014.12.003">https://doi.org/10.1016/j.soilbio.2014.12.003</a>.
</div>
<div id="ref-Mayor2006" class="csl-entry" role="listitem">
Mayor, Jordan R., and Terry W. Henkel. 2006. <span>“Do Ectomycorrhizas Alter Leaf-Litter Decomposition in Monodominant Tropical Forests of Guyana?”</span> <em>New Phytologist</em> 169 (3): 579–88. <a href="https://doi.org/10.1111/j.1469-8137.2005.01607.x">https://doi.org/10.1111/j.1469-8137.2005.01607.x</a>.
</div>
<div id="ref-McGuire2010" class="csl-entry" role="listitem">
McGuire, Krista L., and Kathleen K. Treseder. 2010. <span>“Microbial Communities and Their Relevance for Ecosystem Models: Decomposition as a Case Study.”</span> <em>Soil Biology and Biochemistry</em> 42 (4): 529–35. <a href="https://doi.org/10.1016/j.soilbio.2009.11.016">https://doi.org/10.1016/j.soilbio.2009.11.016</a>.
</div>
<div id="ref-Nakagawa2017" class="csl-entry" role="listitem">
Nakagawa, Shinichi, Daniel W. A. Noble, Alistair M. Senior, and Malgorzata Lagisz. 2017. <span>“Meta-Evaluation of Meta-Analysis: Ten Appraisal Questions for Biologists.”</span> <em><span>BMC</span> Biology</em> 15 (1): 18. <a href="https://doi.org/10.1186/s12915-017-0357-7">https://doi.org/10.1186/s12915-017-0357-7</a>.
</div>
<div id="ref-Nakagawa2012" class="csl-entry" role="listitem">
Nakagawa, Shinichi, and Eduardo S. A. Santos. 2012. <span>“Methodological Issues and Advances in Biological Meta-Analysis.”</span> <em>Evolutionary Ecology</em> 26 (5): 1253–74. <a href="https://doi.org/10.1007/s10682-012-9555-5">https://doi.org/10.1007/s10682-012-9555-5</a>.
</div>
<div id="ref-Olson1963" class="csl-entry" role="listitem">
Olson, Jerry S. 1963. <span>“Energy Storage and the Balance of Producers and Decomposers in Ecological Systems.”</span> <em>Ecology</em> 44 (2): 322–31. <a href="https://doi.org/10.2307/1932179">https://doi.org/10.2307/1932179</a>.
</div>
<div id="ref-Orwin2011" class="csl-entry" role="listitem">
Orwin, Kate H., Miko U. F. Kirschbaum, Mark G. St John, and Ian A. Dickie. 2011. <span>“Organic Nutrient Uptake by Mycorrhizal Fungi Enhances Ecosystem Carbon Storage: A Model-Based Assessment.”</span> <em>Ecology Letters</em> 14 (5): 493–502. <a href="https://doi.org/10.1111/j.1461-0248.2011.01611.x">https://doi.org/10.1111/j.1461-0248.2011.01611.x</a>.
</div>
<div id="ref-Pan2011" class="csl-entry" role="listitem">
Pan, Yude, Richard A. Birdsey, Jingyun Fang, Richard Houghton, Pekka E. Kauppi, Werner A. Kurz, Oliver L. Phillips, et al. 2011. <span>“A Large and Persistent Carbon Sink in the World’s Forests.”</span> <em>Science</em> 333 (6045): 988–93. <a href="https://doi.org/10.1126/science.1201609">https://doi.org/10.1126/science.1201609</a>.
</div>
<div id="ref-R-ape" class="csl-entry" role="listitem">
Paradis, E., and K. Schliep. 2019. <span>“<span class="nocase">ape 5.0</span>: An Environment for Modern Phylogenetics and Evolutionary Analyses in <span>R</span>.”</span> <em>Bioinformatics</em> 35: 526–28.
</div>
<div id="ref-Patil2020srep" class="csl-entry" role="listitem">
Patil, Meenu, Abhishek Kumar, Pardeep Kumar, Navneet Kaur Cheema, Rupinder Kaur, Ramchand Bhatti, and A. N. Singh. 2020. <span>“Comparative Litter Decomposability Traits of Selected Native and Exotic Woody Species from an Urban Environment of North-Western <span>Siwalik</span> Region, <span>India</span>.”</span> <em>Scientific Reports</em> 10 (May): 7888. <a href="https://doi.org/10.1038/s41598-020-64576-2">https://doi.org/10.1038/s41598-020-64576-2</a>.
</div>
<div id="ref-Patil2020azb" class="csl-entry" role="listitem">
Patil, Meenu, Abhishek Kumar, Pardeep Kumar, and A. N. Singh. 2020. <span>“<span class="nocase">Litter decomposability traits and their linkage with the cycling of nutrients in the forest ecosystems under the lens of climate change: A conceptual overview</span>.”</span> <em>Advances in Zoology and Botany</em> 8 (3): 269–77. <a href="https://doi.org/10.13189/azb.2020.080324">https://doi.org/10.13189/azb.2020.080324</a>.
</div>
<div id="ref-Qian2016" class="csl-entry" role="listitem">
Qian, Hong, and Yi Jin. 2016. <span>“An Updated Megaphylogeny of Plants, a Tool for Generating Plant Phylogenies and an Analysis of Phylogenetic Community Structure.”</span> <em>Journal of Plant Ecology</em> 9 (2): 233–39. <a href="https://doi.org/10.1093/jpe/rtv047">https://doi.org/10.1093/jpe/rtv047</a>.
</div>
<div id="ref-R-base" class="csl-entry" role="listitem">
R Core Team. 2023. <em>R: A Language and Environment for Statistical Computing</em> (version 4.3.0). Vienna, Austria: R Foundation for Statistical Computing. <a href="https://www.R-project.org/">https://www.R-project.org/</a>.
</div>
<div id="ref-Rasanayagam1992" class="csl-entry" role="listitem">
Rasanayagam, S., and P. Jeffries. 1992. <span>“Production of Acid Is Responsible for Antibiosis by Some Ectomycorrhizal Fungi.”</span> <em>Mycological Research</em> 96 (11): 971–76. <a href="https://doi.org/10.1016/s0953-7562(09)80600-x">https://doi.org/10.1016/s0953-7562(09)80600-x</a>.
</div>
<div id="ref-Rineau2013" class="csl-entry" role="listitem">
Rineau, F, F Shah, M M Smits, P Persson, T Johansson, R Carleer, C Troein, and A Tunlid. 2013. <span>“Carbon Availability Triggers the Decomposition of Plant Litter and Assimilation of Nitrogen by an Ectomycorrhizal Fungus.”</span> <em>ISME Journal</em> 7 (10): 2010–22. <a href="https://doi.org/10.1038/ismej.2013.91">https://doi.org/10.1038/ismej.2013.91</a>.
</div>
<div id="ref-Rosenberg2005" class="csl-entry" role="listitem">
Rosenberg, Michael S. 2005. <span>“The File-Drawer Problem Revisited: A General Weighted Method for Calculating Fail-Safe Numbers in Meta-Analysis.”</span> <em>Evolution</em> 59 (2): 464–68. <a href="https://doi.org/10.1111/j.0014-3820.2005.tb01004.x">https://doi.org/10.1111/j.0014-3820.2005.tb01004.x</a>.
</div>
<div id="ref-Rosenthal1979" class="csl-entry" role="listitem">
Rosenthal, Robert. 1979. <span>“The File Drawer Problem and Tolerance for Null Results.”</span> <em>Psychological Bulletin</em> 86 (3): 638–41. <a href="https://doi.org/10.1037/0033-2909.86.3.638">https://doi.org/10.1037/0033-2909.86.3.638</a>.
</div>
<div id="ref-Schneider2012" class="csl-entry" role="listitem">
Schneider, Thomas, Katharina M. Keiblinger, Emanuel Schmid, Katja Sterflinger-Gleixner, Günther Ellersdorfer, Bernd Roschitzki, Andreas Richter, Leo Eberl, Sophie Zechmeister-Boltenstern, and Kathrin Riedel. 2012. <span>“Who Is Who in Litter Decomposition Metaproteomics Reveals Major Microbial Players and Their Biogeochemical Functions.”</span> <em>ISME Journal</em> 6 (9): 1749–62. <a href="https://doi.org/10.1038/ismej.2012.11">https://doi.org/10.1038/ismej.2012.11</a>.
</div>
<div id="ref-Senior2016" class="csl-entry" role="listitem">
Senior, Alistair M., Catherine E. Grueber, Tsukushi Kamiya, Malgorzata Lagisz, Katie O’Dwyer, Eduardo S. A. Santos, and Shinichi Nakagawa. 2016. <span>“Heterogeneity in Ecological and Evolutionary Meta-Analyses: Its Magnitude and Implications.”</span> <em>Ecology</em> 97 (12): 3293–99. <a href="https://doi.org/10.1002/ecy.1591">https://doi.org/10.1002/ecy.1591</a>.
</div>
<div id="ref-Shah2016" class="csl-entry" role="listitem">
Shah, Firoz, César Nicolás, Johan Bentzer, Magnus Ellström, Mark Smits, Francois Rineau, Björn Canbäck, et al. 2016. <span>“Ectomycorrhizal Fungi Decompose Soil Organic Matter Using Oxidative Mechanisms Adapted from Saprotrophic Ancestors.”</span> <em>New Phytologist</em> 209 (4): 1705–19. <a href="https://doi.org/10.1111/nph.13722">https://doi.org/10.1111/nph.13722</a>.
</div>
<div id="ref-Shu2019" class="csl-entry" role="listitem">
Shu, Meng, Qingzhou Zhao, Zhen Li, Lin Zhang, Peng Wang, and Shuijin Hu. 2019. <span>“Effects of Global Change Factors and Living Roots on Root Litter Decomposition in a Qinghai-Tibet Alpine Meadow.”</span> <em>Scientific Reports</em> 9 (1): 16924. <a href="https://doi.org/10.1038/s41598-019-53450-5">https://doi.org/10.1038/s41598-019-53450-5</a>.
</div>
<div id="ref-Smith2018" class="csl-entry" role="listitem">
Smith, Stephen A., and Joseph W. Brown. 2018. <span>“Constructing a Broadly Inclusive Seed Plant Phylogeny.”</span> <em>American Journal of Botany</em> 105 (3): 302–14. <a href="https://doi.org/10.1002/ajb2.1019">https://doi.org/10.1002/ajb2.1019</a>.
</div>
<div id="ref-Staaf1988" class="csl-entry" role="listitem">
Staaf, H. 1988. <span>“Litter Decomposition in Beech Forests ? Effects of Excluding Tree Roots.”</span> <em>Biology and Fertility of Soils</em> 6 (4): 302–5. <a href="https://doi.org/10.1007/bf00261017">https://doi.org/10.1007/bf00261017</a>.
</div>
<div id="ref-Steidinger2019" class="csl-entry" role="listitem">
Steidinger, B. S., T. W. Crowther, J. Liang, M. E. Van Nuland, G. D. A. Werner, P. B. Reich, G. J. Nabuurs, et al. 2019. <span>“Climatic Controls of Decomposition Drive the Global Biogeography of Forest-Tree Symbioses.”</span> <em>Nature</em> 569 (7756): 404–8. <a href="https://doi.org/10.1038/s41586-019-1128-0">https://doi.org/10.1038/s41586-019-1128-0</a>.
</div>
<div id="ref-Sterkenburg2018" class="csl-entry" role="listitem">
Sterkenburg, Erica, Karina E. Clemmensen, Alf Ekblad, Roger D. Finlay, and Björn D. Lindahl. 2018. <span>“Contrasting Effects of Ectomycorrhizal Fungi on Early and Late Stage Decomposition in a Boreal Forest.”</span> <em>ISME Journal</em> 12 (9): 2187–97. <a href="https://doi.org/10.1038/s41396-018-0181-2">https://doi.org/10.1038/s41396-018-0181-2</a>.
</div>
<div id="ref-Sterne2006" class="csl-entry" role="listitem">
Sterne, Jonathan A. C., and Matthias Egger. 2006. <span>“Regression Methods to Detect Publication and Other Bias in Meta-Analysis.”</span> In <em>Publication Bias in Meta-Analysis</em>, edited by Hannah R. Rothstein, Alexander J. Sutton, and Michael Borenstein, 99–110. John Wiley <span>&amp;</span> Sons, Ltd. <a href="https://doi.org/10.1002/0470870168.ch6">https://doi.org/10.1002/0470870168.ch6</a>.
</div>
<div id="ref-Sulman2017" class="csl-entry" role="listitem">
Sulman, Benjamin N., Edward R. Brzostek, Chiara Medici, Elena Shevliakova, Duncan N. L. Menge, and Richard P. Phillips. 2017. <span>“Feedbacks Between Plant <span>N</span> Demand and Rhizosphere Priming Depend on Type of Mycorrhizal Association.”</span> <em>Ecology Letters</em> 20 (8): 1043–53. <a href="https://doi.org/10.1111/ele.12802">https://doi.org/10.1111/ele.12802</a>.
</div>
<div id="ref-Talbot2015" class="csl-entry" role="listitem">
Talbot, Jennifer M., Francis Martin, Annegret Kohler, Bernard Henrissat, and Kabir G. Peay. 2015. <span>“Functional Guild Classification Predicts the Enzymatic Role of Fungi in Litter and Soil Biogeochemistry.”</span> <em>Soil Biology and Biochemistry</em> 88 (September): 441–56. <a href="https://doi.org/10.1016/j.soilbio.2015.05.006">https://doi.org/10.1016/j.soilbio.2015.05.006</a>.
</div>
<div id="ref-Tedersoo2019" class="csl-entry" role="listitem">
Tedersoo, Leho, and Mohammad Bahram. 2019. <span>“Mycorrhizal Types Differ in Ecophysiology and Alter Plant Nutrition and Soil Processes.”</span> <em>Biological Reviews</em> 94 (5): 1857–80. <a href="https://doi.org/10.1111/brv.12538">https://doi.org/10.1111/brv.12538</a>.
</div>
<div id="ref-Viechtbauer2005" class="csl-entry" role="listitem">
Viechtbauer, Wolfgang. 2005. <span>“Bias and Efficiency of Meta-Analytic Variance Estimators in the Random-Effects Model.”</span> <em>Journal of Educational and Behavioral Statistics</em> 30 (3): 261–93. <a href="https://doi.org/10.3102/10769986030003261">https://doi.org/10.3102/10769986030003261</a>.
</div>
<div id="ref-R-metafor" class="csl-entry" role="listitem">
———. 2010. <span>“Conducting Meta-Analyses in <span>R</span> with the <span class="nocase">metafor</span> Package.”</span> <em>Journal of Statistical Software</em> 36 (3): 1–48. <a href="https://doi.org/10.18637/jss.v036.i03">https://doi.org/10.18637/jss.v036.i03</a>.
</div>
<div id="ref-Viechtbauer2010" class="csl-entry" role="listitem">
Viechtbauer, Wolfgang, and Mike W.-L. Cheung. 2010. <span>“Outlier and Influence Diagnostics for Meta-Analysis.”</span> <em>Research Synthesis Methods</em> 1 (2): 112–25. <a href="https://doi.org/10.1002/jrsm.11">https://doi.org/10.1002/jrsm.11</a>.
</div>
<div id="ref-R-tidyverse" class="csl-entry" role="listitem">
Wickham, Hadley, Mara Averick, Jennifer Bryan, Winston Chang, Lucy McGowan, Romain François, Garrett Grolemund, et al. 2019. <span>“Welcome to the Tidyverse.”</span> <em>Journal of Open Source Software</em> 4 (43): 1686. <a href="https://doi.org/10.21105/joss.01686">https://doi.org/10.21105/joss.01686</a>.
</div>
<div id="ref-R-ggtree" class="csl-entry" role="listitem">
Yu, Guangchuang, David K. Smith, Huachen Zhu, Yi Guan, and Tommy Tsan-Yuk Lam. 2016. <span>“<span class="nocase">ggtree</span>: An <span>R</span> Package for Visualization and Annotation of Phylogenetic Trees with Their Covariates and Other Associated Data.”</span> <em>Methods in Ecology and Evolution</em> 8 (1): 28–36. <a href="https://doi.org/10.1111/2041-210x.12628">https://doi.org/10.1111/2041-210x.12628</a>.
</div>
<div id="ref-Zanne2014" class="csl-entry" role="listitem">
Zanne, Amy E., David C. Tank, William K. Cornwell, Jonathan M. Eastman, Stephen A. Smith, Richard G. FitzJohn, Daniel J. McGlinn, et al. 2014. <span>“Three Keys to the Radiation of Angiosperms into Freezing Environments.”</span> <em>Nature</em> 506 (7486): 89–92. <a href="https://doi.org/10.1038/nature12872">https://doi.org/10.1038/nature12872</a>.
</div>
<div id="ref-Zhang2008" class="csl-entry" role="listitem">
Zhang, Deqiang, Dafeng Hui, Yiqi Luo, and Guoyi Zhou. 2008. <span>“Rates of Litter Decomposition in Terrestrial Ecosystems: Global Patterns and Controlling Factors.”</span> <em>Journal of Plant Ecology</em> 1 (2): 85–93. <a href="https://doi.org/10.1093/jpe/rtn002">https://doi.org/10.1093/jpe/rtn002</a>.
</div>
<div id="ref-Zhu1996" class="csl-entry" role="listitem">
Zhu, Weixing, and Joan G. Ehrenfeld. 1996. <span>“The Effects of Mycorrhizal Roots on Litter Decomposition, Soil Biota, and Nutrients in a Spodosolic Soil.”</span> <em>Plant and Soil</em> 179 (1): 109–18. <a href="https://doi.org/10.1007/bf00011648">https://doi.org/10.1007/bf00011648</a>.
</div>
</div></section></div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>