<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.353">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Abhishek Kumar">
<meta name="author" content="Pardeep Kumar">
<meta name="author" content="Meenu Patil">
<meta name="author" content="Sheenu Sharma">
<meta name="author" content="Anand Narain Singh">
<meta name="dcterms.date" content="2023-06-17">
<meta name="description" content="We compared the soil restoration potential of exotic and native plant species on coal mine. Our results suggested that native species are more beneficial for soil restoration than the exotic species.">

<title>Abhishek Kumar - Disturbance and vegetation structure in an urban forest of Indian Siwaliks</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>

  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<link rel="stylesheet" href="../../styles.css">
</head>

<body class="floating nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="../../index.html">
    <span class="navbar-title">Abhishek Kumar</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../../about.html" rel="" target="">
 <span class="menu-text">About</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../index.html" rel="" target="">
 <span class="menu-text">Blog</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../publications.html" rel="" target="">
 <span class="menu-text">Publications</span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/kumar-a" rel="" target=""><i class="bi bi-github" role="img">
</i> 
 <span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://twitter.com/abkumar_" rel="" target=""><i class="bi bi-twitter" role="img">
</i> 
 <span class="menu-text"></span></a>
  </li>  
</ul>
            <div class="quarto-navbar-tools">
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation floating overflow-auto">
    <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Contents</h2>
   
  <ul class="collapse">
  <li><a href="#abstract" id="toc-abstract" class="nav-link active" data-scroll-target="#abstract">Abstract</a></li>
  <li><a href="#introduction" id="toc-introduction" class="nav-link" data-scroll-target="#introduction">Introduction</a></li>
  <li><a href="#methodology" id="toc-methodology" class="nav-link" data-scroll-target="#methodology">Methodology</a></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results">Results</a></li>
  <li><a href="#discussion" id="toc-discussion" class="nav-link" data-scroll-target="#discussion">Discussion</a></li>
  <li><a href="#conclusion" id="toc-conclusion" class="nav-link" data-scroll-target="#conclusion">Conclusion</a></li>
  
  
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references">References</a></li>
  </ul>
</nav>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Disturbance and vegetation structure in an urban forest of Indian Siwaliks</h1>
  <div class="quarto-categories">
    <div class="quarto-category">Disturbance</div>
    <div class="quarto-category">Remote Sensing</div>
    <div class="quarto-category">Taxonomic Diversity</div>
    <div class="quarto-category">Phylogenetic Diversity</div>
    <div class="quarto-category">Journal Article</div>
  </div>
  </div>

<div>
  <div class="description">
    <p>We compared the soil restoration potential of exotic and native plant species on coal mine. Our results suggested that native species are more beneficial for soil restoration than the exotic species.</p>
  </div>
</div>

<div class="quarto-title-meta-author">
  <div class="quarto-title-meta-heading">Authors</div>
  <div class="quarto-title-meta-heading">Affiliation</div>
  
    <div class="quarto-title-meta-contents">
    <p class="author"><a href="https://akumar.netlify.app/">Abhishek Kumar</a> <a href="https://orcid.org/0000-0003-2252-7623" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            <a href="https://puchd.ac.in">
            Panjab University, Chandigarh
            </a>
          </p>
      </div>
      <div class="quarto-title-meta-contents">
    <p class="author">Pardeep Kumar </p>
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            <a href="https://puchd.ac.in">
            Panjab University, Chandigarh
            </a>
          </p>
      </div>
      <div class="quarto-title-meta-contents">
    <p class="author">Meenu Patil </p>
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            <a href="https://puchd.ac.in">
            Panjab University, Chandigarh
            </a>
          </p>
      </div>
      <div class="quarto-title-meta-contents">
    <p class="author">Sheenu Sharma </p>
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            <a href="https://puchd.ac.in">
            Panjab University, Chandigarh
            </a>
          </p>
      </div>
      <div class="quarto-title-meta-contents">
    <p class="author"><a href="https://www.researchgate.net/profile/Anand-Singh-15">Anand Narain Singh</a> <a href="https://orcid.org/0000-0002-0148-8680" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            <a href="https://puchd.ac.in">
            Panjab University, Chandigarh
            </a>
          </p>
      </div>
    </div>

<div class="quarto-title-meta">

      
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">June 17, 2023</p>
    </div>
  </div>
  
    
  </div>
  

</header>

<section id="abstract" class="level2">
<h2 class="anchored" data-anchor-id="abstract">Abstract</h2>
<p>Urban forests are facing multiple human-mediated pressures leading to compromised ecosystem structure and functioning. Therefore, understanding ecosystem structure in response to ongoing pressures is crucial for sustaining ecological integrity and human well-being. We aim to assess the disturbance and its effects on vegetation structure of urban forest in Chandigarh using a combination of remote sensing techniques and vegetation survey. The disturbance was assessed as change in NDVI (Normalised Difference Vegetation Index) from 2001 to 2021 by applying the BFAST (Breaks For Additive Season and Trend) algorithm to the MODIS satellite imagery data. A vegetation survey was conducted to compare the species composition, taxonomic and phylogenetic diversity as measures of vegetation structure. While signals of disturbance were evident, the changes in vegetation structure were not well established from our study. Further, vegetation analysis indicated no significant differences in vegetation composition due to disturbance (<em>F<sub>1,12</sub></em> = 0.91, <em>p</em> = 0.575). However, the phylogenetic diversity was substantially lower for disturbed plots as compare to undisturbed plots, though the taxonomic diversity was similar among the disturbed and undisturbed plots. Our results suggests that the effects of disturbance are more prominent on phylogenetic diversity than the taxonomic diversity. These findings can be considered as early signals of disturbance and its effects on vegetation structure of urban forest and contribute to the knowledge base on urban ecology. Our study has implications for facilitating evidence-based decision-making and the development of sustainable management strategies for urban forest ecosystems in Chandigarh.</p>
</section>
<section id="introduction" class="level2">
<h2 class="anchored" data-anchor-id="introduction">Introduction</h2>
<p>Urban forests are vital components of modern cities, providing numerous environmental, social, and economic benefits. These forests influence the well-being of city dwellers through multiple ecosystem services including climate regulation <span class="citation" data-cites="Gupta2019">(<a href="#ref-Gupta2019" role="doc-biblioref">Gupta, Mathew, and Khandelwal 2019</a>)</span>, run-off regulation, water supply, air purification and habitat provisions <span class="citation" data-cites="Berglihn2021 Escobedo2011">(<a href="#ref-Berglihn2021" role="doc-biblioref">Berglihn and Gómez-Baggethun 2021</a>; <a href="#ref-Escobedo2011" role="doc-biblioref">Escobedo, Kroeger, and Wagner 2011</a>)</span>. The quality of ecosystem services ultimately depend on the biological diversity specifically the vegetation composition <span class="citation" data-cites="Elmqvist2013">(<a href="#ref-Elmqvist2013" role="doc-biblioref">Elmqvist et al. 2013</a>)</span>. Like any other ecosystem, urban forests are subjected to various man-made or natural disturbances that can substantially compromise the ecosystem services through their direct and indirect effects on vegetation composition <span class="citation" data-cites="Pickett2011">(<a href="#ref-Pickett2011" role="doc-biblioref">Pickett et al. 2011</a>)</span>. Any natural or anthropogenic factor which causes substantial change in some measurable ecosystem state variable is referred as ‘disturbance’ <span class="citation" data-cites="Jentsch2019 Wohlgemuth2022">(<a href="#ref-Jentsch2019" role="doc-biblioref">Jentsch and White 2019</a>; <a href="#ref-Wohlgemuth2022" role="doc-biblioref">Wohlgemuth, Jentsch, and Seidl 2022</a>)</span>. Thus, disturbances in urban forests can arise from both natural and human-induced factors. Natural disturbances include severe weather events such as storms, drought or flooding, insect outbreaks and diseases. Human-induced disturbances encompass activities such as urbanisation, infrastructure development, pollution, logging, fertilisation, use of pesticides and improper management practices <span class="citation" data-cites="Wohlgemuth2022">(<a href="#ref-Wohlgemuth2022" role="doc-biblioref">Wohlgemuth, Jentsch, and Seidl 2022</a>)</span>.</p>
<p>Disturbance events can lead to changes in vegetation composition and species diversity of urban forests through multiple mechanisms <span class="citation" data-cites="Laska2001">(<a href="#ref-Laska2001" role="doc-biblioref">Łaska 2001</a>)</span>. Disturbance can regulate the vegetation composition of ecological communities through assembly rules via disturbance filters. In this context, the intermediate disturbance hypothesis states that biodiversity peaks at intermediate levels of disturbance because species adapted to both higher and lower disturbance intensity can coexist at intermediate levels of disturbance <span class="citation" data-cites="Connell1978">(<a href="#ref-Connell1978" role="doc-biblioref">Connell 1978</a>)</span>. However, the effects of disturbance on species diversity may depend on productivity because productivity and disturbance are negatively related to competition <span class="citation" data-cites="Huston1994">(<a href="#ref-Huston1994" role="doc-biblioref">Huston 1994</a>)</span>. Alternatively, the mass-ratio hypothesis states that the dominant species regulates the species diversity through its effects on productivity <span class="citation" data-cites="Grime1998">(<a href="#ref-Grime1998" role="doc-biblioref">Grime 1998</a>)</span>. Further, disturbances can initiate successional processes by creating gaps in the canopy or altering resource availability and therefore, leading to changes in species composition <span class="citation" data-cites="Denslow1980">(<a href="#ref-Denslow1980" role="doc-biblioref">Denslow 1980</a>)</span>. Non-native invasive species can occupy and dominate these disturbed habitats leading to a decline in native species diversity <span class="citation" data-cites="Aryal2022">(<a href="#ref-Aryal2022" role="doc-biblioref">Aryal et al. 2022</a>)</span>. Apart from direct disturbance agents, ongoing environmental changes together with expanding urban populations are also projected to indirectly impose severe pressure on structure and functioning of urban forests. Available literature indicates mixed support for either of these hypotheses <span class="citation" data-cites="Kershaw2013">(<a href="#ref-Kershaw2013" role="doc-biblioref">Kershaw and Mallik 2013</a>)</span>. However, there is growing evidence that the disturbance can substantially alter the vegetation structure <span class="citation" data-cites="Wohlgemuth2022">(<a href="#ref-Wohlgemuth2022" role="doc-biblioref">Wohlgemuth, Jentsch, and Seidl 2022</a>)</span>. The altered ecosystem structure of urban forests have substantial implications as it can affect the city residents through compromised ecosystem services. Therefore, monitoring and understanding the impacts of disturbance on urban forest structure are crucial for effective management, mitigation of risks, and conservation efforts.</p>
<p>The city Chandigarh is renowned for its meticulously planned urban landscape and abundant green spaces. Its urban forests, comprising parks, gardens, and extensive green belts, contribute significantly to the city’s ecological well-being and serve as important reservoirs of biodiversity. However, rapid population growth, human activities, and natural events pose potential threats to the vegetation structure of Chandigarh’s urban forests. Further, the lake catchment area also suffers from natural disturbance due to heavy soil erosion during rainy season <span class="citation" data-cites="Singh2002">(<a href="#ref-Singh2002" role="doc-biblioref">Y. Singh 2002</a>)</span>. Ongoing environmental change is expected to increase the frequency of disturbance events by altering the seasonality and intensity of rainfall in the Siwalik region. Thus, the the urban forests of Chandigarh are susceptible to various disturbances that may threaten their integrity and ecological functions. Therefore, detecting and monitoring disturbances in urban forests is essential for the effective management and preservation of these valuable green areas. The objective of present study is to detect the disturbances in urban forests using remote sensing data. Further, this study aims to analyse the vegetation structure in terms of species diversity and vegetation composition in the disturbance affected areas. The ultimate goal is to provide decision-makers, urban planners, and forest management agencies with valuable information to mitigate risks, prioritise conservation efforts, and ensure the long-term sustainability of Chandigarh’s urban forests.</p>
</section>
<section id="methodology" class="level2">
<h2 class="anchored" data-anchor-id="methodology">Methodology</h2>
<section id="study-site" class="level3">
<h3 class="anchored" data-anchor-id="study-site">Study site</h3>
<p>The present study was conducted in the lake reserve forest of the Sukhna lake catchment area (<a href="#fig-site-map">Figure&nbsp;1</a>). Sukhna lake is a man made reservoir situated in union territory of Chandigarh at Siwalik foothills of lower Himalayas <span class="citation" data-cites="Singh2001">(<a href="#ref-Singh2001" role="doc-biblioref">Y. Singh 2001</a>)</span>. The lake has been declared as a national protective wetland by government of India. The geographical area of Sukhna lake is about 3 km<sup>2</sup> with an average elevation of 350 m above mean sea level (<a href="#fig-site-map">Figure&nbsp;1</a>). The lake reserve forest spreads over an area of 1.5 km<sup>2</sup> lies northern side of the lake, harbour several grazers i.e.&nbsp;Sambhar, Asian antelope and several species of reptiles and birds. The vegetation is similar to Siwalik hills comprising of dry mixed deciduous forests with occasional bamboo brakes <span class="citation" data-cites="Kumar2022">(<a href="#ref-Kumar2022" role="doc-biblioref">Kumar et al. 2022</a>)</span>. Climatic conditions are subtropical monsoon type with annual rainfall of 1110 mm, with maximum temperature of 44°C and minimum 37°C in summer and 14°C during winter as maximum and 3°C as minimum. Soils are alluvial, sandy loam highly susceptible to soil erosion by water run-off action <span class="citation" data-cites="Singh2002">(<a href="#ref-Singh2002" role="doc-biblioref">Y. Singh 2002</a>)</span>.</p>
<div id="fig-site-map" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figs/fig1.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;1: Map showing the geographical location of Sukhna Lake and Lake Reserve Forest in Chandigarh. The sampling plots are indicated as red squares. The inset map shows the location of study sites in Chandigarh city.</figcaption>
</figure>
</div>
</section>
<section id="satellite-data-and-pre-processing" class="level3">
<h3 class="anchored" data-anchor-id="satellite-data-and-pre-processing">Satellite data and pre-processing</h3>
<p>Since the normalized difference vegetation index (NDVI) has been regarded as a good proxy for vegetation health and biomass, we selected NDVI to assess forest disturbance. The disturbance induced changes in biomass should be reflected as variations in NDVI values. We downloaded all available NDVI values from 2001 to 2021 from MOD13Q1 (version 061) data product using the <code>R</code> package <code>MODISTools</code> version 1.1.4 <span class="citation" data-cites="R-MODISTools">(<a href="#ref-R-MODISTools" role="doc-biblioref">Hufkens 2023</a>)</span>.</p>
<p>The NDVI values of this product are are derived from atmospherically-corrected reflectance in the red, near infra-red, and blue wavebands from the Moderate Resolution Imaging Spectroradiometer (MODIS) instrument mounted on Terra satellite. This data product provides the NDVI values at a spatial resolution of 250 m generated for every 16 days as a Level 3 product <span class="citation" data-cites="MODIS">(<a href="#ref-MODIS" role="doc-biblioref">Didan 2021</a>)</span>. The algorithm chooses the best available pixel value from all the acquisitions from the 16 day period based on clouds cover, view angle, and the reflectance value. We also downloaded the associated pixel reliability layer to control for pixel values quality. This layer contains simplified ranking of the data that describes overall pixel quality and the good quality data is represented by zero rank <span class="citation" data-cites="MODIS">(<a href="#ref-MODIS" role="doc-biblioref">Didan 2021</a>)</span>.</p>
<p>We acquired a total of 483 images each for NDVI values and associated pixel reliability values. Clear observations with no sub pixel cloud, low aerosol, and low sensor view angle (&lt;30 degrees) are considered as high quality observations. We selected only reliable good quality pixels by masking out the pixels with no data, marginal data, snow/ice or clouds <span class="citation" data-cites="Didan2015">(see Table 4 in <a href="#ref-Didan2015" role="doc-biblioref">Didan et al. 2015</a>)</span>. This data processing step introduced gaps in our initial NDVI time series and about 6.7% of total pixels had missing values. The missing values were imputed using the <code>na.interp()</code> function from the <code>forecast</code> package version 8.21 <span class="citation" data-cites="R-forecast">(<a href="#ref-R-forecast" role="doc-biblioref">Hyndman and Khandakar 2008</a>)</span>. This approach first calculates a robust STL-decomposition <span class="citation" data-cites="Cleveland1990">(<a href="#ref-Cleveland1990" role="doc-biblioref">Cleveland et al. 1990</a>)</span> followed by a linear interpolation of the seasonally adjusted data, and then, the seasonal component is added back. Next, we used the Savitzky-Golay filter <span class="citation" data-cites="Savitzky1964">(<a href="#ref-Savitzky1964" role="doc-biblioref">Savitzky and Golay 1964</a>)</span> for smoothing the NDVI time series. This filter was applied using the <code>R</code> package <code>signal</code> version 0.7.7 <span class="citation" data-cites="R-signal">(<a href="#ref-R-signal" role="doc-biblioref">Ligges et al. 2022</a>)</span>. Thus, we created a processed NDVI time series for all pixels in the initial dataset.</p>
</section>
<section id="detection-of-disturbance" class="level3">
<h3 class="anchored" data-anchor-id="detection-of-disturbance">Detection of disturbance</h3>
<p>To assess the forest disturbance, we implemented the Breaks for Additive Seasonal Trends (BFAST) algorithm using the function <code>bfastmonitor()</code> function from <code>R</code> package <code>bfast</code> version 1.6.1 <span class="citation" data-cites="R-bfast Verbesselt2010a Verbesselt2010b">(<a href="#ref-R-bfast" role="doc-biblioref">Verbesselt et al. 2021</a>; <a href="#ref-Verbesselt2010a" role="doc-biblioref">Verbesselt, Hyndman, Newnham, et al. 2010</a>; <a href="#ref-Verbesselt2010b" role="doc-biblioref">Verbesselt, Hyndman, Zeileis, et al. 2010</a>)</span>. Basically, this algorithm detects structural changes in both the trend and seasonal components of a STL-decomposed time series. We used the season-trend model to detect the changes in NDVI time series as described earlier <span class="citation" data-cites="Verbesselt2012">(<a href="#ref-Verbesselt2012" role="doc-biblioref">Verbesselt, Zeileis, and Herold 2012</a>)</span>. This model assumes a linear trend and harmonic season for <span class="math inline">\(y_t\)</span> observations at time <span class="math inline">\(t\)</span>.</p>
<p><span id="eq-stm"><span class="math display">\[y_t = \alpha_1 + \alpha_2~t + \sum_{j = 1}^k \gamma_j~ \sin \left( \frac{2 \pi j t} {f} + \delta_j \right) + \epsilon_t \tag{1}\]</span></span></p>
<p>where the intercept <span class="math inline">\(\alpha_1\)</span>, slope <span class="math inline">\(\alpha_2\)</span> (i.e., trend), amplitudes <span class="math inline">\(\gamma_1 \ldots \gamma_k\)</span>, and phases <span class="math inline">\(\delta_1 \ldots \delta_k\)</span> (season) are the unknown parameters, <span class="math inline">\(f\)</span> is the known frequency (e.g., <span class="math inline">\(f\)</span> = 23 annual observations for a 16-day time series), and <span class="math inline">\(\epsilon_t\)</span> is the unobservable error term at time <span class="math inline">\(t\)</span> (with standard deviation <span class="math inline">\(\sigma\)</span>). To robustly detect disturbances, we used three harmonic terms (i.e., <span class="math inline">\(k\)</span> = 3) to represent a four-month cycle of NDVI <span class="citation" data-cites="Verbesselt2012">(<a href="#ref-Verbesselt2012" role="doc-biblioref">Verbesselt, Zeileis, and Herold 2012</a>)</span>. The unknown seasonal and trend parameters can be estimated and tested using ordinary least squares (OLS) method <span class="citation" data-cites="Cryer2008">(<a href="#ref-Cryer2008" role="doc-biblioref">Cryer and Chan 2008</a>)</span>.</p>
<p>If there is no disturbance, then this season-trend model (<a href="#eq-stm">Equation&nbsp;1</a>) should remain stable for newly added observations. In other words, a disturbance will be detected when the model will no longer remain stable. The stability of model can be estimated using the MOving SUM (MOSUM) test of the OLS residuals (<span class="math inline">\(\hat \epsilon_t\)</span>) in the monitoring period <span class="math inline">\(t = n + 1, \ldots, N\)</span></p>
<p><span id="eq-mosum"><span class="math display">\[MO_t = \frac{1}{\hat \sigma \sqrt{n}} \sum_{s = t - h + 1}^t \left(y_s - x_s^ \top \hat \beta \right) \tag{2}\]</span></span></p>
<p>where <span class="math inline">\(h\)</span> is the bandwidth of the MOSUM and is typically chosen relative to the size of the history sample (e.g., <span class="math inline">\(h = n/4\)</span>). The OLS-MOSUM test (<a href="#eq-mosum">Equation&nbsp;2</a>) and least squares are used to determine the number and position of the breakpoints in the seasonally adjusted data (<span class="math inline">\(Y_t-S_t\)</span>). The breakpoints are declared if the absolute <span class="math inline">\(|MO_t|\)</span> values fall beyond the 5% probability of structural model stability <span class="citation" data-cites="Verbesselt2012">(<a href="#ref-Verbesselt2012" role="doc-biblioref">Verbesselt, Zeileis, and Herold 2012</a>)</span>. On the other hand, the <span class="math inline">\(|MO_t|\)</span> should be close to zero with random fluctuations for a structurally stable model <span class="citation" data-cites="Verbesselt2012">(<a href="#ref-Verbesselt2012" role="doc-biblioref">Verbesselt, Zeileis, and Herold 2012</a>)</span>.</p>
<p>Firstly, we detected changes at the end of NDVI time series using the function <code>bfastmonitor()</code> <span class="citation" data-cites="Verbesselt2012">(<a href="#ref-Verbesselt2012" role="doc-biblioref">Verbesselt, Zeileis, and Herold 2012</a>)</span>. This approach first computes the trend, season and other covariates using the STL-decomposition. Then, the data is divided into history and monitoring period. We defined the January 2018 as the start period so that the <em>history period</em> is selected from 2001 to 2018 and <em>monitoring period</em> was defined as 2018 onwards for the present study. The stable history was selected based on a reversed-ordered-cumulative sum (CUSUM) of residuals, or ROC <span class="citation" data-cites="Pesaran2002">(<a href="#ref-Pesaran2002" role="doc-biblioref">Pesaran and Timmermann 2002</a>)</span>. A season-trend model was fitted to this selected stable history period and then, the OLS-MOSUM test was used to detect changes in the monitoring period. If a breakpoint is detected, the timing and magnitude of breakpoint was estimated (<a href="#fig-bfmpx">Figure&nbsp;2</a>). We used the Wilcoxon’s rank sum test (also known as Mann-Whitney U test) to compare the disturbance parameters between forest and lake components. This test compares the difference in medians with relaxed assumption of normality of data. We used the <code>rstatix</code> package (version 0.7.2) to compute the test statistics and associated p-values <span class="citation" data-cites="R-rstatix">(<a href="#ref-R-rstatix" role="doc-biblioref">Kassambara 2022</a>)</span>.</p>
<div id="fig-bfmpx" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figs/fig2.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;2: Disturbance monitoring for a single location (76.81912<span class="math inline">\(^\circ\)</span>E, 30.74081<span class="math inline">\(^\circ\)</span>N) in lake reserve forest. A break of 0.03 NDVI units was detected around 16 January 2020.</figcaption>
</figure>
</div>
<p>However, this monitoring approach aims to detect breaks at the end of NDVI time series. Further, it assumes a disturbance free stable history period, which often not a case with real data. Considering these limitations, we focused on a major trend shifts within the NDVI time series. We tested for a single large break using the function <code>bfast01()</code> <span class="citation" data-cites="Jong2013">(<a href="#ref-Jong2013" role="doc-biblioref">Jong et al. 2013</a>)</span>. This approach also uses the similar season-trend model and the single breakpoint was estimated by minimising the segmented residual sum of squares. It identifies a single largest break (if MOSUM is significant) for duration 2001-2021 or no break (i.e., MOSUM is not significant). We used a minimum segment size of 69 observations (3 years) to appropriately estimate the regression parameters (2 trend and 6 season parameters) of the season-trend model (<a href="#fig-bf01px">Figure&nbsp;3</a>). These identified breaks or turning points were then characterised using the function <code>bfast01classify()</code> and categorised into different types of ecological trend shifts as described earlier <span class="citation" data-cites="Bernardino2020">(<a href="#ref-Bernardino2020" role="doc-biblioref">Bernardino et al. 2020</a>)</span>.</p>
<div id="fig-bf01px" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figs/fig3.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;3: Detection of single largest break for location (76.81912<span class="math inline">\(^\circ\)</span>E, 30.74081<span class="math inline">\(^\circ\)</span>N) in lake reserve forest. The single largest break was detected around 16 January 2018. The trend shift suggests a monotonic increase with a positive break.</figcaption>
</figure>
</div>
</section>
<section id="field-sampling" class="level3">
<h3 class="anchored" data-anchor-id="field-sampling">Field sampling</h3>
<p>To study vegetation composition and species diversity, we pre-defined 14 locations along the lake catchment in the lake reserve forest (<a href="#fig-site-map">Figure&nbsp;1</a>). This prior selection of sampling plot allowed us to avoid any preferential or selection bias while sampling vegetation. We sampled plant species and their abundances using the quadrat method during the flowering season (August-September). We laid a 10 m <span class="math inline">\(\times\)</span> 10 m quadrat at each selected location and recorded the abundance of all woody plants. Herbaceous species were sampled in three randomly laid 1 m <span class="math inline">\(\times\)</span> 1 m quadrats within each 10 m <span class="math inline">\(\times\)</span> 10 m quadrat. A total of 50 quadrats, 14 quadrats for trees and 36 quadrats for herbaceous species were sampled, since two 10 m <span class="math inline">\(\times\)</span> 10 m quadrats had no herbaceous species. The geographic location of each plot was recorded with a handheld GPS (Garmin eTrex 10). Most of the plant species were identified in field and few unidentified plant specimens were collected and photographed for further identification. For identification of plant specimens, we consulted the Herbarium of Panjab University, Chandigarh (PAN) and on-line database <a href="https://efloraofindia.com/">eFlora of India</a> <span class="citation" data-cites="eFI2022">(<a href="#ref-eFI2022" role="doc-biblioref">eFI 2022</a>)</span>. All the botanical names and their authorities were standardised according to World Checklist of Vascular Plants (WCVP) data version 10 of 27 October 2022 <span class="citation" data-cites="Govaerts2021">(<a href="#ref-Govaerts2021" role="doc-biblioref">Govaerts et al. 2021</a>)</span> using the <code>R</code> package <code>rWCVP</code> version 1.2.4 <span class="citation" data-cites="R-rWCVP">(<a href="#ref-R-rWCVP" role="doc-biblioref">Brown et al. 2023</a>)</span>. Further, all species were broadly categorised into native, introduced and invasive species following the ‘Plants of World Online’ <span class="citation" data-cites="POWO2023">(<a href="#ref-POWO2023" role="doc-biblioref">POWO 2023</a>)</span> and published alien flora of India <span class="citation" data-cites="Khuroo2012">(<a href="#ref-Khuroo2012" role="doc-biblioref">Khuroo et al. 2012</a>)</span>.</p>
</section>
<section id="phylogenetic-information" class="level3">
<h3 class="anchored" data-anchor-id="phylogenetic-information">Phylogenetic information</h3>
<p>To analyse the phylogenetic relatedness among the species, we used the recently published GBOTB backbone of phylogeny for seed plants, which is based on data from <a href="https://www.ncbi.nlm.nih.gov/genbank/">GenBank</a> and <a href="https://tree.opentreeoflife.org/opentree/argus/opentree13.4@ott93302">Open Tree of Life</a> <span class="citation" data-cites="Smith2018">(<a href="#ref-Smith2018" role="doc-biblioref">Smith and Brown 2018</a>)</span>. It is considered as the largest dated phylogeny for 79,881 taxa of seed plants resolved at species level. We used the package <code>V.PhyloMaker2</code> (version 0.1.0) to prepare a phylogenetic tree for the selected plant species <span class="citation" data-cites="R-VPhyloMaker2">(<a href="#ref-R-VPhyloMaker2" role="doc-biblioref">Jin and Qian 2022</a>)</span>. The phylogeny was generated from the <code>GBOTB.extended.LCVP</code> mega-tree, which is a combination of two published phylogenies for plants <span class="citation" data-cites="Smith2018 Zanne2014">(<a href="#ref-Smith2018" role="doc-biblioref">Smith and Brown 2018</a>; <a href="#ref-Zanne2014" role="doc-biblioref">Zanne et al. 2014</a>)</span>. Plant names in this phylogenetic tree are standardised according to the Leipzig Catalogue of Vascular Plants (LCVP) <span class="citation" data-cites="R-lcvplants">(<a href="#ref-R-lcvplants" role="doc-biblioref">Freiberg et al. 2020</a>)</span>, which closely resemble to WCVP <span class="citation" data-cites="Govaerts2021">(<a href="#ref-Govaerts2021" role="doc-biblioref">Govaerts et al. 2021</a>)</span>. The nodes were built using the <code>nodes.info.1.LCVP</code> function, which returns the most recent common ancestor of all the tips in the largest cluster of the genus, and define it as the basal node of the genus. Further, the phylogeny was generated under the <em>Scenario 3</em> for binding of new genus tip to the family branch as defined in earlier study <span class="citation" data-cites="Qian2016">(<a href="#ref-Qian2016" role="doc-biblioref">Qian and Jin 2015</a>)</span>.</p>
</section>
<section id="vegetation-analysis" class="level3">
<h3 class="anchored" data-anchor-id="vegetation-analysis">Vegetation analysis</h3>
<p>To assess the effects of disturbance, we divided the selected vegetation plots into disturbed and undisturbed plots based on the magnitude of NDVI change from <code>bfastmonitor()</code> output. If the magnitude of NDVI change was negative, the vegetation plots have been considered as <em>disturbed</em> plots (n = 8), whereas vegetation plots with a positive change in NDVI were considered as <em>undisturbed</em> plots (n = 6). Then, we compared the vegetation composition and species diversity in the disturbed and undisturbed plots. However, the efforts to measure and compare species diversity often suffer from sampling bias due undetected rare species. Therefore, equalising or standardisation of samples become necessary to measure and compare diversity in an unbiased manner. Traditionally, diversity samples have been standardised by equalising sampling effort, sample size and coverage. Equalising coverage is recommended because it considers both the sample diversity and true diversity <span class="citation" data-cites="Roswell2021">(<a href="#ref-Roswell2021" role="doc-biblioref">Roswell, Dushoff, and Winfree 2021</a>)</span>. Therefore, to examine sampling completeness of plant species across vegetation plots, we performed coverage-based rarefaction and extrapolation using the <code>R</code> package <code>iNEXT</code> version 3.0.0 <span class="citation" data-cites="R-iNEXT">(<a href="#ref-R-iNEXT" role="doc-biblioref">Hsieh, Ma, and Chao 2016</a>)</span>.</p>
<p>Traditional diversity metrics such as species richness (<span class="math inline">\(S\)</span>) do not incorporate species abundances, and those include species abundances, the Shannon’s index <span class="citation" data-cites="Shannon1948">(<a href="#ref-Shannon1948" role="doc-biblioref">Shannon 1948</a>)</span> and the Simpson’s index <span class="citation" data-cites="Simpson1949">(<a href="#ref-Simpson1949" role="doc-biblioref">Simpson 1949</a>)</span>, do not behave intuitively for subsequent addition or removal of species from a community <span class="citation" data-cites="Chao2014 Roswell2021">(<a href="#ref-Chao2014" role="doc-biblioref">Chao et al. 2014</a>; <a href="#ref-Roswell2021" role="doc-biblioref">Roswell, Dushoff, and Winfree 2021</a>)</span>. To overcome these issues, the use of Hill numbers are recommended <span class="citation" data-cites="Roswell2021">(<a href="#ref-Roswell2021" role="doc-biblioref">Roswell, Dushoff, and Winfree 2021</a>)</span>. Hill numbers are a series of diversity measures that combines species richness and species abundances <span class="citation" data-cites="Hill1973">(<a href="#ref-Hill1973" role="doc-biblioref">Hill 1973</a>)</span>. Hill number for order <span class="math inline">\(q\)</span> can be defined as:</p>
<p><span id="eq-hill"><span class="math display">\[
^qD = \left( \sum_{i = 1}^S p_i^q \right)^{1/(1-q)}
\tag{3}\]</span></span></p>
<p>where <span class="math inline">\(p_i\)</span> is the proportion or relative abundance of species <span class="math inline">\(i\)</span>, and <span class="math inline">\(S\)</span> is the number of species so that <span class="math inline">\(\sum_{i = 1}^S p_i = 1\)</span>. If <span class="math inline">\(n_i\)</span> is the total abundance of species <span class="math inline">\(i\)</span>, then the relative abundance can be calculated as <span class="math inline">\(p_i = n_i / \sum_{i = 1}^S n_i\)</span>. Many commonly used diversity measures are special cases of Hill numbers <span class="citation" data-cites="Chao2014">(<a href="#ref-Chao2014" role="doc-biblioref">Chao et al. 2014</a>)</span>. When <span class="math inline">\(q = 0\)</span>, <a href="#eq-hill">Equation&nbsp;3</a> did not considers the relative abundance of species and get reduced to species richness <span class="math inline">\(S\)</span> (<a href="#eq-richness">Equation&nbsp;4</a>). Although <a href="#eq-hill">Equation&nbsp;3</a> becomes undefined for <span class="math inline">\(q = 1\)</span>, its limit as <span class="math inline">\(q\)</span> tends to 1 represents the exponential of Shannon’s entropy (<a href="#eq-shannon">Equation&nbsp;5</a>). Similarly, when <span class="math inline">\(q = 2\)</span>, the <a href="#eq-hill">Equation&nbsp;3</a> represents the inverse of Simpson concentration (<a href="#eq-simpson">Equation&nbsp;6</a>). The logarithm of Hill numbers represent the corresponding Renyi entropies.</p>
<p><span id="eq-richness"><span class="math display">\[
^0D = S
\tag{4}\]</span></span></p>
<p><span id="eq-shannon"><span class="math display">\[
\lim_{q \to 1} ~ {^qD} = {^1D} =  \exp \left( - \sum_{i = 1}^S p_i \log p_i \right)
\tag{5}\]</span></span></p>
<p><span id="eq-simpson"><span class="math display">\[
^2D = 1 \Big/ \sum_{i = 1}^S p_i^2
\tag{6}\]</span></span></p>
<p>Since the Hill number of order <span class="math inline">\(q = 1\)</span> considers relative abundance of each species, <span class="math inline">\(^1D\)</span> can be interpreted as a measure of the effective number of common species in the assemblage. Similarly, <span class="math inline">\(^2D\)</span> can be interpreted as the effective number of dominant species because it weighs more on the abundant species <span class="citation" data-cites="Chao2014">(<a href="#ref-Chao2014" role="doc-biblioref">Chao et al. 2014</a>)</span>. We compared the first three measures of Hill numbers, i.e., species richness (<span class="math inline">\(q = 0\)</span>), Shannon diversity (<span class="math inline">\(q = 1\)</span>) and Simpson diversity (<span class="math inline">\(q = 2\)</span>) for disturbed and undisturbed plots using the <code>R</code> package <code>iNEXT</code> version 3.0.0 <span class="citation" data-cites="R-iNEXT">(<a href="#ref-R-iNEXT" role="doc-biblioref">Hsieh, Ma, and Chao 2016</a>)</span>. We estimated these diversity measures as the mean and 95% confidence intervals of 100 bootstrap replicates for a standardised sample size of 2000 individuals using the <code>iNEXT()</code> function. We determined significance of difference by analysing the overlap of the 95% confidence intervals. Whenever the 95% confidence intervals did not overlap, we considered significant difference in species diversity at p &lt; 0.05 <span class="citation" data-cites="Colwell2012">(<a href="#ref-Colwell2012" role="doc-biblioref">Colwell et al. 2012</a>)</span>.</p>
<p>Similarly, we compared phylogenetic diversity (PD) among the disturbed and undisturbed plots. We used the phylogenetic diversity measures of order <span class="math inline">\(q\)</span> based on the Hill number framework <span class="citation" data-cites="Chao2010">(<a href="#ref-Chao2010" role="doc-biblioref">Chao, Chiu, and Jost 2010</a>)</span>. If <span class="math inline">\(L_i\)</span> represents the length of branch <span class="math inline">\(i\)</span> and <span class="math inline">\(a_i\)</span> denotes the total relative abundance descended from the <span class="math inline">\(i\)</span>th branch for a phylogenetic tree with <span class="math inline">\(B\)</span> branch segments, the mean branch length (weighted by branch relative abundance) can be expressed as <span class="math inline">\(\bar T = \sum_{i = 1}^B L_i a_i\)</span>. <span class="citation" data-cites="Chao2010">Chao, Chiu, and Jost (<a href="#ref-Chao2010" role="doc-biblioref">2010</a>)</span> defined the phylogenetic diversity of order <span class="math inline">\(q\)</span> as:</p>
<p><span id="eq-hillpd"><span class="math display">\[
^q \text{PD} = \left[ \sum_{i = 1}^B L_i \left( \frac{a_i}{\bar T} \right)^q  \right]^{1/(1-q)}
\tag{7}\]</span></span></p>
<p>The <a href="#eq-hillpd">Equation&nbsp;7</a> becomes the Faith’s PD for <span class="math inline">\(q = 0\)</span>, which do not considers the actual total branch length and branch abundances. The measure <span class="math inline">\(^1 \text{PD}\)</span> represents a monotonic transformation of the phylogenetic entropy <span class="math inline">\(H_P\)</span>, which weighs each branch proportional to its abundance. The measure <span class="math inline">\(^2 \text{PD}\)</span> is a monotonic transformation of Rao’s quadratic entropy <span class="math inline">\(Q\)</span>, which puts more weight on abundant/dominant branches and discounts rare branches <span class="citation" data-cites="Chao2010 Hsieh2017">(<a href="#ref-Chao2010" role="doc-biblioref">Chao, Chiu, and Jost 2010</a>; <a href="#ref-Hsieh2017" role="doc-biblioref">Hsieh and Chao 2017</a>)</span>. The rarefaction and extrapolation methods were used to fairly compare these phylogenetic diversity measures across the plots <span class="citation" data-cites="Chao2015 Hsieh2017">(<a href="#ref-Chao2015" role="doc-biblioref">Chao et al. 2015</a>; <a href="#ref-Hsieh2017" role="doc-biblioref">Hsieh and Chao 2017</a>)</span>. We computed these metrics using the package <code>iNextPD</code> version 0.3.1 <span class="citation" data-cites="R-iNextPD">(<a href="#ref-R-iNextPD" role="doc-biblioref">Hsieh and Chao 2016</a>)</span>.</p>
<p>Further, we used the Non-metric Multidimensional Scaling (NMDS) to find the similarities and patterns among the species assemblages. NMDS is considered as the most robust unconstrained ordination method in vegetation ecology <span class="citation" data-cites="Minchin1987">(<a href="#ref-Minchin1987" role="doc-biblioref">Minchin 1987</a>)</span>. The species abundance data was square-root transformed followed by a Wisconsin double standardisation, which involves dividing each species by its maximum and each site by its total. This standardisation minimises the influence of most abundant groups and therefore, prevents overwhelming distance measures of the dominant species. We used the widely recommended Bray-Curtis distance since it has a good rank order relation to ordering sites along gradients <span class="citation" data-cites="Faith1987">(<a href="#ref-Faith1987" role="doc-biblioref">Faith, Minchin, and Belbin 1987</a>)</span>. The goodness of fit was assessed by the degree of stress, which is a measure of rank-order disagreement between observed and fitted values <span class="citation" data-cites="Kruskal1964">(<a href="#ref-Kruskal1964" role="doc-biblioref">Kruskal 1964</a>)</span>. If <span class="math inline">\(d_{ij}\)</span> represents the distance in original dissimilarity matrix and <span class="math inline">\(\hat{d_{ij}}\)</span> represent the fitted distance in ordination space, then the stress (S) can be expressed as <a href="#eq-stress">Equation&nbsp;8</a>:</p>
<p><span id="eq-stress"><span class="math display">\[
S = \sqrt{\frac{\sum_{i = 1}^{n - 1} \sum_{j = i + 1}^{n} (d_{ij} - \hat{d}_{ij})^2}{\sum_{i = 1}^{n - 1} \sum_{j = i + 1}^{n} d_{ij}^2}}
\tag{8}\]</span></span></p>
<p>A stress value of greater than 0.2 is generally considered as poor fit leading to risk in interpretation of distances <span class="citation" data-cites="Clarke1993">(<a href="#ref-Clarke1993" role="doc-biblioref">Clarke 1993</a>)</span>. The NMDS was performed with two dimensions from minimum 20 to maximum 50 random iterations using the function<code>metaMDS()</code> from the <code>vegan</code> package version 2.6.4 <span class="citation" data-cites="R-vegan">(<a href="#ref-R-vegan" role="doc-biblioref">Oksanen et al. 2022</a>)</span>. To test the differences in the multivariate centroids of the samples, we used the permutational multivariate analyses of variance (perMANOVA) on the dissimilarity matrix with 9,999 permutations using the <code>adonis2()</code> function from the <code>vegan</code> package. Across all the computed p-values, we considered p &lt; 0.05 as significant. All statistical analyses were performed in <code>R</code> statistical environment version 4.3.0 <span class="citation" data-cites="R-base">(<a href="#ref-R-base" role="doc-biblioref">R Core Team 2023</a>)</span> and package <code>tidyverse</code> was used for general data manipulation and visualisation <span class="citation" data-cites="R-tidyverse">(<a href="#ref-R-tidyverse" role="doc-biblioref">Wickham et al. 2019</a>)</span></p>
</section>
</section>
<section id="results" class="level2">
<h2 class="anchored" data-anchor-id="results">Results</h2>
<p>The bounds of our study area consisted of total 220 pixels, out of which 58 pixels had their centres within the boundary of lake reserve forest and 52 pixels within the Sukhna lake. The median values of NDVI showed seasonal variations for both forest and lake areas (<a href="#fig-monthly-NDVI">Figure&nbsp;4</a>). In general, the median NDVI values for forest were higher during July to September reaching maximum in August and lower during April to June reaching minimum in May.</p>
<div id="fig-monthly-NDVI" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figs/fig4.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;4: Monthly median NDVI values during 2001-2021 for the Sukhna lake and lake reserve forest indicating seasonality in the vegetation and water components.</figcaption>
</figure>
</div>
<section id="disturbance-pattern" class="level3">
<h3 class="anchored" data-anchor-id="disturbance-pattern">Disturbance pattern</h3>
<p>During 2001 to 2021, the median NDVI value for the forest was 0.5 reaching a maximum of 0.65. The Sukhna lake had an median NDVI of 0.17 reaching a minimum of –0.19 (<a href="#fig-bfm">Figure&nbsp;5</a> A). A dense patch of vegetation is present in the eastern part of the lake reserve forest. <a href="#fig-bfm">Figure&nbsp;5</a> B shows the length of stable history selected based on ROC to detect changes in the monitoring period. A shorter length of stable history indicates more recent disturbance. The timing and magnitude of breaks detected in NDVI time series are shown in <a href="#fig-bfm">Figure&nbsp;5</a> C and D.</p>
<div id="fig-bfm" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figs/fig5.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;5: Results of NDVI time series analysis during 2001-2021 for study area. (A) Median NDVI for all MODIS images, (B) Length of stable history period in years, (C) Timing of break in decimal years, and (D) Magnitude of changes in NDVI units.</figcaption>
</figure>
</div>
<p>The results of Wilcoxon’s rank sum test suggested that the forest area has significantly higher median NDVI (<em>W</em> = 3225, <em>p</em> &lt; 0.0001), longer stable history period (<em>W</em> = 2814, <em>p</em> &lt; 0.0001), more recent timings of breaks during 2019-2021 (<em>W</em> = 2592, <em>p</em> &lt; 0.0001), and more positive magnitude of NDVI change (<em>W</em> = 3122, <em>p</em> &lt; 0.0001) than the corresponding lake area (<a href="#fig-bfm-fl">Figure&nbsp;6</a>).</p>
<div id="fig-bfm-fl" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figs/fig6.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;6: Comparison of Median NDVI, Length of stable history, Timing of break, and Magnitude of change in NDVI time series analysis for Sukhna lake (n = 53) and lake reserve forest (n = 61) during 2001-2021. The significance of difference was determined with Wilcoxon’s rank sum test and <em>p</em>-values &lt;0.0001, &lt;0.001, &lt;0.01, &lt;0.05 and &gt;0.05 corresponds to <code>****</code>, <code>***</code>, <code>**</code>, <code>*</code>, and <code>ns</code>, respectively.</figcaption>
</figure>
</div>
<p>The detected single largest shifts in trend component of NDVI were characterised following the recently proposed typology of trend shifts <span class="citation" data-cites="Bernardino2020 R-bfast">(<a href="#ref-Bernardino2020" role="doc-biblioref">Bernardino et al. 2020</a>; <a href="#ref-R-bfast" role="doc-biblioref">Verbesselt et al. 2021</a>)</span>. The proposed eight shift types corresponds to (1) monotonic increase, (2) monotonic decrease, (3) monotonic increase (with positive break), (4) monotonic decrease (with negative break), (5) interruption: increase with negative break, (6) interruption: decrease with positive break, (7) reversal: increase to decrease, and (8) reversal: decrease to increase <span class="citation" data-cites="Bernardino2020 R-bfast">(<a href="#ref-Bernardino2020" role="doc-biblioref">Bernardino et al. 2020</a>; <a href="#ref-R-bfast" role="doc-biblioref">Verbesselt et al. 2021</a>)</span>. Out of proposed eight types, our study area showed five types of trend shifts in NDVI time series (<a href="#fig-bf01-bty">Figure&nbsp;7</a>).</p>
<div id="fig-bf01-bty" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figs/fig7.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;7: Examples of major trend shifts in NDVI time series observed for present study area <span class="citation" data-cites="Bernardino2020">(<a href="#ref-Bernardino2020" role="doc-biblioref">Bernardino et al. 2020</a>)</span>.</figcaption>
</figure>
</div>
<p>Of the total analysed 218 pixels, about 57% pixels (n = 124) indicated an interrupted increase in NDVI with a negative break and about 17% pixels (n = 38) indicated monotonic increase with positive break whereas about 4% pixels (n = 8) showed consistent monotonic increase with little or no change in NDVI trend during 2001-2021. Further, about 9% pixels (n = 20) indicated a negative reversal from increase to decrease and about 13% pixels (n = 28) indicated positive reversal from decrease to increase in NDVI (<a href="#fig-bf01">Figure&nbsp;8</a> A).</p>
<div id="fig-bf01" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figs/fig8.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;8: (A) Map showing the spatial distribution of chracterised trend shift types in study area during 2001-2021. (B) Distribution of different shift types across pixels covering the lake reserve forest (n = 61) and Sukhna lake (n = 53). The shift type key corresponds to 1, monotonic increase; 2, monotonic decrease; 3, monotonic increase (with positive break); 4, monotonic decrease (with negative break); 5, interruption: increase with negative break; 6, interruption: decrease with positive break; 7, reversal: increase to decrease; 8, reversal: decrease to increase.</figcaption>
</figure>
</div>
<p>About 85% pixels (n = 52) covering the forest area depicted a positive trend in NDVI. Out of these 85% pixels, about 8% pixels (n = 5) indicated no break, about 29.5% (n = 18) indicated positive break and 47.5% pixels (n = 29) indicated a negative break in the NDVI time series (<a href="#fig-bf01">Figure&nbsp;8</a> B). Out of total 61 pixels covering forest area, nine pixels showed a trend reversal in NDVI. Out of these nine pixels, two pixels indicated negative reversal from increasing to decreasing trend in NDVI. On the other hand, out of total 53 pixels covered by Sukhna lake about 64% pixels showed monotonic increase whereas 36% pixels indicated a trend reversal in NDVI. Among 19 pixels indicating a trend reversal, 14 pixels suggested a negative reversal from increasing to decreasing NDVI values whereas five pixels showed a positive reversal (<a href="#fig-bf01">Figure&nbsp;8</a> B).</p>
</section>
<section id="vegetation-structure" class="level3">
<h3 class="anchored" data-anchor-id="vegetation-structure">Vegetation structure</h3>
<p>During the field sampling, we recorded a total of 2,124 individuals representing 38 species of 34 genera belonging to 22 families of flowering plants. These 38 species consisted of 18 herbs, 13 trees, five shrubs and two climbers. Further, 17 species were native and 14 species were invasive to the region, whereas seven species were introduced in the region. Family Fabaceae was most dominant (n = 8) followed by Asteraceae (n = 4), Malvaceae (n = 4) and Poaceae (n = 3). Among all the species, about 50% of total species (n = 19) were present in both disturbed and undisturbed plots. However, the disturbed plots showed more number of unique species (n = 11) as compared to undisturbed plots (n = 8). Further, nine invasive species were common in both disturbed and undisturbed plots. Interestingly, undisturbed plots recorded greater number of unique invasive species (n = 4) than the disturbed plots (n = 1). Among all the plots, <em>Oplismenus burmanni</em> was the most common species, whereas <em>Ageratum houstonianum</em> was the most abundant invasive species.</p>
<div id="fig-diversity" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figs/fig9.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;9: Sample coverage-based rarefaction and extrapolation curves of species diversity (A) species richness (q = 0), (B) Shannon diversity (q = 1), and (C) Simpson diveristy (q = 2), and phylogenetic diversity (D) Faith’s PD (q = 0), (E) Phylogenetic entropy (q = 1), and (F) Rao’s Q (q = 2) based on first three orders of Hill numbers for the disturbed and undisturbed plots in the study region.</figcaption>
</figure>
</div>
<p>Both species diversity and phylogenetic diversity generally increased with sample coverage (<a href="#fig-diversity">Figure&nbsp;9</a>). However, the rarefaction curves for first three orders of Hill numbers usually overlapped for disturbed and undisturbed plots. Thus, little or no difference was observed in species diversity or phylogenetic diversity due to disturbance. Further, the curves disproportionately represented the rarefaction and extrapolation is achieved for only first order Hill number (Species richness and Faith’s PD). Such curves indicate that current samples of vegetation may be incomplete and diversity may differ with increased number of samples.</p>
<div class="cell">
<div class="cell-output-display">
<div id="tbl-estimates" class="anchored">

<div id="dcqscxvqag" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#dcqscxvqag table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#dcqscxvqag thead, #dcqscxvqag tbody, #dcqscxvqag tfoot, #dcqscxvqag tr, #dcqscxvqag td, #dcqscxvqag th {
  border-style: none;
}

#dcqscxvqag p {
  margin: 0;
  padding: 0;
}

#dcqscxvqag .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: 100%;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 0px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#dcqscxvqag .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#dcqscxvqag .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#dcqscxvqag .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#dcqscxvqag .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#dcqscxvqag .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#dcqscxvqag .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #000000;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#dcqscxvqag .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#dcqscxvqag .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#dcqscxvqag .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#dcqscxvqag .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#dcqscxvqag .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#dcqscxvqag .gt_spanner_row {
  border-bottom-style: hidden;
}

#dcqscxvqag .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#dcqscxvqag .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#dcqscxvqag .gt_from_md > :first-child {
  margin-top: 0;
}

#dcqscxvqag .gt_from_md > :last-child {
  margin-bottom: 0;
}

#dcqscxvqag .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#dcqscxvqag .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#dcqscxvqag .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#dcqscxvqag .gt_row_group_first td {
  border-top-width: 2px;
}

#dcqscxvqag .gt_row_group_first th {
  border-top-width: 2px;
}

#dcqscxvqag .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#dcqscxvqag .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#dcqscxvqag .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#dcqscxvqag .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#dcqscxvqag .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#dcqscxvqag .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#dcqscxvqag .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#dcqscxvqag .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#dcqscxvqag .gt_table_body {
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #000000;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #000000;
}

#dcqscxvqag .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#dcqscxvqag .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#dcqscxvqag .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#dcqscxvqag .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#dcqscxvqag .gt_left {
  text-align: left;
}

#dcqscxvqag .gt_center {
  text-align: center;
}

#dcqscxvqag .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#dcqscxvqag .gt_font_normal {
  font-weight: normal;
}

#dcqscxvqag .gt_font_bold {
  font-weight: bold;
}

#dcqscxvqag .gt_font_italic {
  font-style: italic;
}

#dcqscxvqag .gt_super {
  font-size: 65%;
}

#dcqscxvqag .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#dcqscxvqag .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#dcqscxvqag .gt_indent_1 {
  text-indent: 5px;
}

#dcqscxvqag .gt_indent_2 {
  text-indent: 10px;
}

#dcqscxvqag .gt_indent_3 {
  text-indent: 15px;
}

#dcqscxvqag .gt_indent_4 {
  text-indent: 20px;
}

#dcqscxvqag .gt_indent_5 {
  text-indent: 25px;
}
</style>
<table class="gt_table" data-quarto-disable-processing="false" data-quarto-bootstrap="false"><caption>Table&nbsp;1:  <p>Summary of observed and estimated values for species diversity and
phylogenetic diversity measures for disturbed (D) and undisturbed (U)
vegetation plots. The estimated values are represented as mean ± SE and
the 95% confidence intervals were estimated with 100 bootstrap
replications.</p> </caption>
  <thead>
    
    <tr class="gt_col_headings gt_spanner_row">
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="2" colspan="1" scope="col" id="Diversity">Diversity</th>
      <th class="gt_center gt_columns_top_border gt_column_spanner_outer" rowspan="1" colspan="2" scope="colgroup" id="Observed">
        <span class="gt_column_spanner">Observed</span>
      </th>
      <th class="gt_center gt_columns_top_border gt_column_spanner_outer" rowspan="1" colspan="2" scope="colgroup" id="Estimated">
        <span class="gt_column_spanner">Estimated</span>
      </th>
      <th class="gt_center gt_columns_top_border gt_column_spanner_outer" rowspan="1" colspan="2" scope="colgroup" id="95% CI">
        <span class="gt_column_spanner">95% CI</span>
      </th>
    </tr>
    <tr class="gt_col_headings">
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="D">D</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="U">U</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="D">D</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="U">U</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="D">D</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="U">U</th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr><td headers="Diversity" class="gt_row gt_left">Species richness</td>
<td headers="Observed.d" class="gt_row gt_left">30</td>
<td headers="Observed.u" class="gt_row gt_left">27</td>
<td headers="Estimated.d" class="gt_row gt_left">40.66 ± 7.08</td>
<td headers="Estimated.u" class="gt_row gt_left">44.98 ± 11.3</td>
<td headers="95% CI.d" class="gt_row gt_left">[30, 54.54]</td>
<td headers="95% CI.u" class="gt_row gt_left">[27, 67.14]</td></tr>
    <tr><td headers="Diversity" class="gt_row gt_left">Shannon diversity</td>
<td headers="Observed.d" class="gt_row gt_left">6.77</td>
<td headers="Observed.u" class="gt_row gt_left">6.03</td>
<td headers="Estimated.d" class="gt_row gt_left">6.9 ± 0.33</td>
<td headers="Estimated.u" class="gt_row gt_left">6.14 ± 0.31</td>
<td headers="95% CI.d" class="gt_row gt_left">[6.25, 7.56]</td>
<td headers="95% CI.u" class="gt_row gt_left">[5.53, 6.76]</td></tr>
    <tr><td headers="Diversity" class="gt_row gt_left">Simpson diversity</td>
<td headers="Observed.d" class="gt_row gt_left">3.15</td>
<td headers="Observed.u" class="gt_row gt_left">3.12</td>
<td headers="Estimated.d" class="gt_row gt_left">3.15 ± 0.15</td>
<td headers="Estimated.u" class="gt_row gt_left">3.13 ± 0.15</td>
<td headers="95% CI.d" class="gt_row gt_left">[2.86, 3.44]</td>
<td headers="95% CI.u" class="gt_row gt_left">[2.84, 3.42]</td></tr>
    <tr><td headers="Diversity" class="gt_row gt_left">Faith's PD</td>
<td headers="Observed.d" class="gt_row gt_left">2265.69</td>
<td headers="Observed.u" class="gt_row gt_left">2184.26</td>
<td headers="Estimated.d" class="gt_row gt_left">3053.96 ± 466.17</td>
<td headers="Estimated.u" class="gt_row gt_left">3644.34 ± 792.19</td>
<td headers="95% CI.d" class="gt_row gt_left">[2265.69, 3967.63]</td>
<td headers="95% CI.u" class="gt_row gt_left">[2184.26, 5197.01]</td></tr>
    <tr><td headers="Diversity" class="gt_row gt_left">Phylogenetic entropy</td>
<td headers="Observed.d" class="gt_row gt_left">615.44</td>
<td headers="Observed.u" class="gt_row gt_left">641.3</td>
<td headers="Estimated.d" class="gt_row gt_left">622.07 ± 15.39</td>
<td headers="Estimated.u" class="gt_row gt_left">648.22 ± 18.04</td>
<td headers="95% CI.d" class="gt_row gt_left">[615.44, 652.24]</td>
<td headers="95% CI.u" class="gt_row gt_left">[641.3, 683.57]</td></tr>
    <tr><td headers="Diversity" class="gt_row gt_left">Rao's Q</td>
<td headers="Observed.d" class="gt_row gt_left">351.54</td>
<td headers="Observed.u" class="gt_row gt_left">390.81</td>
<td headers="Estimated.d" class="gt_row gt_left">352.08 ± 8.07</td>
<td headers="Estimated.u" class="gt_row gt_left">391.49 ± 9.28</td>
<td headers="95% CI.d" class="gt_row gt_left">[351.54, 367.89]</td>
<td headers="95% CI.u" class="gt_row gt_left">[390.81, 409.68]</td></tr>
  </tbody>
  
  
</table>
</div>
</div>
</div>
</div>
<p>The observed and estimated measures of species and phylogenetic diversity were found to be similar among disturbed and undisturbed plots (<a href="#tbl-estimates">Table&nbsp;1</a>). Despite non-significant differences, the disturbed plots exhibited higher values of species richness, Shannon diversity and Simpson diversity as compared to undisturbed plots (<a href="#fig-div-estimates">Figure&nbsp;10</a>). Further, the estimated values were substantially higher than the observed values of diversity metrics. On average, the observed species richness and Faith’s phylogenetic diversity was 26% lower in disturbed plots and 40% lower in undisturbed plots than the estimated values of species richness. However, the observed and estimated values of Simpson diversity and Rao’s Q values were almost identical for both disturbed and undisturbed plots. Among all measures of species and phylogenetic diversity, the 95% confidence intervals were non-overlapping only for Rao’s Q values, suggesting that there were significant differences in the Rao’s Q phylogenetic diversity for the disturbed and undisturbed plots (<a href="#tbl-estimates">Table&nbsp;1</a>). The undisturbed plots exhibited significantly higher value of Rao’Q than the disturbed plots.</p>
<div id="fig-div-estimates" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figs/fig10.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;10: Comparison of asymptotically estimated species and phylogenetic diversity metrics (A) species richness, (B) Shannon’s diversity, (C) Simpson’s diversity, (D) Faith’s phylogenetic diversity, (E) phylogenetic entropy and (F) Rao’s quadratic entropy for disturbed and undisturbed vegetation plots <span class="citation" data-cites="Chao2010 Chao2014">(<a href="#ref-Chao2010" role="doc-biblioref">Chao, Chiu, and Jost 2010</a>; <a href="#ref-Chao2014" role="doc-biblioref">Chao et al. 2014</a>)</span>. The points represent the estimated mean values, the error bars represent the associated standard error (SE) and the shaded region represent the 95% confidence intervals from 100 bootstrap replications.</figcaption>
</figure>
</div>
<p>Non-metric multidimensional scaling (NMDS) ordination analysis indicated intermixing of disturbed and undisturbed vegetation plots (<a href="#fig-nmds">Figure&nbsp;11</a>). The permutational multivariate analysis of variance (perMANOVA) showed that the disturbed and undisturbed plots did not significantly differ in vegetation composition (<em>F</em><sub>1,12</sub> = 0.91, <em>p</em> = 0.575). There was a good to fair fit of ordination as indicated by relatively low stress (stress = 0.11). Despite high similarity in vegetation composition, two disturbed plots indicated unique species composition. One disturbed plot is solely dominated by <em>Justicia adhatoda</em>, whereas other is dominated by <em>Dendrocalamus strictus</em> with few <em>Bougainvillea spectabilis</em>.</p>
<div id="fig-nmds" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figs/fig11.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;11: Non-metric multidimensional scaling (NMDS) for differences in species composition between the disturbed and undisturbed vegetation plots. Permutational multivariate analysis of variance (perMANOVA) with 9,999 permutations was used to test the differences between disturbed and undisturbed plots. The goodness-of-fit was assessed by stress value.</figcaption>
</figure>
</div>
</section>
</section>
<section id="discussion" class="level2">
<h2 class="anchored" data-anchor-id="discussion">Discussion</h2>
<p>The aims of this investigation were to detect the disturbance and analyse its effects on vegetation structure of urban forest ecosystem. Our analysis of remote sensing data has provided valuable insights about the extent and magnitude of disturbance in urban forest of Chandigarh. Consistent with our expectations, the spatial pattern and severity of disturbance varied for the Sukhna lake and lake reserve forest. Interestingly, the lake reserve forest showed an overall slight positive change in NDVI suggesting lower disturbance and stable vegetation health. The stable vegetation of lake reserve forest can be attributed to the plantation and management efforts of the forest and wildlife department of Chandigarh administration. However, the interface between forest and lake indicated low disturbance as indicated by small negative change in NDVI values. These negative changes indicate a decrease in vegetation density and productivity across the lake-forest interface. This reduction in vegetation can be attributed to various factors, including the dominance of invasive species, increased pollution levels, flooding and limited access to essential resources such as water and nutrients. Thus, multiple factors may contribute to an overall declining vegetation health at the lake margin. Therefore, there is a need to take substantial measures to maintain vegetation at lake-forest interface.</p>
<p>One notable finding was the overall declining trend in NDVI values observed over the Sukhna lake. Sukhna lake not only showed greater decrease in NDVI but also indicated high variation in magnitude of NDVI decrease. In context of Sukhna lake, this trend suggests a degradation of water quality. The negative changes in NDVI reflect the deterioration of the lake ecosystem. These changes can have adverse effects on water quality, biodiversity, and overall ecological balance <span class="citation" data-cites="CSE2012">(<a href="#ref-CSE2012" role="doc-biblioref">CSE 2012</a>)</span>. Since more than half of the Sukhna lake catchment is severely prone to soil erosion <span class="citation" data-cites="Semwal2017">(<a href="#ref-Semwal2017" role="doc-biblioref">Semwal, Khobragade, and Nainwal 2017</a>)</span>, sedimentation and siltation appear to be most prominent factor contributing to the declining water quality. Further, it has been suggested that the effects of siltation are more prominent than the urbanisation and climate change <span class="citation" data-cites="Singh2019">(<a href="#ref-Singh2019" role="doc-biblioref">D. K. Singh and Singh 2019</a>)</span>. Despite numerous efforts from the administration, siltation has remained a major problem during past decades <span class="citation" data-cites="Shrimali2001 Singh2002">(<a href="#ref-Shrimali2001" role="doc-biblioref">Shrimali, Aggarwal, and Samra 2001</a>; <a href="#ref-Singh2002" role="doc-biblioref">Y. Singh 2002</a>)</span>. Apart from siltation, the degraded water quality can be attributed to numerous factors including pollution, sedimentation or changes in water flow patterns. These findings align with previous studies highlighting the degradation of the Sukhna lake <span class="citation" data-cites="Maansi2021 Singh2019">(<a href="#ref-Maansi2021" role="doc-biblioref">Maansi, Jindal, and Wats 2021</a>; <a href="#ref-Singh2019" role="doc-biblioref">D. K. Singh and Singh 2019</a>)</span>.</p>
<p>Our study indicated that both lake reserve forest and Sukhna lake are experiencing substantial changes in ecosystem health during the past two decades <span class="citation" data-cites="Singh2002">(<a href="#ref-Singh2002" role="doc-biblioref">Y. Singh 2002</a>)</span>. These findings highlight the vulnerability of urban forests to anthropogenic influences and the need for effective management and conservation strategies. The multi-temporal remote sensing datasets are useful for monitoring the temporary and long-term disturbances <span class="citation" data-cites="Verbesselt2012">(<a href="#ref-Verbesselt2012" role="doc-biblioref">Verbesselt, Zeileis, and Herold 2012</a>)</span>. The integration of satellite imagery analysis and statistical algorithms are emerging as potential tools for monitoring changes in vegetation patterns due to climate change or invasive species encroachment. Such studies can provide valuable insights into the vegetation dynamics in urban forests due to disturbance events. However, the accuracy and reliability of disturbance detection need to be further enhanced by incorporating ground data and validation techniques. The remote sensing approach can prove to be a powerful tool for identification and mapping of various disturbances such as construction activities, pollution, invasive species, and climate change impacts. Such techniques can capture large-scale disturbances, enabling comprehensive assessments and informed decision-making for forest management.</p>
<p>Another objective of present study was to analyse the effects of disturbance events on vegetation structure of urban forests and therefore, we compared the taxonomic and phylogenetic diversity between the disturbed and undisturbed plots. Contrary to our expectations, we did not find any significant differences in taxonomic and phylogenetic diversity metrics for the disturbed vegetation plots as compared to undisturbed plots. The species richness and Simpson diversity was similar for the disturbed and undisturbed plots suggesting that disturbance had little influence on the species richness and Simpson diversity. However, the Shannon’s diversity was higher for disturbed plots than the undisturbed plots, though there was little statistical evidence for significant differences. The higher Shannon’s diversity in disturbed plots indicate that disturbance might have promoted the evenness of species. Further, disturbance may have selectively promoted some species while discouraging others. These findings are consistent with previous studies investigating the disturbance effects on vegetation structure <span class="citation" data-cites="Erfanian2021 Gogoi2018">(<a href="#ref-Erfanian2021" role="doc-biblioref">Erfanian, Alatalo, and Ejtehadi 2021</a>; <a href="#ref-Gogoi2018" role="doc-biblioref">Gogoi and Sahoo 2018</a>)</span>. However, multiple direct and indirect effects of disturbance can shape the biodiversity and vegetation structure <span class="citation" data-cites="Bowd2021 Roman2018">(<a href="#ref-Bowd2021" role="doc-biblioref">Bowd et al. 2021</a>; <a href="#ref-Roman2018" role="doc-biblioref">Roman et al. 2018</a>)</span>.</p>
<p>An important finding of the present study is the significantly lower phylogenetic Rao’s Quadratic Entropy (Rao’s Q) for disturbed plots as compared to the undisturbed plots (<a href="#fig-div-estimates">Figure&nbsp;10</a> F). The lower phylogenetic diversity of disturbed plots suggests the presence of closely related evolutionary lineages. On the other hand, the higher phylogenetic diversity of undisturbed plots suggests presence of species from diverse evolutionary lineages. It appears that disturbance events discourage the coexistence of species lineages that are more distantly-related in evolutionary history. Disturbance may promote phylogenetically similar species reducing the phylogenetic diversity of species within a community. This observation is complemented by lower levels of Faith’s phylogenetic diversity (PD) and phylogenetic entropy for disturbed plots. The lower levels of Faith’s PD for disturbed plots indicates the presence of species with shared evolutionary history. Similarly, the lower phylogenetic entropy for disturbed plots suggests species are clustered on the phylogenetic tree indicating dominance or specialisation of some specific lineages. These findings align with previous studies examining the disturbance effects on phylogenetic structure of vegetation <span class="citation" data-cites="Dinnage2009 Erfanian2021 Liendo2020">(<a href="#ref-Dinnage2009" role="doc-biblioref">Dinnage 2009</a>; <a href="#ref-Erfanian2021" role="doc-biblioref">Erfanian, Alatalo, and Ejtehadi 2021</a>; <a href="#ref-Liendo2020" role="doc-biblioref">Liendo et al. 2020</a>)</span>.</p>
<p>Our study suggest that the effects of disturbance are more prominent on phylogenetic structure of vegetation as compared to taxonomic diversity. While taxonomic diversity remained similar, the phylogenetic diversity appeared to be declined under the influence of disturbance. This reduction could be a result of indirect effects of disturbance through various factors, such as invasive species, climate change or altered community dynamics. Invasive species were found to have a considerable influence on the vegetation structure in the urban forests <span class="citation" data-cites="Aryal2022">(<a href="#ref-Aryal2022" role="doc-biblioref">Aryal et al. 2022</a>)</span>. The encroachment of invasive species, such as <em>Lantana camara</em> and <em>Prosopis juliflora</em>, might have disrupted the native plant communities, reducing the phylogenetic diversity and altering the composition of the forest under-story. The spread of invasive species can also affect ecosystem processes, such as nutrient cycling <span class="citation" data-cites="Patil2020">(<a href="#ref-Patil2020" role="doc-biblioref">Patil et al. 2020</a>)</span>, further impacting the overall vegetation structure and ecological functioning of the urban forests. Ongoing climate change is another important factor affecting the vegetation structure in urban forests. Specifically, hydrological disturbance due to erratic frequency and intensity of rainfall events might have impacted the tree growth, species composition, and overall forest structure <span class="citation" data-cites="Shafroth2002">(<a href="#ref-Shafroth2002" role="doc-biblioref">Shafroth, Stromberg, and Patten 2002</a>)</span>. Further, water logged conditions and increased susceptibility to diseases may be leading to decreased tree vitality and increased mortality rates in some areas.</p>
</section>
<section id="conclusion" class="level2">
<h2 class="anchored" data-anchor-id="conclusion">Conclusion</h2>
<p>In conclusion, our study provides the initial signals of disturbance and its effects on vegetation structure in urban forest of Chandigarh. Our findings indicate that the Sukhna lake and adjacent reserved forest are experiencing substantial changes in their hydrological and ecological conditions. These changes are more prominent for Sukhna lake and adjacent riparian forest zone, indicating prevalence of multiple disturbance agents. However, little changes were observed in species diversity and composition of forest vegetation. While the taxonomic diversity was similar, the phylogenetic diversity was substantially lower in disturbed plots as compared to undisturbed plots. These findings highlight the importance of monitoring the disturbance and its effects on the vegetation structure. Further, our study emphasises the integration of remote sensing technologies and ground surveys while monitoring and managing the urban forest ecosystems. Timely interventions, such as habitat restoration, invasive species management, and urban planning, can aid in preserving the integrity and functionality of the urban forest ecosystem. Finally, we call for a collaborative approach involving various stakeholders, including city planners, policy-makers, environmentalists, and local communities, to achieve sustainable management and conservation of urban forests.</p>
</section>


<section id="references" class="level2">




</section>

<div id="quarto-appendix" class="default"><section id="acknowledgments" class="level2 appendix"><h2 class="anchored quarto-appendix-heading">Acknowledgments</h2><div class="quarto-appendix-contents">

<p>Authors are thankful to Forest and Wildlife Department, Chandigarh for providing permission to conduct study in Sukhna lake catchment area.</p>
</div></section><section id="data-availability" class="level2 appendix"><h2 class="anchored quarto-appendix-heading">Data availability</h2><div class="quarto-appendix-contents">

<p>A reproducible preprint with all data and <code>R</code> codes used for analysis is currently available from <a href="https://github.com/kumar-a" class="uri">https://github.com/kumar-a</a>. We will be deposit our all data and code to suitable free repository after acceptance of article.</p>
</div></section><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" role="list">
<div id="ref-Aryal2022" class="csl-entry" role="listitem">
Aryal, Prakash Chandra, Chandramani Aryal, Kiran Bhusal, Devendra Chapagain, Man Kumar Dhamala, Sanu Raja Maharjan, and Parveen K. Chhetri. 2022. <span>“Forest Structure and Anthropogenic Disturbances Regulate Plant Invasion in Urban Forests.”</span> <em>Urban Ecosystems</em> 25 (2): 367–77. <a href="https://doi.org/10.1007/s11252-021-01159-7">https://doi.org/10.1007/s11252-021-01159-7</a>.
</div>
<div id="ref-Berglihn2021" class="csl-entry" role="listitem">
Berglihn, Elisabeth Cornelia, and Erik Gómez-Baggethun. 2021. <span>“Ecosystem Services from Urban Forests: The Case of Oslomarka, Norway.”</span> <em>Ecosystem Services</em> 51 (October): 101358. <a href="https://doi.org/10.1016/j.ecoser.2021.101358">https://doi.org/10.1016/j.ecoser.2021.101358</a>.
</div>
<div id="ref-Bernardino2020" class="csl-entry" role="listitem">
Bernardino, Paulo N., Wanda De Keersmaecker, Rasmus Fensholt, Jan Verbesselt, Ben Somers, and Stéphanie Horion. 2020. <span>“Global-Scale Characterization of Turning Points in Arid and Semi-Arid Ecosystem Functioning.”</span> <em>Global Ecology and Biogeography</em> 29 (7): 1230–45. <a href="https://doi.org/10.1111/geb.13099">https://doi.org/10.1111/geb.13099</a>.
</div>
<div id="ref-Bowd2021" class="csl-entry" role="listitem">
Bowd, Elle, Wade Blanchard, Lachlan McBurney, and David Lindenmayer. 2021. <span>“Direct and Indirect Disturbance Impacts on Forest Biodiversity.”</span> <em>Ecosphere</em> 12 (12): e03823. <a href="https://doi.org/10.1002/ecs2.3823">https://doi.org/10.1002/ecs2.3823</a>.
</div>
<div id="ref-R-rWCVP" class="csl-entry" role="listitem">
Brown, Matilda J. M., Barnaby E. Walker, Nicholas Black, Rafaël H. A. Govaerts, Ian Ondo, Robert Turner, and Eimear Nic Lughadha. 2023. <span>“<span class="nocase">rWCVP</span>: A Companion <span>R</span> Package for the World Checklist of Vascular Plants.”</span> <em>New Phytologist</em>, May. <a href="https://doi.org/10.1111/nph.18919">https://doi.org/10.1111/nph.18919</a>.
</div>
<div id="ref-Chao2015" class="csl-entry" role="listitem">
Chao, Anne, Chun-Huo Chiu, T. C. Hsieh, Thomas Davis, David A. Nipperess, and Daniel P. Faith. 2015. <span>“Rarefaction and Extrapolation of Phylogenetic Diversity.”</span> <em>Methods in Ecology and Evolution</em> 6 (4): 380–88. <a href="https://doi.org/10.1111/2041-210x.12247">https://doi.org/10.1111/2041-210x.12247</a>.
</div>
<div id="ref-Chao2010" class="csl-entry" role="listitem">
Chao, Anne, Chun-Huo Chiu, and Lou Jost. 2010. <span>“Phylogenetic Diversity Measures Based on Hill Numbers.”</span> <em>Philosophical Transactions of the Royal Society B: Biological Sciences</em> 365 (1558): 3599–609. <a href="https://doi.org/10.1098/rstb.2010.0272">https://doi.org/10.1098/rstb.2010.0272</a>.
</div>
<div id="ref-Chao2014" class="csl-entry" role="listitem">
Chao, Anne, Nicholas J. Gotelli, T. C. Hsieh, Elizabeth L. Sander, K. H. Ma, Robert K. Colwell, and Aaron M. Ellison. 2014. <span>“Rarefaction and Extrapolation with Hill Numbers: A Framework for Sampling and Estimation in Species Diversity Studies.”</span> <em>Ecological Monographs</em> 84 (1): 45–67. <a href="https://doi.org/10.1890/13-0133.1">https://doi.org/10.1890/13-0133.1</a>.
</div>
<div id="ref-Clarke1993" class="csl-entry" role="listitem">
Clarke, K. R. 1993. <span>“Non-Parametric Multivariate Analyses of Changes in Community Structure.”</span> <em>Australian Journal of Ecology</em> 18 (1): 117–43. <a href="https://doi.org/10.1111/j.1442-9993.1993.tb00438.x">https://doi.org/10.1111/j.1442-9993.1993.tb00438.x</a>.
</div>
<div id="ref-Cleveland1990" class="csl-entry" role="listitem">
Cleveland, Robert B., William S. Cleveland, Jean E. McRae, and Irma Terpenning. 1990. <span>“STL: A Seasonal-Trend Decomposition Procedure Based on Loess.”</span> <em>Journal of Official Statistics</em> 6 (1): 3–33.
</div>
<div id="ref-Colwell2012" class="csl-entry" role="listitem">
Colwell, R. K., A. Chao, N. J. Gotelli, S.-Y. Lin, C. X. Mao, R. L. Chazdon, and J. T. Longino. 2012. <span>“Models and Estimators Linking Individual-Based and Sample-Based Rarefaction, Extrapolation and Comparison of Assemblages.”</span> <em>Journal of Plant Ecology</em> 5 (1): 3–21. <a href="https://doi.org/10.1093/jpe/rtr044">https://doi.org/10.1093/jpe/rtr044</a>.
</div>
<div id="ref-Connell1978" class="csl-entry" role="listitem">
Connell, Joseph H. 1978. <span>“Diversity in Tropical Rain Forests and Coral Reefs.”</span> <em>Science</em> 199 (4335): 1302–10. <a href="https://doi.org/10.1126/science.199.4335.1302">https://doi.org/10.1126/science.199.4335.1302</a>.
</div>
<div id="ref-Cryer2008" class="csl-entry" role="listitem">
Cryer, Jonathan D., and Kung-Sik Chan. 2008. <em>Time Series Analysis: With Applications in <span>R</span></em>. 2nd ed. New York: Springer. <a href="https://doi.org/10.1007/978-0-387-75959-3">https://doi.org/10.1007/978-0-387-75959-3</a>.
</div>
<div id="ref-CSE2012" class="csl-entry" role="listitem">
CSE. 2012. <em>Protection and Management of Urban Lakes in India</em>. New Delhi: Centre for Science <span>and</span> Environment. <a href="https://www.jstor.org/stable/resrep37849">https://www.jstor.org/stable/resrep37849</a>.
</div>
<div id="ref-Denslow1980" class="csl-entry" role="listitem">
Denslow, Julie Sloan. 1980. <span>“Patterns of Plant Species Diversity During Succession Under Different Disturbance Regimes.”</span> <em>Oecologia</em> 46 (1): 18–21. <a href="https://doi.org/10.1007/bf00346960">https://doi.org/10.1007/bf00346960</a>.
</div>
<div id="ref-MODIS" class="csl-entry" role="listitem">
Didan, Kamel. 2021. <em>MODIS/Terra Vegetation Indices 16-Day L3 Global 250m SIN Grid V061</em>. NASA EOSDIS Land Processes DAAC. <a href="https://doi.org/10.5067/modis/mod13q1.061">https://doi.org/10.5067/modis/mod13q1.061</a>.
</div>
<div id="ref-Didan2015" class="csl-entry" role="listitem">
Didan, Kamel, Armando Barreto Munoz, Ramon Solano, and Alfredo Huete. 2015. <em>MODIS Vegetation Index User’s Guide (MOD13 Series) Version 3.00, (Collection 6)</em>. Vegetation Index <span>and</span> Phenology Lab, University of Arizona. <a href="https://vip.arizona.edu/documents/MODIS/MODIS_VI_UsersGuide_June_2015_C6.pdf">https://vip.arizona.edu/documents/MODIS/MODIS_VI_UsersGuide_June_2015_C6.pdf</a>.
</div>
<div id="ref-Dinnage2009" class="csl-entry" role="listitem">
Dinnage, Russell. 2009. <span>“Disturbance Alters the Phylogenetic Composition and Structure of Plant Communities in an Old Field System.”</span> <em><span>PLoS</span> <span>ONE</span></em> 4 (9): e7071. <a href="https://doi.org/10.1371/journal.pone.0007071">https://doi.org/10.1371/journal.pone.0007071</a>.
</div>
<div id="ref-eFI2022" class="csl-entry" role="listitem">
eFI. 2022. <em><span class="nocase">eFlora of India: Database of Plants of Indian Subcontinent</span></em>. <a href="https://efloraofindia.com/">https://efloraofindia.com/</a>.
</div>
<div id="ref-Elmqvist2013" class="csl-entry" role="listitem">
Elmqvist, Thomas, Michail Fragkias, Julie Goodness, Burak Güneralp, Peter J. Marcotullio, Robert I. McDonald, Susan Parnell, et al., eds. 2013. <em>Urbanization, Biodiversity and Ecosystem Services: Challenges and Opportunities</em>. Dordrecht: Springer. <a href="https://doi.org/10.1007/978-94-007-7088-1">https://doi.org/10.1007/978-94-007-7088-1</a>.
</div>
<div id="ref-Erfanian2021" class="csl-entry" role="listitem">
Erfanian, Mohammad Bagher, Juha M. Alatalo, and Hamid Ejtehadi. 2021. <span>“Severe Vegetation Degradation Associated with Different Disturbance Types in a Poorly Managed Urban Recreation Destination in Iran.”</span> <em>Scientific Reports</em> 11 (October): 19695. <a href="https://doi.org/10.1038/s41598-021-99261-5">https://doi.org/10.1038/s41598-021-99261-5</a>.
</div>
<div id="ref-Escobedo2011" class="csl-entry" role="listitem">
Escobedo, Francisco J., Timm Kroeger, and John E. Wagner. 2011. <span>“Urban Forests and Pollution Mitigation: Analyzing Ecosystem Services and Disservices.”</span> <em>Environmental Pollution</em> 159 (8-9): 2078–87. <a href="https://doi.org/10.1016/j.envpol.2011.01.010">https://doi.org/10.1016/j.envpol.2011.01.010</a>.
</div>
<div id="ref-Faith1987" class="csl-entry" role="listitem">
Faith, Daniel P., Peter R. Minchin, and Lee Belbin. 1987. <span>“Compositional Dissimilarity as a Robust Measure of Ecological Distance.”</span> <em>Vegetatio</em> 69 (1-3): 57–68. <a href="https://doi.org/10.1007/bf00038687">https://doi.org/10.1007/bf00038687</a>.
</div>
<div id="ref-R-lcvplants" class="csl-entry" role="listitem">
Freiberg, Martin, Marten Winter, Alessandro Gentile, Alexander Zizka, Alexandra Nora Muellner-Riehl, Alexandra Weigelt, and Christian Wirth. 2020. <span>“<span>LCVP</span>, the Leipzig Catalogue of Vascular Plants, a New Taxonomic Reference List for All Known Vascular Plants.”</span> <em>Scientific Data</em> 7: 416. <a href="https://doi.org/10.1038/s41597-020-00702-z">https://doi.org/10.1038/s41597-020-00702-z</a>.
</div>
<div id="ref-Gogoi2018" class="csl-entry" role="listitem">
Gogoi, Anudip, and Uttam Kumar Sahoo. 2018. <span>“Impact of Anthropogenic Disturbance on Species Diversity and Vegetation Structure of a Lowland Tropical Rainforest of Eastern Himalaya, India.”</span> <em>Journal of Mountain Science</em> 15 (11): 2453–65. <a href="https://doi.org/10.1007/s11629-017-4713-4">https://doi.org/10.1007/s11629-017-4713-4</a>.
</div>
<div id="ref-Govaerts2021" class="csl-entry" role="listitem">
Govaerts, Rafaël, Eimear Nic Lughadha, Nicholas Black, Robert Turner, and Alan Paton. 2021. <span>“The World Checklist of Vascular Plants, a Continuously Updated Resource for Exploring Global Plant Diversity.”</span> <em>Scientific Data</em> 8 (August): 215. <a href="https://doi.org/10.1038/s41597-021-00997-6">https://doi.org/10.1038/s41597-021-00997-6</a>.
</div>
<div id="ref-Grime1998" class="csl-entry" role="listitem">
Grime, J. P. 1998. <span>“Benefits of Plant Diversity to Ecosystems: Immediate, Filter and Founder Effects.”</span> <em>Journal of Ecology</em> 86 (6): 902–10. <a href="https://doi.org/10.1046/j.1365-2745.1998.00306.x">https://doi.org/10.1046/j.1365-2745.1998.00306.x</a>.
</div>
<div id="ref-Gupta2019" class="csl-entry" role="listitem">
Gupta, Neha, Aneesh Mathew, and Sumit Khandelwal. 2019. <span>“Analysis of Cooling Effect of Water Bodies on Land Surface Temperature in Nearby Region: A Case Study of Ahmedabad and Chandigarh Cities in India.”</span> <em>Egyptian Journal of Remote Sensing and Space Science</em> 22 (1): 81–93. <a href="https://doi.org/10.1016/j.ejrs.2018.03.007">https://doi.org/10.1016/j.ejrs.2018.03.007</a>.
</div>
<div id="ref-Hill1973" class="csl-entry" role="listitem">
Hill, M. O. 1973. <span>“Diversity and Evenness: A Unifying Notation and Its Consequences.”</span> <em>Ecology</em> 54 (2): 427–32. <a href="https://doi.org/10.2307/1934352">https://doi.org/10.2307/1934352</a>.
</div>
<div id="ref-R-iNextPD" class="csl-entry" role="listitem">
Hsieh, T. C., and Anne Chao. 2016. <em><span class="nocase">iNextPD</span>: Interpolation and Extrapolation for Phylogenetic Diversity</em> (version 0.3.1). <a href="https://github.com/JohnsonHsieh/iNextPD">https://github.com/JohnsonHsieh/iNextPD</a>.
</div>
<div id="ref-Hsieh2017" class="csl-entry" role="listitem">
———. 2017. <span>“Rarefaction and Extrapolation: Making Fair Comparison of Abundance-Sensitive Phylogenetic Diversity Among Multiple Assemblages.”</span> <em>Systematic Biology</em> 66 (1): 100–111. <a href="https://doi.org/10.1093/sysbio/syw073">https://doi.org/10.1093/sysbio/syw073</a>.
</div>
<div id="ref-R-iNEXT" class="csl-entry" role="listitem">
Hsieh, T. C., K. H. Ma, and Anne Chao. 2016. <span>“<span class="nocase">iNEXT</span>: An <span>R</span> Package for Rarefaction and Extrapolation of Species Diversity (Hill Numbers).”</span> <em>Methods in Ecology and Evolution</em> 7 (12): 1451–56. <a href="https://doi.org/10.1111/2041-210x.12613">https://doi.org/10.1111/2041-210x.12613</a>.
</div>
<div id="ref-R-MODISTools" class="csl-entry" role="listitem">
Hufkens, Koen. 2023. <em>The <span>MODISTools</span> Package: An Interface to the MODIS Land Products Subsets Web Services</em> (version 1.1.4). <a href="https://github.com/bluegreen-labs/MODISTools">https://github.com/bluegreen-labs/MODISTools</a>.
</div>
<div id="ref-Huston1994" class="csl-entry" role="listitem">
Huston, Michael A. 1994. <em>Biological Diversity: The Coexistence of Species on Changing Landscapes</em>. Cambridge: Cambridge University Press.
</div>
<div id="ref-R-forecast" class="csl-entry" role="listitem">
Hyndman, Rob J., and Yeasmin Khandakar. 2008. <span>“Automatic Time Series Forecasting: The <span class="nocase">forecast</span> Package for <span>R</span>.”</span> <em>Journal of Statistical Software</em> 27 (3): 1–22. <a href="https://doi.org/10.18637/jss.v027.i03">https://doi.org/10.18637/jss.v027.i03</a>.
</div>
<div id="ref-Jentsch2019" class="csl-entry" role="listitem">
Jentsch, Anke, and Peter White. 2019. <span>“A Theory of Pulse Dynamics and Disturbance in Ecology.”</span> <em>Ecology</em> 100 (7): e02734. <a href="https://doi.org/10.1002/ecy.2734">https://doi.org/10.1002/ecy.2734</a>.
</div>
<div id="ref-R-VPhyloMaker2" class="csl-entry" role="listitem">
Jin, Yi, and Hong Qian. 2022. <span>“<span>V.PhyloMaker2</span>: An Updated and Enlarged <span>R</span> Package That Can Generate Very Large Phylogenies for Vascular Plants.”</span> <em>Plant Diversity</em> 44 (4): 335–39. <a href="https://doi.org/10.1016/j.pld.2022.05.005">https://doi.org/10.1016/j.pld.2022.05.005</a>.
</div>
<div id="ref-Jong2013" class="csl-entry" role="listitem">
Jong, Rogier de, Jan Verbesselt, Achim Zeileis, and Michael Schaepman. 2013. <span>“Shifts in Global Vegetation Activity Trends.”</span> <em>Remote Sensing</em> 5 (3): 1117–33. <a href="https://doi.org/10.3390/rs5031117">https://doi.org/10.3390/rs5031117</a>.
</div>
<div id="ref-R-rstatix" class="csl-entry" role="listitem">
Kassambara, Alboukadel. 2022. <em><span class="nocase">rstatix</span>: Pipe-Friendly Framework for Basic Statistical Tests</em> (version 0.7.1). <a href="https://cran.r-project.org/package=rstatix">https://cran.r-project.org/package=rstatix</a>.
</div>
<div id="ref-Kershaw2013" class="csl-entry" role="listitem">
Kershaw, H. Maureen, and Azim U. Mallik. 2013. <span>“Predicting Plant Diversity Response to Disturbance: Applicability of the Intermediate Disturbance Hypothesis and Mass Ratio Hypothesis.”</span> <em>Critical Reviews in Plant Sciences</em> 32 (6): 383–95. <a href="https://doi.org/10.1080/07352689.2013.791501">https://doi.org/10.1080/07352689.2013.791501</a>.
</div>
<div id="ref-Khuroo2012" class="csl-entry" role="listitem">
Khuroo, Anzar A., Zafar A. Reshi, Akhtar H. Malik, Ewald Weber, Irfan Rashid, and G. H. Dar. 2012. <span>“Alien Flora of India: Taxonomic Composition, Invasion Status and Biogeographic Affiliations.”</span> <em>Biological Invasions</em> 14 (1): 99–113. <a href="https://doi.org/10.1007/s10530-011-9981-2">https://doi.org/10.1007/s10530-011-9981-2</a>.
</div>
<div id="ref-Kruskal1964" class="csl-entry" role="listitem">
Kruskal, J. B. 1964. <span>“Nonmetric Multidimensional Scaling: A Numerical Method.”</span> <em>Psychometrika</em> 29 (2): 115–29. <a href="https://doi.org/10.1007/bf02289694">https://doi.org/10.1007/bf02289694</a>.
</div>
<div id="ref-Kumar2022" class="csl-entry" role="listitem">
Kumar, Abhishek, Meenu Patil, Pardeep Kumar, Manoj Kumar, and Anand Narain Singh. 2022. <span>“Plant Ecology in Indian Siwalik Range: A Systematic Map and Its Bibliometric Analysis.”</span> <em>Tropical Ecology</em> 63 (3): 338–50. <a href="https://doi.org/10.1007/s42965-022-00229-x">https://doi.org/10.1007/s42965-022-00229-x</a>.
</div>
<div id="ref-Laska2001" class="csl-entry" role="listitem">
Łaska, Grażyna. 2001. <span>“The Disturbance and Vegetation Dynamics: A Review and an Alternative Framework.”</span> <em>Plant Ecology</em> 157 (1): 77–99. <a href="https://doi.org/10.1023/a:1013760320805">https://doi.org/10.1023/a:1013760320805</a>.
</div>
<div id="ref-Liendo2020" class="csl-entry" role="listitem">
Liendo, Diego, Idoia Biurrun, Juan Antonio Campos, Itziar Garcı́a-Mijangos, and Peter B. Pearman. 2020. <span>“Effects of Disturbance and Alien Plants on the Phylogenetic Structure of Riverine Communities.”</span> <em>Journal of Vegetation Science</em> 32 (1): e12933. <a href="https://doi.org/10.1111/jvs.12933">https://doi.org/10.1111/jvs.12933</a>.
</div>
<div id="ref-R-signal" class="csl-entry" role="listitem">
Ligges, Uwe, Tom Short, Paul Kienzle, Sarah Schnackenberg, David Billinghurst, Hans-Werner Borchers, Andre Carezia, et al. 2022. <em><span class="nocase">signal</span>: Signal Processing</em> (version 0.7.7). <a href="https://cran.r-project.org/package=signal">https://cran.r-project.org/package=signal</a>.
</div>
<div id="ref-Maansi2021" class="csl-entry" role="listitem">
Maansi, Rajinder Jindal, and Meenu Wats. 2021. <span>“Evaluation of Surface Water Quality Using Water Quality Indices (<span>WQIs</span>) in Lake Sukhna, Chandigarh, India.”</span> <em>Applied Water Science</em> 12 (November): 2. <a href="https://doi.org/10.1007/s13201-021-01534-x">https://doi.org/10.1007/s13201-021-01534-x</a>.
</div>
<div id="ref-Minchin1987" class="csl-entry" role="listitem">
Minchin, Peter R. 1987. <span>“An Evaluation of the Relative Robustness of Techniques for Ecological Ordination.”</span> <em>Vegetatio</em> 69 (1-3): 89–107. <a href="https://doi.org/10.1007/bf00038690">https://doi.org/10.1007/bf00038690</a>.
</div>
<div id="ref-R-vegan" class="csl-entry" role="listitem">
Oksanen, Jari, Gavin L. Simpson, F. Guillaume Blanchet, Roeland Kindt, Pierre Legendre, Peter R. Minchin, R. B. O’Hara, et al. 2022. <em><span class="nocase">vegan</span>: Community Ecology Package</em> (version 2.6.4). <a href="https://cran.r-project.org/package=vegan">https://cran.r-project.org/package=vegan</a>.
</div>
<div id="ref-Patil2020" class="csl-entry" role="listitem">
Patil, Meenu, Abhishek Kumar, Pardeep Kumar, Navneet Kaur Cheema, Rupinder Kaur, Ramchand Bhatti, and A. N. Singh. 2020. <span>“Comparative Litter Decomposability Traits of Selected Native and Exotic Woody Species from an Urban Environment of North-Western Siwalik Region, India.”</span> <em>Scientific Reports</em> 10 (May): 7888. <a href="https://doi.org/10.1038/s41598-020-64576-2">https://doi.org/10.1038/s41598-020-64576-2</a>.
</div>
<div id="ref-Pesaran2002" class="csl-entry" role="listitem">
Pesaran, M.Hashem, and Allan Timmermann. 2002. <span>“Market Timing and Return Prediction Under Model Instability.”</span> <em>Journal of Empirical Finance</em> 9 (5): 495–510. <a href="https://doi.org/10.1016/s0927-5398(02)00007-5">https://doi.org/10.1016/s0927-5398(02)00007-5</a>.
</div>
<div id="ref-Pickett2011" class="csl-entry" role="listitem">
Pickett, S. T. A., M. L. Cadenasso, J. M. Grove, Christopher G. Boone, Peter M. Groffman, Elena Irwin, Sujay S. Kaushal, et al. 2011. <span>“Urban Ecological Systems: Scientific Foundations and a Decade of Progress.”</span> <em>Journal of Environmental Management</em> 92 (3): 331–62. <a href="https://doi.org/10.1016/j.jenvman.2010.08.022">https://doi.org/10.1016/j.jenvman.2010.08.022</a>.
</div>
<div id="ref-POWO2023" class="csl-entry" role="listitem">
POWO. 2023. <em>Plants of the World Online</em>. Kew: Royal Botanic Gardens. <a href="http://www.plantsoftheworldonline.org/">http://www.plantsoftheworldonline.org/</a>.
</div>
<div id="ref-Qian2016" class="csl-entry" role="listitem">
Qian, Hong, and Yi Jin. 2015. <span>“An Updated Megaphylogeny of Plants, a Tool for Generating Plant Phylogenies and an Analysis of Phylogenetic Community Structure.”</span> <em>Journal of Plant Ecology</em> 9 (2): 233–39. <a href="https://doi.org/10.1093/jpe/rtv047">https://doi.org/10.1093/jpe/rtv047</a>.
</div>
<div id="ref-R-base" class="csl-entry" role="listitem">
R Core Team. 2023. <em>R: A Language and Environment for Statistical Computing</em> (version 4.3.0). Vienna, Austria: R Foundation for Statistical Computing. <a href="https://www.R-project.org/">https://www.R-project.org/</a>.
</div>
<div id="ref-Roman2018" class="csl-entry" role="listitem">
Roman, Lara A., Hamil Pearsall, Theodore S. Eisenman, Tenley M. Conway, Robert T. Fahey, Shawn Landry, Jess Vogt, et al. 2018. <span>“Human and Biophysical Legacies Shape Contemporary Urban Forests: A Literature Synthesis.”</span> <em>Urban Forestry <span>&amp;</span> Urban Greening</em> 31 (April): 157–68. <a href="https://doi.org/10.1016/j.ufug.2018.03.004">https://doi.org/10.1016/j.ufug.2018.03.004</a>.
</div>
<div id="ref-Roswell2021" class="csl-entry" role="listitem">
Roswell, Michael, Jonathan Dushoff, and Rachael Winfree. 2021. <span>“A Conceptual Guide to Measuring Species Diversity.”</span> <em>Oikos</em> 130 (3): 321–38. <a href="https://doi.org/10.1111/oik.07202">https://doi.org/10.1111/oik.07202</a>.
</div>
<div id="ref-Savitzky1964" class="csl-entry" role="listitem">
Savitzky, Abraham, and M. J. E. Golay. 1964. <span>“Smoothing and Differentiation of Data by Simplified Least Squares Procedures.”</span> <em>Analytical Chemistry</em> 36 (8): 1627–39. <a href="https://doi.org/10.1021/ac60214a047">https://doi.org/10.1021/ac60214a047</a>.
</div>
<div id="ref-Semwal2017" class="csl-entry" role="listitem">
Semwal, P., S. D. Khobragade, and H. C. Nainwal. 2017. <span>“Modelling of Recent Erosion Rates in a Lake Catchment in the North-Western Siwalik Himalayas.”</span> <em>Environmental Processes</em> 4 (2): 355–74. <a href="https://doi.org/10.1007/s40710-017-0234-y">https://doi.org/10.1007/s40710-017-0234-y</a>.
</div>
<div id="ref-Shafroth2002" class="csl-entry" role="listitem">
Shafroth, Patrick B., Juliet C. Stromberg, and Duncan T. Patten. 2002. <span>“Riparian Vegetation Response to Altered Disturbance and Stress Regimes.”</span> <em>Ecological Applications</em> 12 (1): 107–23. <a href="https://doi.org/10.1890/1051-0761(2002)012[0107:rvrtad]2.0.co;2">https://doi.org/10.1890/1051-0761(2002)012[0107:rvrtad]2.0.co;2</a>.
</div>
<div id="ref-Shannon1948" class="csl-entry" role="listitem">
Shannon, C. E. 1948. <span>“A Mathematical Theory of Communication.”</span> <em>Bell System Technical Journal</em> 27 (3): 379–423. <a href="https://doi.org/10.1002/j.1538-7305.1948.tb01338.x">https://doi.org/10.1002/j.1538-7305.1948.tb01338.x</a>.
</div>
<div id="ref-Shrimali2001" class="csl-entry" role="listitem">
Shrimali, S. S., S. P. Aggarwal, and J. S. Samra. 2001. <span>“Prioritizing Erosion-Prone Areas in Hills Using Remote Sensing and <span>GIS</span> – a Case Study of the Sukhna Lake Catchment, Northern India.”</span> <em>International Journal of Applied Earth Observation and Geoinformation</em> 3 (1): 54–60. <a href="https://doi.org/10.1016/s0303-2434(01)85021-2">https://doi.org/10.1016/s0303-2434(01)85021-2</a>.
</div>
<div id="ref-Simpson1949" class="csl-entry" role="listitem">
Simpson, E. H. 1949. <span>“Measurement of Diversity.”</span> <em>Nature</em> 163 (April): 688–88. <a href="https://doi.org/10.1038/163688a0">https://doi.org/10.1038/163688a0</a>.
</div>
<div id="ref-Singh2019" class="csl-entry" role="listitem">
Singh, Durgesh Kumar, and Nandita Singh. 2019. <span>“Drying Urban Lakes: A Consequence of Climate Change, Urbanization or Other Anthropogenic Causes? An Insight from Northern India.”</span> <em>Lakes <span>&amp;</span> Reservoirs</em> 24 (2): 115–26. <a href="https://doi.org/10.1111/lre.12262">https://doi.org/10.1111/lre.12262</a>.
</div>
<div id="ref-Singh2001" class="csl-entry" role="listitem">
Singh, Yadvinder. 2001. <span>“Geo-Ecology of the Trans Satluj Punjab-Haryana Siwalik Hills, NW India.”</span> <em>ENVIS Bulletin: Himalayan Ecology</em> 9 (2): 19–38. <a href="http://gbpihedenvis.nic.in/ENVIS%20Bullitin/Vol.%209/vol9_2.PDF">http://gbpihedenvis.nic.in/ENVIS%20Bullitin/Vol.%209/vol9_2.PDF</a>.
</div>
<div id="ref-Singh2002" class="csl-entry" role="listitem">
———. 2002. <span>“Siltation Problems in <span class="nocase">Sukhna lake</span> in <span>Chandigarh</span>, NW <span>India</span> and Comments on Geohydrogical Changes in the <span>Yamuna-Satluj</span> Region.”</span> <em>ENVIS Bulletin: Himalayan Ecology</em> 10 (2): 18–31. <a href="http://gbpihedenvis.nic.in/ENVIS%20Bullitin/Vol.%2010/vol10_2.pdf">http://gbpihedenvis.nic.in/ENVIS%20Bullitin/Vol.%2010/vol10_2.pdf</a>.
</div>
<div id="ref-Smith2018" class="csl-entry" role="listitem">
Smith, Stephen A., and Joseph W. Brown. 2018. <span>“Constructing a Broadly Inclusive Seed Plant Phylogeny.”</span> <em>American Journal of Botany</em> 105 (3): 302–14. <a href="https://doi.org/10.1002/ajb2.1019">https://doi.org/10.1002/ajb2.1019</a>.
</div>
<div id="ref-Verbesselt2010a" class="csl-entry" role="listitem">
Verbesselt, Jan, Rob Hyndman, Glenn Newnham, and Darius Culvenor. 2010. <span>“Detecting Trend and Seasonal Changes in Satellite Image Time Series.”</span> <em>Remote Sensing of Environment</em> 114 (1): 106–15. <a href="https://doi.org/10.1016/j.rse.2009.08.014">https://doi.org/10.1016/j.rse.2009.08.014</a>.
</div>
<div id="ref-Verbesselt2010b" class="csl-entry" role="listitem">
Verbesselt, Jan, Rob Hyndman, Achim Zeileis, and Darius Culvenor. 2010. <span>“Phenological Change Detection While Accounting for Abrupt and Gradual Trends in Satellite Image Time Series.”</span> <em>Remote Sensing of Environment</em> 114 (12): 2970–80. <a href="https://doi.org/10.1016/j.rse.2010.08.003">https://doi.org/10.1016/j.rse.2010.08.003</a>.
</div>
<div id="ref-R-bfast" class="csl-entry" role="listitem">
Verbesselt, Jan, Dainius Masiliunas, Achim Zeileis, Rob Hyndman, Marius Appel, Martin Jung, Andrei Mirt, Paulo Negri Bernardino, and Dongdong Kong. 2021. <em><span class="nocase">bfast</span>: Breaks for Additive Season and Trend</em> (version 1.6.1). <a href="https://cran.r-project.org/package=bfast">https://cran.r-project.org/package=bfast</a>.
</div>
<div id="ref-Verbesselt2012" class="csl-entry" role="listitem">
Verbesselt, Jan, Achim Zeileis, and Martin Herold. 2012. <span>“Near Real-Time Disturbance Detection Using Satellite Image Time Series.”</span> <em>Remote Sensing of Environment</em> 123 (August): 98–108. <a href="https://doi.org/10.1016/j.rse.2012.02.022">https://doi.org/10.1016/j.rse.2012.02.022</a>.
</div>
<div id="ref-R-tidyverse" class="csl-entry" role="listitem">
Wickham, Hadley, Mara Averick, Jennifer Bryan, Winston Chang, Lucy McGowan, Romain François, Garrett Grolemund, et al. 2019. <span>“Welcome to the Tidyverse.”</span> <em>Journal of Open Source Software</em> 4 (43): 1686. <a href="https://doi.org/10.21105/joss.01686">https://doi.org/10.21105/joss.01686</a>.
</div>
<div id="ref-Wohlgemuth2022" class="csl-entry" role="listitem">
Wohlgemuth, Thomas, Anke Jentsch, and Rupert Seidl, eds. 2022. <em>Disturbance Ecology</em>. Cham: Springer. <a href="https://doi.org/10.1007/978-3-030-98756-5">https://doi.org/10.1007/978-3-030-98756-5</a>.
</div>
<div id="ref-Zanne2014" class="csl-entry" role="listitem">
Zanne, Amy E., David C. Tank, William K. Cornwell, Jonathan M. Eastman, Stephen A. Smith, Richard G. FitzJohn, Daniel J. McGlinn, et al. 2014. <span>“Three Keys to the Radiation of Angiosperms into Freezing Environments.”</span> <em>Nature</em> 506 (7486): 89–92. <a href="https://doi.org/10.1038/nature12872">https://doi.org/10.1038/nature12872</a>.
</div>
</div></section></div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>